## Detecting bacterial adaptation within individual microbiomes.
 23.08.2022, _TD Lieberman_


_Acclimatization, Bacteria, Genome, Genomics, Humans, Microbiota_
The human microbiome harbours a large capacity for within-person adaptive mutations. Commensal bacterial strains can stably colonize a person for decades, and billions of mutations are generated daily within each person's microbiome. Adaptive mutations emerging during health might be driven by selective forces that vary across individuals, vary within an individual, or are completely novel to the human population. Mutations emerging within individual microbiomes might impact the immune system, the metabolism of nutrients or drugs, and the stability of the community to perturbations. Despite this potential, relatively little attention has been paid to the possibility of adaptive evolution within complex human-associated microbiomes. This review discusses the promise of studying within-microbiome adaptation, the conceptual and technical limitations that may have contributed to an underappreciation of adaptive de novo mutations occurring within microbiomes to date, and methods for detecting recent adaptive evolution. This article is part of a discussion meeting issue 'Genomic population structures of microbial pathogens'.

[Source](https://royalsocietypublishing.org/doi/10.1098/rstb.2021.0243)

10.1098/rstb.2021.0243

---

## Stunting as a Risk Factor of Soil-Transmitted Helminthiasis in Children: A Literature Review.
 16.08.2022, _N Fauziah, MA Ar-Rizqi, S Hana, NM Patahuddin, A Diptyanusa_


As a high-burden neglected tropical disease, soil-transmitted helminth (STH) infections remain a major problem in the world, especially among children under five years of age. Since young children are at high risk of being infected, STH infection can have a long-term negative impact on their life, including impaired growth and development. Stunting, a form of malnutrition in young children, has been long assumed as one of the risk factors in acquiring the STH infections. However, the studies on STH infection in children under five with stunting have been lacking, resulting in poor identification of the risk. Accordingly, we collected and reviewed existing related research articles to provide an overview of STH infection in a susceptible population of stunted children under five years of age in terms of prevalence and risk factors. There were 17 studies included in this review related to infection with Ascaris lumbricoides, Trichuris trichiura, hookworm, and Strongyloides stercoralis from various countries. The prevalence of STH infection in stunted children ranged from 12.5% to 56.5%. Increased inflammatory markers and intestinal microbiota dysbiosis might have increased the intensity of STH infection in stunted children that caused impairment in the immune system. While the age from 2 to 5 years along with poor hygiene and sanitation has shown to be the most common risk factors of STH infections in stunted children; currently there are no studies that show direct results of stunting as a risk factor for STH infection. While stunting itself may affect the pathogenesis of STH infection, further research on stunting as a risk factor for STH infection is encouraged.

[Source](https://www.nature.com/articles/s41579-022-00779-w)

10.1155/2022/8929025

---

## Soil microbiomes and one health

[Source](https://www.nature.com/articles/s41579-022-00779-w)

---

## Microbial Analysis of Umbilical Cord Blood Reveals Novel Pathogens Associated with Stillbirth and Early Preterm Birth.
 22.08.2022, _EL Vander Haar, G Wu, C Gyamfi-Bannerman, C Thomas, RJ Wapner, UM Reddy, L Zhao, RM Silver, RL Goldenberg, YW Han_


Stillbirths account for half of all perinatal mortality, but the underlying cause of a significant portion of the cases remains unknown. We set out to test the potential role and extent of microbial infection in stillbirth through a case-control analysis of fetal cord blood collected from the multisite Stillbirth Collaborative Research Network. Cases (n = 60) were defined as stillbirths at &gt;20 weeks of gestation, and controls (n = 176) were live births. The bacterial presence, abundance, and composition were analyzed by endpoint PCR of full-length 16S rRNA and the V4 amplicon sequence variants (ASVs). The results demonstrate that bacterial prevalence and abundance were both significantly increased in stillbirth, even after adjusting for maternal age, race, body mass index, number of pregnancies, gestational age, and multiple gestations. Composition of bacterial communities in the cord blood also differed significantly. Using a group of 25 ASVs differentially abundant between the two groups, a Random Forest classification model achieved an accuracy score of 0.76 differentiating stillbirth and live birth, with Group B Streptococcus as the most enriched species in stillbirth. Positive PCR was also significantly associated with early preterm birth. A group of oral anaerobes, including Actinomyces, Campylobacter, Fusobacterium, Peptostreptococcus, Porphyromonas, and Prevotella, were enriched in live early preterm birth, suggesting possible oral origin of infection. Our ASV-based microbiome analysis revealed specific candidate pathogens associated with infections in stillbirth and early preterm birth. The cord blood microbial signatures may be markers of adverse pregnancy outcomes. Our study will help identify possible mechanism of infection and improve our ability to prevent stillbirth and early preterm birth. IMPORTANCE Stillbirth accounts for half of all perinatal mortality, but the underlying cause of a substantial portion of all cases remains elusive. We examined the umbilical cord blood microbiome in stillbirths (n = 60) and live births (n = 176) and discovered that the bacterial prevalence and abundance were significantly higher in stillbirths than live births. The microbial compositions also differed significantly. Group B Streptococcus was the most prevalent species detected in stillbirth. In addition, pathogens previously unknown to be associated with stillbirth were identified. A group of oral anaerobes including Fusobacterium nucleatum were found to be specifically enriched in the cord blood in early preterm live birth. This is by far the most comprehensive study to examine the microbial signatures in umbilical cord blood. Cord blood microbial signatures may be markers for adverse birth outcomes. Detection of key microbial signatures will help identify individuals at risk and develop effective preventative strategies.

[Source](https://journals.asm.org/doi/10.1128/mbio.02036-22)

10.1128/mbio.02036-22

---

## The respiratory microbiota alpha-diversity in chronic lung diseases: first systematic review and meta-analysis

[Source](https://respiratory-research.biomedcentral.com/articles/10.1186/s12931-022-02132-4)

---

## Saliva microbiome, dietary, and genetic markers are associated with suicidal ideation in university students.
 22.08.2022, _AP Ahrens, DE Sanchez-Padilla, JC Drew, MW Oli, LFW Roesch, EW Triplett_


Here, salivary microbiota and major histocompatibility complex (MHC) human leukocyte antigen (HLA) alleles were compared between 47 (12.6%) young adults with recent suicidal ideation (SI) and 325 (87.4%) controls without recent SI. Several bacterial taxa were correlated with SI after controlling for sleep issues, diet, and genetics. Four MHC class II alleles were protective for SI including DRB1*04, which was absent in every subject with SI while present in 21.7% of controls. Increased incidence of SI was observed with four other MHC class II alleles and two MHC class I alleles. Associations between these HLA alleles and salivary bacteria were also identified. Furthermore, rs10437629, previously associated with attempted suicide, was correlated here with SI and the absence of Alloprevotella rava, a producer of an organic acid known to promote brain energy homeostasis. Hence, microbial-genetic associations may be important players in the diathesis-stress model for suicidal behaviors.

[Source](https://www.nature.com/articles/s41598-022-18020-2)

10.1038/s41598-022-18020-2

---

## Fate of Escherichia coli artificially inoculated in Tenebrio molitor L. larvae rearing chain for human consumption.
 29.06.2022, _C Cesaro, C Mannozzi, A Lepre, I Ferrocino, L Corsi, I Franciosa, L Belleggia, V Milanović, F Cardinali, C Garofalo, L Cocolin, L Aquilanti, S Ruschioni, N Isidoro, P Riolo, A Osimani_


_Animals, Edible Insects, Escherichia coli, Humans, Larva, Pupa, Tenebrio_
The edible insect food chain represents a relatively novel food-producing system; hence, associated biological risks still need to be exhaustively evaluated. In the present study, the dynamics of Escherichia coli during the whole living period of Tenebrio molitor larvae (from eggs to pupae) were studied. To this end, a rearing substrate consisting of organic wheat middlings was spiked with E. coli cells at two initial contamination levels: 1 log cfu g-1 (low) and 6 log cfu g-1 (high). Microbial viability counting coupled with metataxonomic analyses was used to assess i) the persistence and growth of E. coli in the rearing substrate (wheat middlings); ii) the colonization and growth of E. coli in the insect larvae; and iii) the occurrence and load of E. coli in the frass (excrement from larvae mixed with substrate residues). The results highlighted a very limited persistence of the pathogen in all analyzed samples. In more detail, the results suggested that when E. coli was present at very low levels in the eggs of the insect, the pathogen was not able to reach concerning levels in the larvae. Moreover, when E. coli was present in the wheat middlings used for rearing, the environmental conditions of the substrate (low aw values) were not favorable for its survival and multiplication, irrespective of the presence of the larvae and their frass. Surprisingly, under the conditions applied in the present study, the larvae fed wheat middlings contaminated with E. coli seemed to be inhospitable or even hostile environments for microbial survival or multiplication. To explain the low levels of E. coli cells in the larvae reared in the present study, many factors can be considered, including the immune response of the host, microbial composition and interactions established in the gut of larvae, and insect species. Of note, part of the major fraction of the microbiota of larvae at the end of rearing was represented by Lactococcus, thus suggesting a possible effect of this lactic acid bacterium on E. coli decay. Further research is needed to better clarify the interactions between E. coli and the insect gut, as well as the interactions established among the target microorganism and those naturally harbored by the insect gut.

[Source](https://www.nature.com/articles/s41586-020-2971-8)

10.1016/j.foodres.2022.111269

---

## The gut microbiota is associated with immune cell dynamics in humans

[Source](https://www.nature.com/articles/s41586-020-2971-8)

---

## The systemic anti-microbiota IgG repertoire can identify gut bacteria that translocate across gut barrier surfaces.
 19.08.2022, _I Vujkovic-Cvijin, HC Welles, CWY Ha, L Huq, S Mistry, JM Brenchley, G Trinchieri, S Devkota, Y Belkaid_


_Animals, Bacteria, Gastrointestinal Microbiome, Humans, Immunoglobulin G, Inflammatory Bowel Diseases, Mice, Microbiota_
Unique gut microbiota compositions have been associated with inflammatory diseases, but identifying gut bacterial functions linked to immune activation in humans remains challenging. Translocation of pathogens from mucosal surfaces into peripheral tissues can elicit immune activation, although whether and which gut commensal bacteria translocate in inflammatory diseases is difficult to assess. We report that a subset of commensal gut microbiota constituents that translocate across the gut barrier in mice and humans are associated with heightened systemic immunoglobulin G (IgG) responses. We present a modified high-throughput, culture-independent approach to quantify systemic IgG against gut commensal bacteria in human serum samples without the need for paired stool samples. Using this approach, we highlight several commensal bacterial species that elicit elevated IgG responses in patients with inflammatory bowel disease (IBD) including taxa within the clades Collinsella, Bifidobacterium, Lachnospiraceae, and Ruminococcaceae. These and other taxa identified as translocating bacteria or targets of systemic immunity in IBD concomitantly exhibited heightened transcriptional activity and growth rates in IBD patient gut microbiomes. Our approach represents a complementary tool to illuminate interactions between the host and its gut microbiota and may provide an additional method to identify microbes linked to inflammatory disease.

[Source](https://www.science.org/doi/abs/10.1126/scitranslmed.abl3927)

10.1126/scitranslmed.abl3927

---

## Higher pathogen load in children from Mozambique vs. USA revealed by comparative fecal microbiome profiling

[Source](https://www.nature.com/articles/s43705-022-00154-z)

---

## Correlation of inflammatory biomarkers with the diversity of Bacteroidaceae, Bifidobacteriaceae, Prevotellaceae and Lactobacillaceae families in the intestinal microbiota of patients with end stage renal disease.
 22.08.2022, _M Asgharian, P Gholizadeh, H Samadi Kafil, M Ghojazadeh, A Samadi, J Soleymani, A Jouyban, H Tayebi Khosroshahi_


Serum levels of inflammatory cytokines and uremic toxins, and their inter-correlations with the diversity of Bacteroidaceae, Bifidobacteriaceae, Prevotellaceae and Lactobacillaceae families in intestinal microbiota were investigated in patients with end stage renal disease (ESRD).

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1896112622000281)

10.1016/j.advms.2022.07.004

---

## Impact of the Host Microbiome on Vaccine Responsiveness: Lessons Learned and Future Perspective.
 22.08.2022, _G Stefanetti, DL Kasper_


Vaccination shows high variability in the elicited immune responses among individuals and populations for reasons still poorly understood. An increasing number of studies is supporting the evidence that gut microbiota, along with other interplaying variables, is able to modulate both humoral and cellular responses to infection and vaccination. Importantly, vaccine immunogenicity is often suboptimal at the extremes of age and also in low- and middle-income countries (LMICs), where the microbiota is believed to have an important role on immune responses. Still, contrasting findings and lack of causal evidence are calling for sophisticated methodologies to be able to overcome scientific and technical challenges to better decipher the immunomodulatory role of microbiota. In this perspective, we briefly review the status of the vaccine field in relation to the microbiome and offer possible scientific approaches to better understand the impact of the host microbiome on vaccine responsiveness.

[Source](https://pubs.acs.org/doi/10.1021/acs.biochem.2c00309)

10.1021/acs.biochem.2c00309

---

## Location-specific signatures of Crohn’s disease at a multi-omics scale

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01331-x)

---

## Association of food insecurity on gut microbiome and metabolome profiles in a diverse college-based sample

[Source](https://www.nature.com/articles/s41598-022-18515-y)

---

## The Diversity and Functional Capacity of Microbes Associated with Coastal Macrophytes

[Source](https://journals.asm.org/doi/10.1128/msystems.00592-22)

---

## Machine learning algorithm to characterize antimicrobial resistance associated with the International Space Station surface microbiom

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-022-01332-w)

---

## Extending and improving metagenomic taxonomic profiling with uncharacterized species with MetaPhlAn 4

[Source](https://www.biorxiv.org/content/10.1101/2022.08.22.504593v1)

---

## Linear and nonlinear correlation estimators unveil undescribed taxa interactions in microbiome data

[Source](https://www.nature.com/articles/s41467-022-32243-x)

---

## Primers matter: Influence of the primer selection on human fungal detection using high throughput sequencing

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2022.2110638)

---

## Using Nanopore Sequencing to Obtain Complete Bacterial Genomes from Saliva Samples

[Source](https://journals.asm.org/doi/10.1128/msystems.00491-22)

---

