## Novel biomarkers for hepatocellular carcinoma detection and treatment.
 07.11.2024, _H Yi, H Xu_



[Source](https://doi.org/10.21037/hbsn-24-517)

10.21037/hbsn-24-517

---

## Gut microbiota modulates depressive-like behaviors induced by chronic ethanol exposure through short-chain fatty acids.
 07.11.2024, _H Shen, C Zhang, Q Zhang, Q Lv, H Liu, H Yuan, C Wang, F Meng, Y Guo, J Pei, C Yu, J Tie, X Chen, H Yu, G Zhang, X Wang_


Chronic ethanol exposure (CEE) is recognized as an important risk factor for depression, and the gut-brain axis has emerged as a key mechanism underlying chronic ethanol exposure-induced anxiety and depression-like behaviors. Short-chain fatty acids (SCFAs), which are the key metabolites generated by gut microbiota from insoluble dietary fiber, exert protective roles on the central nervous system, including the reduction of neuroinflammation. However, the link between gut microbial disturbances caused by chronic ethanol exposure, production of SCFAs, and anxiety and depression-like behaviors remains unclear.

[Source](https://doi.org/10.1186/s12974-024-03282-6)

10.1186/s12974-024-03282-6

---

## Host genetics and gut microbiota influence lipid metabolism and inflammation: potential implications for ALS pathophysiology in SOD1G93A mice.
 07.11.2024, _E Niccolai, L Di Gloria, MC Trolese, P Fabbrizio, S Baldi, G Nannini, C Margotta, C Nastasi, M Ramazzotti, G Bartolucci, C Bendotti, G Nardo, A Amdei_


_Animals, Gastrointestinal Microbiome, Amyotrophic Lateral Sclerosis, Mice, Mice, Transgenic, Lipid Metabolism, Inflammation, Mice, Inbred C57BL, Superoxide Dismutase-1, Disease Models, Animal_
Amyotrophic Lateral Sclerosis (ALS) is a devastating neurodegenerative disorder characterized by the progressive loss of motor neurons, with genetic and environmental factors contributing to its complex pathogenesis. Dysregulated immune responses and altered energetic metabolism are key features, with emerging evidence implicating the gut microbiota (GM) in disease progression. We investigated the interplay among genetic background, GM composition, metabolism, and immune response in two distinct ALS mouse models: 129Sv_G93A and C57Ola_G93A, representing rapid and slow disease progression, respectively.Using 16 S rRNA sequencing and fecal metabolite analysis, we characterized the GM composition and metabolite profiles in non-transgenic (Ntg) and SOD1G93A mutant mice of both strains. Our results revealed strain-specific differences in GM composition and functions, particularly in the abundance of taxa belonging to Erysipelotrichaceae and the levels of short and medium-chain fatty acids in fecal samples. The SOD1 mutation induces significant shifts in GM colonization in both strains, with C57Ola_G93A mice showing changes resembling those in 129 Sv mice, potentially affecting disease pathogenesis. ALS symptom progression does not significantly alter microbiota composition, suggesting stability.Additionally, we assessed systemic immunity and inflammatory responses revealing strain-specific differences in immune cell populations and cytokine levels.Our findings underscore the substantial influence of genetic background on GM composition, metabolism, and immune response in ALS mouse models. These strain-specific variations may contribute to differences in disease susceptibility and progression rates. Further elucidating the mechanisms underlying these interactions could offer novel insights into ALS pathogenesis and potential therapeutic targets.

[Source](https://doi.org/10.1186/s40478-024-01877-x)

10.1186/s40478-024-01877-x

---

## <em>Lactobacillus rhamnosus</em> modulates murine neonatal gut microbiota and inflammation caused by pathogenic <em>Escherichia coli</em>

[Source](https://doi.org/10.1186/s12866-024-03598-6)

---

## Introduction into natural environments shifts the gut microbiome of captivity-raised filter-feeding bivalves.
 07.11.2024, _SN Vaughn, GW Hopper, IS González, JR Bucholz, RC Garrick, JD Lozier, PD Johnson, CL Atkinson, CR Jackson_


The gut microbiome is influenced by host species and the environment, but how the environment influences the microbiome of animals introduced into a new ecosystem has rarely been investigated. Freshwater mussels are aquatic fauna, with some threatened or endangered species propagated in hatcheries and introduced into natural systems as part of conservation efforts. The effects of the environment on the freshwater mussel gut microbiome were assessed for two hatchery-propagated species (Lampsilis ovata, Lampsilis ornata) introduced into rivers within their natural range. Mussels were placed in rivers for 8 weeks, after which one subset was collected, another subset remained in that river, and a third subset was reciprocally transplanted to another river in the same river basin for a further 8 weeks. Gut microbiome composition and diversity were characterized for all mussels. After the initial 8 weeks, mussels showed increased gut bacterial species richness and distinct community composition compared to hatchery mussels, but gut microbiome diversity then decreased for mussels that remained in the same river for all 16 weeks. The gut bacterial community of mussels transplanted between rivers shifted to resemble that of mussels placed initially into the recipient river and that remained there for the whole study. All mussels showed high proportions of Firmicutes in their gut microbiome after 8 weeks, suggesting an essential role of this phylum in the gut of Lampsilis species. These findings show that the mussel gut microbiome shifts in response to new environments and provide insights into conservation strategies that involve species reintroductions.

[Source](https://doi.org/10.1093/ismeco/ycae125)

10.1093/ismeco/ycae125

---

## A metagenomic catalogue of the ruminant gut archaeome.
 06.11.2024, _J Mi, X Jing, C Ma, F Shi, Z Cao, X Yang, Y Yang, A Kakade, W Wang, R Long_


_Animals, Ruminants, Gastrointestinal Microbiome, Metagenomics, Archaea, Genome, Archaeal, Metagenome, Methane, Phylogeny, Gastrointestinal Tract_
While the ruminant gut archaeome regulates the gut microbiota and hydrogen balance, it is also a major producer of the greenhouse gas methane. However, ruminant gut archaeome diversity within the gastrointestinal tract (GIT) of ruminant animals worldwide remains largely underexplored. Here, we construct a catalogue of 998 unique archaeal genomes recovered from the GITs of ruminants, utilizing 2270 metagenomic samples across 10 different ruminant species. Most of the archaeal genomes (669/998 = 67.03%) belong to Methanobacteriaceae and Methanomethylophilaceae (198/998 = 19.84%). We recover 47/279 previously undescribed archaeal genomes at the strain level with completeness of &gt;80% and contamination of &lt;5%. We also investigate the archaeal gut biogeography across various ruminants and demonstrate that archaeal compositional similarities vary significantly by breed and gut location. The catalogue contains 42,691 protein clusters, and the clustering and methanogenic pathway analysis reveal strain- and host-specific dependencies among ruminant animals. We also find that archaea potentially carry antibiotic and metal resistance genes, mobile genetic elements, virulence factors, quorum sensors, and complex archaeal viromes. Overall, this catalogue is a substantial repository for ruminant archaeal recourses, providing potential for advancing our understanding of archaeal ecology and discovering strategies to regulate methane production in ruminants.

[Source](https://doi.org/10.1038/s41467-024-54025-3)

10.1038/s41467-024-54025-3

---

## Large-scale metagenomic assembly provide new insights into the genetic evolution of gut microbiomes in plateau ungulates.
 06.11.2024, _B Xu, P Song, F Jiang, Z Cai, H Gu, H Gao, B Li, C Liang, W Qin, J Zhang, J Yan, D Liu, G Sun, T Zhang_


_Gastrointestinal Microbiome, Animals, Metagenomics, Evolution, Molecular, China, Metagenome, Bacteria, Phylogeny, Fatty Acids, Volatile, Genome, Bacterial_
Trillions of microbes colonize the ungulate gastrointestinal tract, playing a pivotal role in enhancing host nutrient utilization by breaking down cellulose and hemicellulose present in plants. Here, through large-scale metagenomic assembly, we established a catalog of 131,416 metagenome-assembled genomes (MAGs) and 11,175 high-quality species-level genome bins (SGBs) from 17 species of ungulates in China. Our study revealed the convergent evolution of high relative abundances of carbohydrate-active enzymes (CAZymes) in the gut microbiomes of plateau-dwelling ungulates. Notably, two significant factors contribute to this phenotype: structural variations in their gut microbiome genomes, which contain more CAZymes, and the presence of novel gut microbiota species, particularly those in the genus Cryptobacteroides, which are undergoing independent rapid evolution and speciation and have higher gene densities of CAZymes. Furthermore, these enrichment CAZymes in the gut microbiomes are highly enrichment in known metabolic pathways for short-chain fatty acid (SCFA) production. Our findings not only provide a valuable genomic resource for understanding the gut microbiomes of ungulates but also offer fresh insights into the interaction between gut microbiomes and their hosts, as well as the co-adaptation of hosts and their gut microbiomes to their environments.

[Source](https://doi.org/10.1038/s41522-024-00597-3)

10.1038/s41522-024-00597-3

---

## Diet affects reproductive development and microbiota composition in honey bees.
 06.11.2024, _A Zumkhawala-Cook, P Gallagher, K Raymann_


Gut microbes are important to the health and fitness of many animals. Many factors have been shown to affect gut microbial communities including diet, lifestyle, and age. Most animals have very complex physiologies, lifestyles, and microbiomes, making it virtually impossible to disentangle what factors have the largest impact on microbiota composition. Honeybees are an excellent model to study host-microbe interactions due to their relatively simple gut microbiota, experimental tractability, and eusociality. Worker honey bees have distinct gut microbiota from their queen mothers despite being close genetic relatives and living in the same environment. Queens and workers differ in numerous ways including development, physiology, pheromone production, diet, and behavior. In the prolonged absence of a queen or Queen Mandibular Pheromones (QMP), some but not all workers will develop ovaries and become "queen-like". Using this inducible developmental change, we aimed to determine if diet and/or reproductive development impacts the gut microbiota of honey bee workers.

[Source](https://doi.org/10.1186/s42523-024-00350-3)

10.1186/s42523-024-00350-3

---

## Biological and experimental factors that define the effectiveness of microbial inoculation on plant traits: a meta-analysis.
 07.11.2024, _H Azarbad, RR Junker_


Bacterial and fungal microbiomes associated with plants can significantly affect the host's phenotype. Inoculating plants with one or multiple bacterial and fungal species can affect specific plant traits, which is exploited in attempts to increase plant performance and stress tolerance by microbiome engineering. Currently, we lack a comprehensive synthesis on the generality of these effects related to different biological (e.g. plant models, plant traits, and microbial taxa) and experimental factors. In a meta-analysis, we showed that the plant trait under consideration and the microbial taxa used to inoculate plants significantly influenced the strength of the effect size. In a methodological context, experiments under sterilized conditions and short-term periods resulted in larger positive effects on plant traits than those of unsterilized and long-term experiments. We recommend that future studies should not only consider (short-term) laboratory experiments with sterilized plants and single inoculants but also and more often (long-term) field or greenhouse experiments with naturally occurring microbial communities associated with the plants and inoculated consortia including both bacteria and fungi.

[Source](https://doi.org/10.1093/ismeco/ycae122)

10.1093/ismeco/ycae122

---

## Can microbiome adaptations protect crops from pests and climate change?
 06.11.2024, _C Beans_



[Source](https://doi.org/10.1073/pnas.2421369121)

10.1073/pnas.2421369121

---

## Ecological roles of nano-picoplankton in stratified waters of an embayment in the southern Benguela.
 07.11.2024, _NR Dames, E Rocke, G Pitcher, E Rybicki, M Pfaff, CL Moloney_


Nano-picoplankton are the dominant primary producers during the post-upwelling period in St Helena Bay, South Africa. Their dynamics on short time scales are not well understood and neither are the community composition, structure, and potential functionality of the surrounding microbiome. Samples were collected over five consecutive days in March 2018 from three depths (1 m, 25 m, 50 m) at a single sampling station in St Helena Bay. There was clear depth-differentiation between the surface and depth in both diversity and function throughout the sampling period for the archaea, bacteria and eukaryotes. Daily difference in eukaryote diversity, was more pronounced at 1 m and 25 m with increased abundances of Syndiniales and Bacillariophyta. Surface waters were dominated by photosynthetic and photoheterotrophic microorganisms, while samples at depth were linked to nitrogen cycling processes, with high abundances of nitrifiers and denitrifiers. Strong depth gradients found in the nutrient transporters for ammonia were good indicators of measured uptake rates. This study showed that nano-picoplankton dynamics were driven by light availability, nutrient concentrations, carbon biomass and oxygenation. The nano-picoplankton help sustain ecosystem functioning in St Helena Bay through their ecological roles, which emphasizes the need to monitor this size fraction of the plankton.

[Source](https://doi.org/10.1093/femsle/fnae094)

10.1093/femsle/fnae094

---

## Metagenomic insights into inhibition of soil microbial carbon metabolism by phosphorus limitation during vegetation succession.
 07.11.2024, _H Wang, H Wang, TW Crowther, K Isobe, PB Reich, R Tateno, W Shi_


There is growing awareness of the need for regenerative practices in the fight against biodiversity loss and climate change. Yet, we lack a mechanistic understanding of how microbial community composition and functioning are likely to change alongside transition from high-density tillage to large-scale vegetation restoration. Here, we investigated the functional dynamics of microbial communities following a complete vegetation successional chronosequence in a subtropical zone, Southwestern China, using shotgun metagenomics approaches. The contents of total soil phosphorus (P), available P, litter P, and microbial biomass P decreased significantly during vegetation succession, indicating that P is the most critical limiting nutrient. The abundance of genes related to P-uptake and transport, inorganic P-solubilization, organic P-mineralization, and P-starvation response regulation significantly increased with successional time, indicating an increased microbial "mining" for P under P limitation. Multi-analysis demonstrated microbial P limitation strongly inhibits carbon (C) catabolism potential, resulting in a significant decrease in carbohydrate-active enzyme family gene abundances. Nevertheless, over successional time, microorganisms increased investment in genes involved in degradation-resistant compounds (lignin and its aromatic compounds) to acquire P resources in the litter. Our study provides functional gene-level insights into how P limitation during vegetation succession in subtropical regions inhibits soil microbial C metabolic processes, thereby advancing our understanding of belowground C cycling and microbial metabolic feedback during forest restoration.

[Source](https://doi.org/10.1093/ismeco/ycae128)

10.1093/ismeco/ycae128

---

## Cross-comparison of gut metagenomic profiling strategies.
 06.11.2024, _G Gulyás, B Kakuk, Á Dörmő, T Járay, I Prazsák, Z Csabai, MM Henkrich, Z Boldogkői, D Tombácz_


_Metagenomics, Animals, Gastrointestinal Microbiome, Dogs, Feces, Computational Biology, Software, Sequence Analysis, DNA, High-Throughput Nucleotide Sequencing, Metagenome, Gene Library_
The rapid advancements in sequencing technologies and bioinformatics have enabled metagenomic research of complex microbial systems, but reliable results depend on consistent laboratory and bioinformatics approaches. Current efforts to identify best practices often focus on optimizing specific steps, making it challenging to understand the influence of each stage on microbial population analysis and compare data across studies. This study evaluated DNA extraction, library construction methodologies, sequencing platforms, and computational approaches using a dog stool sample, two synthetic microbial community mixtures, and various sequencing data sources. Our work, the most comprehensive evaluation of metagenomic methods to date. We developed a software tool, termed minitax, which provides consistent results across the range of platforms and methodologies. Our findings showed that the Zymo Research Quick-DNA HMW MagBead Kit, Illumina DNA Prep library preparation method, and the minitax bioinformatics tool were the most effective for high-quality microbial diversity analysis. However, the effectiveness of pipelines or method combinations is sample-specific, making it difficult to identify a universally optimal approach. Therefore, employing multiple approaches is crucial for obtaining reliable outcomes in microbial systems.

[Source](https://doi.org/10.1038/s42003-024-07158-6)

10.1038/s42003-024-07158-6

---

