## The early life education of the immune system: Moms, microbes and (missed) opportunities.
 12.10.2020, _N Jain_


The early life immune system is characterized by unique developmental milestones. Functionally diverse immune cells arise from distinct waves of hematopoietic stem cells, a phenomenon referred to as 'layered' immunity. This stratified development of immune cells extends to lineages of both innate and adaptive cells. The defined time window for the development of these immune cells lends itself to the influence of specific exposures typical of the early life period. The perinatal immune system develops in a relatively sterile fetal environment but emerges into one filled with a multitude of antigenic encounters. A major burden of this comes in the form of the microbiota that is being newly established at mucosal surfaces of the newborn. Accumulating evidence suggests that early life microbial exposures, including those arising in utero, can imprint long-lasting changes in the offspring's immune system and determine disease risk throughout life. In this review, I highlight unique features of early life immunity and explore the role of intestinal bacteria in educating the developing immune system.

[Source](https://doi.org/10.1080/19490976.2020.1824564)

10.1080/19490976.2020.1824564

---

## Delayed Establishment of Gut Microbiota in Infants Delivered by Cesarean Section.
 06.10.2020, _G Kim, J Bae, MJ Kim, H Kwon, G Park, SJ Kim, YH Choe, J Kim, SH Park, BH Choe, H Shin, B Kang_


The maternal vaginal microbiome is an important source for infant gut microbiome development. However, infants delivered by Cesarean section (CS) do not contact the maternal vaginal microbiome and this delivery method may perturb the early establishment and development of the gut microbiome. The aim of this study was to investigate the early gut microbiota of Korean newborns receiving the same postpartum care services for two weeks after birth by delivery mode using fecal samples collected at days 3, 7, and 14. Early gut microbiota development patterns were examined using 16S rRNA gene-based sequencing from 132 infants either born vaginally (VD, n = 64) or via Cesarean section (CS, n = 68). VD-born neonates showed increased alpha diversity in infant fecal samples collated at days 7 and 14 compared to those from day 3, while those of CS infants did not differ (p &lt; 0.015). Bacterial structures of infants from both groups separated at day 7 (p &lt; 0.001) and day 14 (p &lt; 0.01). The bacterial structure of VD infants gradually changed over time (day 3 vs. day 7, p &lt; 0.012; day 3 vs. day 14, p &lt; 0.001). Day 14 samples of CS infants differed from day 3 and 7 samples (day 3 vs. day 14, p &lt; 0.001). VD infant relative abundance of Bifidobacterium (days 7, 14), Bacteroides (days 7, 14), and Lachnospiraceae (day 7) significantly increased compared to CS infants, with a lower abundance of Enterobacteriaceae (found in all periods of the CS group) (LDA &gt; 3.0). Relative abundances of Bifidobacterium, Lactobacillus, and Staphylococcus were significantly increased in both VD and CS groups at day 14 (LDA &gt; 3.0). Predicted functional analysis showed that VD infants had overrepresented starch/sucrose, amino acid and nucleotide metabolism in gut microbiota with depleted lipopolysaccharide biosynthesis until day 14 compared to CS infants. This study confirmed that delivery mode is the major determinant of neonatal intestinal microbiome establishment and provides a profile of microbiota perturbations in CS infants. Our findings provide preliminary insight for establishing recovery methods to supply the specific microbes missing in CS infants.

[Source](https://doi.org/10.3389/fmicb.2020.02099)

10.3389/fmicb.2020.02099

---

## Proteins produced by Streptococcus species in the lower respiratory tract can modify antiviral responses against influenza virus in respiratory epithelial cells.
 10.10.2020, _K Nishioka, M Kyo, T Nakaya, N Shime_


Seasonal influenza spreads during winter in temperate countries. Primary viral pneumoniae resulting from aggravation triggers acute respiratory distress syndrome, which is a serious respiratory disorder. We have identified a unique pattern of lung microbiota in patients with the syndrome. In this study, we hypothesized that the unique microbiota was also associated with primary influenza viral pneumoniae. Bacterial culture supernatants of Streptococcus oralis and Streptococcus mitis detected from the patients significantly increased viral replication (maximum 10-fold increase) in lung epithelial cells. Our results suggest that the lung environment microbiota is significantly involved in viral replication.

[Source](https://doi.org/10.1016/j.micinf.2020.09.010)

10.1016/j.micinf.2020.09.010

---

## Vaginal host immune-microbiome interactions in a cohort of primarily African-American women who ultimately underwent spontaneous preterm birth or delivered at term

[Source](https://doi.org/10.1016/j.cyto.2020.155316)

---

## Isolation and Characterization of Nitrate-Reducing Bacteria as Potential Probiotics for Oral and Systemic Health.
 13.10.2020, _BT Rosier, EM Moya-Gonzalvez, P Corell-Escuin, A Mira_


Recent evidence indicates that the reduction of salivary nitrate by oral bacteria can contribute to prevent oral diseases, as well as increase systemic nitric oxide levels that can improve conditions such as hypertension and diabetes. The objective of the current manuscript was to isolate nitrate-reducing bacteria from the oral cavity of healthy donors and test their in vitro probiotic potential to increase the nitrate-reduction capacity (NRC) of oral communities. Sixty-two isolates were obtained from five different donors of which 53 were confirmed to be nitrate-reducers. Ten isolates were selected based on high NRC as well as high growth rates and low acidogenicity, all being Rothia species. The genomes of these ten isolates confirmed the presence of nitrate- and nitrite reductase genes, as well as lactate utilization genes, and the absence of antimicrobial resistance, mobile genetic elements and virulence genes. The pH at which most nitrate was reduced differed between strains. However, acidic pH 6 always stimulated the reduction of nitrite compared to neutral pH 7 or slightly alkaline pH 7.5 (p &lt; 0.01). We tested the effect of six out of 10 isolates on in vitro oral biofilm development in the presence or absence of 6.5 mM nitrate. The integration of the isolates into in vitro communities was confirmed by Illumina sequencing. The NRC of the bacterial communities increased when adding the isolates compared to controls without isolates (p &lt; 0.05). When adding nitrate (prebiotic treatment) or isolates in combination with nitrate (symbiotic treatment), a smaller decrease in pH derived from sugar metabolism was observed (p &lt; 0.05), which for some symbiotic combinations appeared to be due to lactate consumption. Interestingly, there was a strong correlation between the NRC of oral communities and ammonia production even in the absence of nitrate (R = 0.814, p &lt; 0.01), which indicates that bacteria involved in these processes are related. As observed in our study, individuals differ in their NRC. Thus, some may have direct benefits from nitrate as a prebiotic as their microbiota naturally reduces significant amounts, while others may benefit more from a symbiotic combination (nitrate + nitrate-reducing probiotic). Future clinical studies should test the effects of these treatments on oral and systemic health.

[Source](https://doi.org/10.3389/fmicb.2020.555465)

10.3389/fmicb.2020.555465

---

## Translocation of Viable Gut Microbiota to Mesenteric Adipose Drives Formation of Creeping Fat in Humans.
 01.10.2020, _CWY Ha, A Martin, GD Sepich-Poore, B Shi, Y Wang, K Gouin, G Humphrey, K Sanders, Y Ratnayake, KSL Chan, G Hendrick, JR Caldera, C Arias, JE Moskowitz, SJ Ho Sui, S Yang, D Underhill, MJ Brady, S Knott, K Kaihara, MJ Steinbaugh, H Li, DPB McGovern, R Knight, P Fleshner, S Devkota_


A mysterious feature of Crohn's disease (CD) is the extra-intestinal manifestation of "creeping fat" (CrF), defined as expansion of mesenteric adipose tissue around the inflamed and fibrotic intestine. In the current study, we explore whether microbial translocation in CD serves as a central cue for CrF development. We discovered a subset of mucosal-associated gut bacteria that consistently translocated and remained viable in CrF in CD ileal surgical resections, and identified Clostridium innocuum as a signature of this consortium with strain variation between mucosal and adipose isolates, suggesting preference for lipid-rich environments. Single-cell RNA sequencing characterized CrF as both pro-fibrotic and pro-adipogenic with a rich milieu of activated immune cells responding to microbial stimuli, which we confirm in gnotobiotic mice colonized with C. innocuum. Ex vivo validation of expression patterns suggests C. innocuum stimulates tissue remodeling via M2 macrophages, leading to an adipose tissue barrier that serves to prevent systemic dissemination of bacteria.

[Source](https://doi.org/10.1016/j.cell.2020.09.009)

10.1016/j.cell.2020.09.009

---

## Gastrointestinal Surgery for Inflammatory Bowel Disease Persistently Lowers Microbiome and Metabolome Diversity.
 07.10.2020, _X Fang, Y Vázquez-Baeza, E Elijah, F Vargas, G Ackermann, G Humphrey, R Lau, KC Weldon, JG Sanders, M Panitchpakdi, C Carpenter, AK Jarmusch, J Neill, A Miralles, P Dulai, S Singh, M Tsai, AD Swafford, L Smarr, DL Boyle, BO Palsson, JT Chang, PC Dorrestein, WJ Sandborn, R Knight, BS Boland_


Many studies have investigated the role of the microbiome in inflammatory bowel disease (IBD), but few have focused on surgery specifically or its consequences on the metabolome that may differ by surgery type and require longitudinal sampling. Our objective was to characterize and contrast microbiome and metabolome changes after different surgeries for IBD, including ileocolonic resection and colectomy.

[Source](https://doi.org/10.1093/ibd/izaa262)

10.1093/ibd/izaa262

---

## α-Defensins Promote Bacteroides Colonization on Mucosal Reservoir to Prevent Antibiotic-Induced Dysbiosis.
 06.10.2020, _J Ou, S Liang, XK Guo, X Hu_


In addition to their established functions in host defense, accumulating evidence has suggested an emerging role for antimicrobial proteins (AMPs) in shaping commensal microbiota. However, the role of α-defensins, the most abundant AMPs of intestine, in regulating microbial ecology remains inconclusive. Here, we report that α-defensins promote commensal Bacteroides colonization by enhancing bacterial adhesion to the mucosal reservoir. Experiments utilizing mice deficient in matrix metalloproteinase 7 (MMP7), the α-defensin-activating enzyme, with rigorous littermate controls showed that α-defensin deficiency did not significantly influence steady-state intestinal microbiota. In contrast, α-defensins are essential for replenishment of commensal Bacteroides from the mucosal reservoir following antibiotics-induced dysbiosis, shown by markedly compromised recovery of Bacteroides in Mmp7-/- animals. Mechanistically, α-defensins promote Bacteroides colonization on epithelial surfaces in vivo and adhesion to epithelial cells in vitro. Moreover, α-defensins unexpectedly does not show any microbicidal activities against Bacteroides. Together, we propose that α-defensins promote commensal bacterial colonization and recovery to maintain microbial diversity upon environmental challenges.

[Source](https://doi.org/10.3389/fimmu.2020.02065)

10.3389/fimmu.2020.02065

---

## In situ reprogramming of gut bacteria by oral delivery.
 08.10.2020, _BB Hsu, IN Plant, L Lyon, FM Anastassacos, JC Way, PA Silver_


Abundant links between the gut microbiota and human health indicate that modification of bacterial function could be a powerful therapeutic strategy. The inaccessibility of the gut and inter-connections between gut bacteria and the host make it difficult to precisely target bacterial functions without disrupting the microbiota and/or host physiology. Herein we describe a multidisciplinary approach to modulate the expression of a specific bacterial gene within the gut by oral administration. We demonstrate that an engineered temperate phage λ expressing a programmable dCas9 represses a targeted E. coli gene in the mammalian gut. To facilitate phage administration while minimizing disruption to host processes, we develop an aqueous-based encapsulation formulation with a microbiota-based release mechanism and show that it facilitates oral delivery of phage in vivo. Finally we combine these technologies and show that bacterial gene expression in the mammalian gut can be precisely modified in situ with a single oral dose.

[Source](https://doi.org/10.1038/s41467-020-18614-2)

10.1038/s41467-020-18614-2

---

## The intestinal microbial metabolite desaminotyrosine is an anti-inflammatory molecule that modulates local and systemic immune homeostasis.
 13.10.2020, _Y Wei, J Gao, Y Kou, M Liu, L Meng, X Zheng, S Xu, M Liang, H Sun, Z Liu, Y Wang_


It is considered that intestinal barrier dysfunction and systemic endotoxemia drive obesity and its related complications. However, what causes barrier dysfunction remains to be elucidated. Here, we showed that the gut microbiota from high-fat diet (HFD)-fed mice had impaired ability to degrade dietary flavonoids, and in correspondence, the microbial-derived flavonoid metabolite desaminotyrosine (DAT) was reduced. Supplementation of DAT in the drinking water was able to counter the HFD-induced body fat mass accumulation and body weight increment. This is correlated with the role of DAT in maintaining mucosal immune homeostasis to protect barrier integrity. DAT could attenuate dextran sodium sulfate (DSS)-induced mucosal inflammation in a type I interferon signal-dependent manner. Furthermore, intraperitoneal injection of DAT-protected mice from bacterial endotoxin-induced septic shock. Together, we identified DAT as a gut microbiota-derived anti-inflammatory metabolite that functions to modulate local and systemic immune homeostasis. Our data support the notion of dysbiosis being an important driving force of mucosal barrier dysfunction and systemic metabolic complications.

[Source](https://doi.org/10.1096/fj.201902900RR)

10.1096/fj.201902900RR

---

## Modulation of experimental autoimmune encephalomyelitis through colonisation of the gut with Escherichia coli.
 13.10.2020, _JE Libbey, JMS Sanchez, BA Fleming, DJ Doty, AB DePaula-Silva, MA Mulvey, RS Fujinami_


Multiple sclerosis (MS) is a neuro-inflammatory autoimmune disease of the central nervous system (CNS) that affects young adults. It is characterised by the development of demyelinating lesions and inflammation within the CNS. Although the causes of MS are still elusive, recent work using patient samples and experimental animal models has demonstrated a strong relationship between the gut microbiota and its contribution to CNS inflammation and MS. While there is no cure for MS, alteration of the gut microbiota composition through the use of probiotics is a very promising treatment. However, while most recent works have focused on the use of probiotics to modify pre-existing disease, little is known about its role in protecting from the establishment of MS. In this study, we determined whether colonisation with the probiotic bacterium Escherichia coli strain Nissle 1917 (EcN) could be used as a prophylactic strategy to prevent or alter the development of experimental autoimmune encephalomyelitis (EAE), a preclinical model of MS. We found that double gavage (two doses) of EcN before induction of EAE delayed disease onset and decreased disease severity. We also found that EcN-treated mice had decreased amounts of perivascular cuffing, CD4+ T cell infiltration into the CNS, together with significantly decreased absolute numbers of Th1 cells, and reduced activation of microglia. Although further studies are necessary to comprehend the exact protective mechanisms induced, our study supports a promising use of EcN as a probiotic for the prevention of MS.

[Source](https://doi.org/10.3920/BM2020.0012)

10.3920/BM2020.0012

---

## Molecular patterns from a human gut-derived Lactobacillus strain suppress pathogenic infiltration of leukocytes into the central nervous system.
 09.10.2020, _JMS Sanchez, DJ Doty, AB DePaula-Silva, DG Brown, R Bell, KA Klag, A Truong, JE Libbey, JL Round, RS Fujinami_


Multiple sclerosis (MS) is an inflammatory demyelinating disease that affects 2.5 million people worldwide. Growing evidence suggests that perturbation of the gut microbiota, the dense collection of microorganisms that colonize the gastrointestinal tract, plays a functional role in MS. Indeed, specific gut-resident bacteria are altered in patients with MS compared to healthy individuals, and colonization of gnotobiotic mice with MS-associated microbiota exacerbates preclinical models of MS. However, defining the molecular mechanisms by which gut commensals can remotely affect the neuroinflammatory process remains a critical gap in the field.

[Source](https://doi.org/10.1186/s12974-020-01959-2)

10.1186/s12974-020-01959-2

---

## Soil-microorganism-mediated invasional meltdown in plants.
 06.10.2020, _Z Zhang, Y Liu, C Brunel, M van Kleunen_


While most alien species fail to establish, some invade native communities and become widespread. Our understanding of invasion success is derived mainly from pairwise interactions between aliens and natives, while interactions among more than two species remain largely unexplored. Here, we experimentally tested whether and how a third plant species, either native or alien, affected the competitive outcomes between alien and native plants through its soil legacy. We first conditioned soil with one of ten species (six natives and four aliens) or without plants. We then grew on these 11 soils five aliens and five natives without competition, or with intra- or interspecific competition. We found that aliens were not more competitive than natives when grown on soil conditioned by other natives or on non-conditioned soil. However, aliens were more competitive than natives on soil conditioned by other aliens (that is, invasional meltdown). Soil conditioning did not change competitive outcomes by affecting the strength of competition between later plants. Instead, soil conditioned by aliens pushed competitive outcomes towards later aliens by affecting the growth of aliens less negatively than that of natives. Microbiome analysis verified this finding, as we showed that the soil-legacy effects of a species on later species were less negative when their fungal endophyte communities were less similar, and that fungal endophyte communities were less similar between two aliens than between aliens and natives. Our study reveals invasional meltdown in multispecies communities and identifies soil microorganisms as a driver of the invasion success of alien plants.

[Source](https://doi.org/10.1038/s41559-020-01311-0)

10.1038/s41559-020-01311-0

---

## Microbiome Management by Biological and Chemical Treatments in Maize Is Linked to Plant Health.
 05.10.2020, _P Kusstatscher, WA Wicaksono, DP Thenappan, E Adam, H Müller, G Berg_


The targeted application of plant growth-promoting rhizobacteria (PGPR) provides the key for a future sustainable agriculture with reduced pesticide application. PGPR interaction with the indigenous microbiota is poorly understood, but essential to develop reliable applications. Therefore, Stenotrophomonas rhizophila SPA-P69 was applied as a seed coating and in combination with a fungicide based on the active ingredients fludioxonil, metalaxyl-M, captan and ziram. The plant performances and rhizosphere compositions of treated and non-treated maize plants of two field trials were analyzed. Plant health was significantly increased by treatment; however, overall corn yield was not changed. By applying high-throughput amplicon sequencing of the 16S rRNA and the ITS genes, the bacterial and fungal changes in the rhizosphere due to different treatments were determined. Despite the fact that treatments had a significant impact on the rhizosphere microbiota (9-12%), the field site was identified as the main driver (27-37%). The soil microbiota composition from each site was significantly different, which explains the site-specific effects. In this study we were able to show the first indications how PGPR treatments increase plant health via microbiome shifts in a site-specific manner. This way, first steps towards a detailed understanding of PGPRs and developments of consistently efficient applications in diverse environments are made.

[Source](https://doi.org/10.3390/microorganisms8101506)

10.3390/microorganisms8101506

---

## Disruption of Firmicutes and Actinobacteria abundance in tomato rhizosphere causes the incidence of bacterial wilt disease.
 08.10.2020, _SM Lee, HG Kong, GC Song, CM Ryu_


Enrichment of protective microbiota in the rhizosphere facilitates disease suppression. However, how the disruption of protective rhizobacteria affects disease suppression is largely unknown. Here, we analyzed the rhizosphere microbial community of a healthy and diseased tomato plant grown &lt;30-cm apart in a greenhouse at three different locations in South Korea. The abundance of Gram-positive Actinobacteria and Firmicutes phyla was lower in diseased rhizosphere soil (DRS) than in healthy rhizosphere soil (HRS) without changes in the causative Ralstonia solanacearum population. Artificial disruption of Gram-positive bacteria in HRS using 500-μg/mL vancomycin increased bacterial wilt occurrence in tomato. To identify HRS-specific and plant-protective Gram-positive bacteria species, Brevibacterium frigoritolerans HRS1, Bacillus niacini HRS2, Solibacillus silvestris HRS3, and Bacillus luciferensis HRS4 were selected from among 326 heat-stable culturable bacteria isolates. These four strains did not directly antagonize R. solanacearum but activated plant immunity. A synthetic community comprising these four strains displayed greater immune activation against R. solanacearum and extended plant protection by 4 more days in comparison with each individual strain. Overall, our results demonstrate for the first time that dysbiosis of the protective Gram-positive bacterial community in DRS promotes the incidence of disease.

[Source](https://doi.org/10.1038/s41396-020-00785-x)

10.1038/s41396-020-00785-x

---

## Keystone Microbiomes Revealed by 14 Years of Field Restoration of the Degraded Agricultural Soil Under Distinct Vegetation Scenarios.
 06.10.2020, _Z Zhang, X Han, J Yan, W Zou, E Wang, X Lu, X Chen_


Agricultural intensification accelerates the degradation of cropland, and restoration has been managed by changing its vegetation. However, the keystone microbiome that drives the decomposition of plant-associated organic matter in the restoration is poorly understood. In this study, we established a 14-year field restoration experiment on a degraded cropland with four treatments: (1) bare land soil without biomass input (BL), (2) maize cropland (CL) without fertilization and biomass input, (3) natural grassland (GL), and (4) alfalfa cropland (AL) with biomass left in the fields. The activity of total soil microbiome was assessed by community-level physiological profiling (CLPP) with Biolog EcoPlates analysis, and keystone microbiome was identified as phylotypes showing statistically significant increase in the restored soils (GL and AL) relative to the degraded BL soil. The results showed that GL and AL treatments improved soil fertility as indicated by significant increase in soil organic carbon, total nitrogen, and available phosphorus when compared to BL treatment. The significant difference was not observed between CL and BL treatments except for pH and available phosphorus, indicating that the input and microbial decomposition of plant-associated organic matter were the key for restoration of soil fertility. Similar results were obtained for soil microbial activities of carbon utilization efficiency via CLPP analysis, and real-time quantitative polymerase chain reaction of 16S rRNA genes further revealed significantly higher abundance of total soil microbial community in GL and AL soils than in BL and CL. High-throughput sequencing of total 16S rRNA genes revealed the Bacteroidetes as the only keystone taxa at phylum level, and 106 and 120 genera were keystone phylotypes. Compared with BL and CL, the genera that increase significantly in GL and AL are called keystone genera of ecological restoration. The dominant keystone genera included Reyranella, Mesorhizobium, Devosia, Haliangium, Nocardioides, and Pseudonocardia. Significantly higher abundance of Bacillus genus in BL soil implied it might serve as an indicator of agricultural land degradation. Statistical analysis showed that soil organic carbon and pH were significantly correlated with the input of plant-associated organic matters and dynamic changes of keystone taxa. These results suggest that the vegetation of natural grass (GL) and alfalfa plant (AL) and subsequent decomposition of plant-associated materials could serve as effective strategies for restoration of the degraded cropland by stimulating the keystone microbiomes and improving their physiological metabolisms of carbon utilization efficiency.

[Source](https://doi.org/10.3389/fmicb.2020.01915)

10.3389/fmicb.2020.01915

---

## Dynamic Changes in the Microbiome of Rice During Shoot and Root Growth Derived From Seeds.
 06.10.2020, _M Wang, AW Eyre, MR Thon, Y Oh, RA Dean_


Microbes form close associations with host plants including rice as both surface (epiphytes) and internal (endophytes) inhabitants. Yet despite rice being one of the most important cereal crops agriculturally and economically, knowledge of its microbiome, particularly core inhabitants and any functional properties bestowed is limited. In this study, the microbiome in rice seedlings derived directly from seeds was identified, characterized and compared to the microbiome of the seed. Rice seeds were sourced from two different locations in Arkansas, USA of two different rice genotypes (Katy, M202) from two different harvest years (2013, 2014). Seeds were planted in sterile media and bacterial as well as fungal communities were identified through 16S and ITS sequencing, respectively, for four seedling compartments (root surface, root endosphere, shoot surface, shoot endosphere). Overall, 966 bacterial and 280 fungal ASVs were found in seedlings. Greater abundance and diversity were detected for the microbiome associated with roots compared to shoots and with more epiphytes than endophytes. The seedling compartments were the driving factor for microbial community composition rather than other factors such as rice genotype, location and harvest year. Comparison with datasets from seeds revealed that 91 (out of 296) bacterial and 11 (out of 341) fungal ASVs were shared with seedlings with the majority being retained within root tissues. Core bacterial and fungal microbiome shared across seedling samples were identified. Core bacteria genera identified in this study such as Rhizobium, Pantoea, Sphingomonas, and Paenibacillus have been reported as plant growth promoting bacteria while core fungi such as Pleosporales, Alternaria and Occultifur have potential as biocontrol agents.

[Source](https://doi.org/10.3389/fmicb.2020.559728)

10.3389/fmicb.2020.559728

---

## Root-Secreted Coumarins and the Microbiota Interact to Improve Iron Nutrition in Arabidopsis.
 07.10.2020, _CJ Harbort, M Hashimoto, H Inoue, Y Niu, R Guan, AD Rombolà, S Kopriva, MJEEE Voges, ES Sattely, R Garrido-Oter, P Schulze-Lefert_


Plants benefit from associations with a diverse community of root-colonizing microbes. Deciphering the mechanisms underpinning these beneficial services are of interest for improving plant productivity. We report a plant-beneficial interaction between Arabidopsis thaliana and the root microbiota under iron deprivation that is dependent on the secretion of plant-derived coumarins. Disrupting this pathway alters the microbiota and impairs plant growth in iron-limiting soil. Furthermore, the microbiota improves iron-limiting plant performance via a mechanism dependent on plant iron import and secretion of the coumarin fraxetin. This beneficial trait is strain specific yet functionally redundant across phylogenetic lineages of the microbiota. Transcriptomic and elemental analyses revealed that this interaction between commensals and coumarins promotes growth by relieving iron starvation. These results show that coumarins improve plant performance by eliciting microbe-assisted iron nutrition. We propose that the bacterial root microbiota, stimulated by secreted coumarins, is an integral mediator of plant adaptation to iron-limiting soils.

[Source](https://doi.org/10.1016/j.chom.2020.09.006)

10.1016/j.chom.2020.09.006

---

## Pinpointing secondary metabolites that shape the composition and function of the plant microbiome.
 30.09.2020, _RP Jacoby, A Koprivova, S Kopriva_


One of the major questions in contemporary plant science involves determining the functional mechanisms that plants use to shape their microbiome. Plants produce a plethora of chemically diverse secondary metabolites, many of which exert bioactive functions on microorganisms. Several recent publications have unequivocally shown that plant secondary metabolites affect microbiome composition and function. These studies have pinpointed that the microbiome can be influenced by a diverse set of molecules, including: coumarins, glucosinolates, benzoxazinoids, camalexin and triterpenes. In this review, we summarise the role of secondary metabolites in shaping the plant microbiome, highlighting recent literature. A body of knowledge is now emerging that links specific plant metabolites with distinct microbial responses, mediated via defined biochemical mechanisms. There is significant potential to boost agricultural sustainability via the targeted enhancement of beneficial microbial traits, and here we argue that the recently discovered mechanistic links between root chemistry and microbiome composition could provide a new set of tools for rationally manipulating the plant microbiome.

[Source](https://doi.org/10.1093/jxb/eraa424)

10.1093/jxb/eraa424

---

## Microbiomes of fish, sediment and seagrass suggest connectivity of coral reef microbial populations.
 03.10.2020, _R León-Zayas, M McCargar, JA Drew, JF Biddle_


The benthic environments of coral reefs are heavily shaped by physiochemical factors, but also the ecological interactions of the animals and plants in the reef ecosystem. Microbial populations may be shared within the ecosystem of sediments, seagrasses and reef fish. In this study, we hypothesize that coral reef and seagrass environments share members of the microbial community that are rare in some habitats and enriched in others, and that animals may integrate this connectivity. We investigated the potential connectivity between the microbiomes of sediments, seagrass blades and roots (Syringodium isoetifolium), and a seagrass-specialist parrotfish (C. spinidens) guts in reef areas of Fiji. We contrasted these with sediment samples from the Florida Keys, gut samples from surgeonfish (A. nigricauda, Acanthurinae sp. unknown, C. striatus), and ocean water microbiomes from the Atlantic, Pacific and Indian Oceans to test the robustness of our characterizations of microbiome environments. In general, water, sediment and fish gut samples were all distinct microbiomes. Sediment microbiomes were mostly similar between Fiji and Florida, but also showed some regional similarities. In Fiji, we show connectivity of a shared microbiome between seagrass, fish and sediments. Additionally, we identified an environmental reservoir of a surgeonfish symbiont, Epulopiscium. The connection of these ecosystem components suggests that the total microbiome of these environments may vary as their animal inhabitants shift in a changing ocean.

[Source](https://peerj.com/articles/10026/)

10.7717/peerj.10026

---

## Size Shapes the Active Microbiome of Methanogenic Granules, Corroborating a Biofilm Life Cycle.
 06.10.2020, _AC Trego, S O'Sullivan, C Quince, S Mills, UZ Ijaz, G Collins_


Methanogenic archaea are key players in cycling organic matter in nature but also in engineered waste treatment systems, where they generate methane, which can be used as a renewable energy source. In such systems in the built environment, complex methanogenic consortia are known to aggregate into highly organized, spherical granular biofilms comprising the interdependent microbial trophic groups mediating the successive stages of the anaerobic digestion (AD) process. This study separated methanogenic granules into a range of discrete size fractions, hypothesizing different biofilm growth stages, and separately supplied each with specific substrates to stimulate the activity of key AD trophic groups, including syntrophic acid oxidizers and methanogens. Rates of specific methanogenic activity were measured, and amplicon sequencing of 16S rRNA gene transcripts was used to resolve phylotranscriptomes across the series of size fractions. Increased rates of methane production were observed in each of the size fractions when hydrogen was supplied as the substrate compared with those of volatile fatty acids (acetate, propionate, and butyrate). This was connected to a shift toward hydrogenotrophic methanogenesis dominated by Methanobacterium and Methanolinea Interestingly, the specific active microbiomes measured in this way indicated that size was significantly more important than substrate in driving the structure of the active community in granules. Multivariate integration studywise discriminant analysis identified 56 genera shaping changes in the active community across both substrate and size. Half of those were found to be upregulated in the medium-sized granules, which were also the most active and potentially of the most important size, or life stage, for precision management of AD systems.IMPORTANCE Biological wastewater conversion processes collectively constitute one of the single biggest worldwide applications of microbial communities. There is an obvious requirement, therefore, to study the microbial systems central to the success of such technologies. Methanogenic granules, in particular, are architecturally fascinating biofilms that facilitate highly organized cooperation within the metabolic network of the anaerobic digestion (AD) process and, thus, are especially intriguing model systems for microbial ecology. This study, in a way not previously reported, provoked syntrophic and methanogenic activity and the structure of the microbial community, using specific substrates targeting the key trophic groups in AD. Unexpectedly, granule size more strongly than substrate shaped the active portion of the microbial community. Importantly, the findings suggest the size, or age, of granules inherently shapes the active microbiome linked to a life cycle. This provides exciting insights into the function of, and the potential for additional modeling of biofilm development in, methanogenic granules.

[Source](https://msystems.asm.org/content/5/5/e00323-20)

10.1128/mSystems.00323-20

---

## Self-regulating microbiome networks ensure functional resilience of biofilms in sand biofilters during manganese load fluctuations.
 14.10.2020, _X Zhao, Y Yang, K Feng, X Wang, B Liu, G Xie, D Xing_


Sand biofilters (SBFs) are commonly used to remove manganese (Mn(II)) from drinking water. Mn(II) load variation frequently occurs in SBFs due to fluctuating influent Mn(II) concentrations or flow rates. Therefore, it is important to understand the responses of microbial biofilms in SBFs to environmental disturbances and how they affect Mn(II) oxidation efficiency. Here, the responses of microbial ecological networks and Mn(II) removal in SBFs to increasing Mn(II) load were investigated. The Mn(II) removal efficiency in two SBFs remained at 99.8% despite an increase in influent Mn(II) from 2 mg/L to 4 mg/L, but significantly deteriorated (50.1-58.5%) upon increasing the filtration rate. A canonical correlation analysis of the microbial communities indicated that the local Mn(II) concentration and biofilter depth impacted community compositions of biofilms. The dominant species within the biofilms exhibited clear stratification, with simple associations in the upper layer of the SBFs and more complex interspecies interactions in the bottom layers. Putative manganese-oxidizing bacteria Hyphomicrobium and Pedomicrobium dominated the microbiomes in different layers of SBFs, and changed relatively little in abundance when Mn(II) and filtration rate increased. The community networks showed that biofilm microbiomes in SBFs were resilient to the disturbance of Mn(II) load, primarily via regulating microbial interactions. High manganese loads negatively affected the functional modules for Mn(II) removal. Furthermore, the relatively rare species Candidatus Entotheonella palauensis was identified as a module hub, implying taxa with low abundances can have important roles in ecosystem function. These results shed new light on the ecological rules guiding responses of microbiomes in sand biofilters to environmental stress.

[Source](https://doi.org/10.1016/j.watres.2020.116473)

10.1016/j.watres.2020.116473

---

## Groundwater Microbial Communities in Times of Climate Change.
 07.10.2020, _A Retter, C Karwautz, C Griebler_


Climate change has a massive impact on the global water cycle. Subsurface ecosystems, the earth largest reservoir of liquid freshwater, currently experience a significant increase in temperature and serious consequences from extreme hydrological events. Extended droughts as well as heavy rains and floods have measurable impacts on groundwater quality and availability. In addition, the growing water demand puts increasing pressure on the already vulnerable groundwater ecosystems. Global change induces undesired dynamics in the typically nutrient and energy poor aquifers that are home to a diverse and specialized microbiome and fauna. Current and future changes in subsurface environmental conditions, without doubt, alter the composition of communities, as well as important ecosystem functions, for instance the cycling of elements such as carbon and nitrogen. A key role is played by the microbes. Understanding the interplay of biotic and abiotic drivers in subterranean ecosystems is required to anticipate future effects of climate change on groundwater resources and habitats. This review summarizes potential threats to groundwater ecosystems with emphasis on climate change and the microbial world down below our feet in the water saturated subsurface.

[Source](https://doi.org/10.21775/cimb.041.509)

10.21775/cimb.041.509

---

## Combining IL-2-based immunotherapy with commensal probiotics produces enhanced antitumor immune response and tumor clearance.
 08.10.2020, _L Shi, J Sheng, G Chen, P Zhu, C Shi, B Li, C Park, J Wang, B Zhang, Z Liu, X Yang_


Interleukin-2 (IL-2) serves as a pioneer of immunotherapeutic agent in cancer treatment. However, there is a considerable proportion of patients who cannot benefit from this therapy due to the limited clinical responses and dose-limiting toxicities. Mounting evidence indicates that commensal microbiota shapes the outcome of cancer immunotherapies. In this study, we aim to investigate the enhancing effect of Akkermansia muciniphila (AKK), a beneficial commensal microbe receiving considerable attentions, on the antitumor efficacy of IL-2 and explore the underlying molecular mechanism.

[Source](https://jitc.bmj.com/content/8/2/e000973)

10.1136/jitc-2020-000973

---

## Mom's diet matters: Maternal prebiotic intake in mice reduces anxiety and alters brain gene expression and the fecal microbiome in offspring.
 08.10.2020, _JC Hebert, DE Radford-Smith, F Probert, N Ilott, K Wai Chan, DC Anthony, P W J Burnet_


Compelling evidence links enteric microbes to brain function and behavior. Galacto-oligosaccharide prebiotics have been shown to modulate the composition of gut flora and induce metabolic, neurochemical, and behavioral changes in adult rodents. Despite the brain being most susceptible to environmental factors, such as nutrients and toxins, during the earliest stages of development, it is unknown whether maternal prebiotic supplementation during gestation and lactation influences the offspring gut microbiome, brain, or behavior. The aim of this study was to test whether maternal galacto-oligosaccharide intake during pregnancy and lactation alters the brain and behavior in naïve and endotoxin-challenged offspring. CD1 female mice received either normal drinking water or water supplemented with Bimuno® galacto-oligosaccharides (B-GOS) during gestation and suckling. Offspring behavior was tested at weaning age or adulthood, and a cross-foster design was employed in a separate cohort to differentiate between effects of prenatal and postnatal maternal B-GOS intake. Lipopolysaccharide was also administered to pups at postnatal day 9 to determine whether maternal B-GOS influences the neurobiological and behavioral effects of a neonatal pro-inflammatory challenge in adulthood. Fecal microbiome composition and metabolites were analyzed to explore potential relationships between the maternal microbiome, the offspring gut microbiome, and the offspring brain and behavior. Maternal B-GOS supplementation increased exploratory behavior and reduced expression of hippocampal glutamate receptor genes in young, weaning-age offspring. In addition, postnatal, but not prenatal, B-GOS supplementation increased fecal butyrate and propionate levels. Finally, in adult offspring, perinatal B-GOS intake increased cortical glutamate receptor subunits in females, increased social preference, and reduced anxiety. We provide novel and comprehensive evidence for the influence of maternal prebiotic intake on offspring behavior, brain gene expression, and gut microbiome composition in mice.

[Source](https://doi.org/10.1016/j.bbi.2020.09.034)

10.1016/j.bbi.2020.09.034

---

## Prophylactic effects of oral administration of Lactobacillus casei on house dust mite-induced asthma in mice.
 13.10.2020, _L Li, Z Fang, YK Lee, J Zhao, H Zhang, W Lu, W Chen_


This study investigated the prophylactic effects of five Lactobacillus casei strains on house dust mite (HDM)-induced asthma in mice. BALB/c mice were orally administered with L. casei strains for one week before HDM treatment. Ketotifen and Lactobacillus rhamnosus GG were used as positive controls. All L. casei strains decreased the number of granulocytes and the levels of Th2 and Th17 inflammatory cytokines in the lungs, L. casei3 significantly decreased the airway inflammation score. Further studies showed that L. casei3, L. casei4, and L. casei5 decreased the chemokine levels, L. casei2, L. casei4, and L. casei5 promoted the secretion of secretory immunoglobulin A (sIgA), L. casei2 upregulated the interleukin (IL)-10 levels, and L. casei1 had no effect on these immune indices. L. casei1 and L. casei4 decreased the serum levels of total IgE and HDM-specific IgG1, respectively. L. casei3 and L. casei5 decreased both HDM-specific IgG1 and total IgE levels. L. casei2 did not affect the levels of these immunoglobulins. The gut microbiota analysis revealed that all five L. casei strains enhanced the richness of the gut microbiota mainly by increasing the abundance of Firmicutes, while there were differences at the genus level.Thus, the prophylactic effects of L. casei on HDM-induced mixed chronic airway inflammatory asthma exerted as they differentially affected the immune responses and gut microbiota composition. L. casei3, which exhibited the highest prophylactic effect, increased the acetate and propionate contents in a strain-dependent manner.

[Source](https://pubs.rsc.org/en/content/articlelanding/2020/FO/D0FO01363C#!divAbstract)

10.1039/d0fo01363c

---

## INTEDE: interactome of drug-metabolizing enzymes.
 12.10.2020, _J Yin, F Li, Y Zhou, M Mou, Y Lu, K Chen, J Xue, Y Luo, J Fu, X He, J Gao, S Zeng, L Yu, F Zhu_


Drug-metabolizing enzymes (DMEs) are critical determinant of drug safety and efficacy, and the interactome of DMEs has attracted extensive attention. There are 3 major interaction types in an interactome: microbiome-DME interaction (MICBIO), xenobiotics-DME interaction (XEOTIC) and host protein-DME interaction (HOSPPI). The interaction data of each type are essential for drug metabolism, and the collective consideration of multiple types has implication for the future practice of precision medicine. However, no database was designed to systematically provide the data of all types of DME interactions. Here, a database of the Interactome of Drug-Metabolizing Enzymes (INTEDE) was therefore constructed to offer these interaction data. First, 1047 unique DMEs (448 host and 599 microbial) were confirmed, for the first time, using their metabolizing drugs. Second, for these newly confirmed DMEs, all types of their interactions (3359 MICBIOs between 225 microbial species and 185 DMEs; 47 778 XEOTICs between 4150 xenobiotics and 501 DMEs; 7849 HOSPPIs between 565 human proteins and 566 DMEs) were comprehensively collected and then provided, which enabled the crosstalk analysis among multiple types. Because of the huge amount of accumulated data, the INTEDE made it possible to generalize key features for revealing disease etiology and optimizing clinical treatment. INTEDE is freely accessible at: https://idrblab.org/intede/.

[Source](http://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkaa755/5921284)

10.1093/nar/gkaa755

---

## GIMICA: host genetic and immune factors shaping human microbiota.
 12.10.2020, _J Tang, X Wu, M Mou, C Wang, L Wang, F Li, M Guo, J Yin, W Xie, X Wang, Y Wang, Y Ding, W Xue, F Zhu_


Besides the environmental factors having tremendous impacts on the composition of microbial community, the host factors have recently gained extensive attentions on their roles in shaping human microbiota. There are two major types of host factors: host genetic factors (HGFs) and host immune factors (HIFs). These factors of each type are essential for defining the chemical and physical landscapes inhabited by microbiota, and the collective consideration of both types have great implication to serve comprehensive health management. However, no database was available to provide the comprehensive factors of both types. Herein, a database entitled 'Host Genetic and Immune Factors Shaping Human Microbiota (GIMICA)' was constructed. Based on the 4257 microbes confirmed to inhabit nine sites of human body, 2851 HGFs (1368 single nucleotide polymorphisms (SNPs), 186 copy number variations (CNVs), and 1297 non-coding ribonucleic acids (RNAs)) modulating the expression of 370 microbes were collected, and 549 HIFs (126 lymphocytes and phagocytes, 387 immune proteins, and 36 immune pathways) regulating the abundance of 455 microbes were also provided. All in all, GIMICA enabled the collective consideration not only between different types of host factor but also between the host and environmental ones, which is freely accessible without login requirement at: https://idrblab.org/gimica/.

[Source](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkaa851/5921285)

10.1093/nar/gkaa851

---

## metaFlye: scalable long-read metagenome assembly using repeat graphs.
 06.10.2020, _M Kolmogorov, DM Bickhart, B Behsaz, A Gurevich, M Rayko, SB Shin, K Kuhn, J Yuan, E Polevikov, TPL Smith, PA Pevzner_


Long-read sequencing technologies have substantially improved the assemblies of many isolate bacterial genomes as compared to fragmented short-read assemblies. However, assembling complex metagenomic datasets remains difficult even for state-of-the-art long-read assemblers. Here we present metaFlye, which addresses important long-read metagenomic assembly challenges, such as uneven bacterial composition and intra-species heterogeneity. First, we benchmarked metaFlye using simulated and mock bacterial communities and show that it consistently produces assemblies with better completeness and contiguity than state-of-the-art long-read assemblers. Second, we performed long-read sequencing of the sheep microbiome and applied metaFlye to reconstruct 63 complete or nearly complete bacterial genomes within single contigs. Finally, we show that long-read assembly of human microbiomes enables the discovery of full-length biosynthetic gene clusters that encode biomedically important natural products.

[Source](https://www.nature.com/articles/s41592-020-00971-x)

10.1038/s41592-020-00971-x

---

## Diagnosing and Predicting Mixed-Culture Fermentations with Unicellular and Guild-Based Metabolic Models.
 02.10.2020, _MJ Scarborough, JJ Hamilton, EA Erb, TJ Donohue, DR Noguera_


Multispecies microbial communities determine the fate of materials in the environment and can be harnessed to produce beneficial products from renewable resources. In a recent example, fermentations by microbial communities have produced medium-chain fatty acids (MCFAs). Tools to predict, assess, and improve the performance of these communities, however, are limited. To provide such tools, we constructed two metabolic models of MCFA-producing microbial communities based on available genomic, transcriptomic, and metabolomic data. The first model is a unicellular model (iFermCell215), while the second model (iFermGuilds789) separates fermentation activities into functional guilds. Ethanol and lactate are fermentation products known to serve as substrates for MCFA production, while acetate is another common cometabolite during MCFA production. Simulations with iFermCell215 predict that low molar ratios of acetate to ethanol favor MCFA production, whereas the products of lactate and acetate coutilization are less dependent on the acetate-to-lactate ratio. In simulations of an MCFA-producing community fed a complex organic mixture derived from lignocellulose, iFermGuilds789 predicted that lactate was an extracellular cometabolite that served as a substrate for butyrate (C4) production. Extracellular hexanoic (C6) and octanoic (C8) acids were predicted by iFermGuilds789 to be from community members that directly metabolize sugars. Modeling results provide several hypotheses that can improve our understanding of microbial roles in an MCFA-producing microbiome and inform strategies to increase MCFA production. Further, these models represent novel tools for exploring the role of mixed microbial communities in carbon recycling in the environment, as well as in beneficial reuse of organic residues.IMPORTANCE Microbiomes are vital to human health, agriculture, and protecting the environment. Predicting behavior of self-assembled or synthetic microbiomes, however, remains a challenge. In this work, we used unicellular and guild-based metabolic models to investigate production of medium-chain fatty acids by a mixed microbial community that is fed multiple organic substrates. Modeling results provided insights into metabolic pathways of three medium-chain fatty acid-producing guilds and identified potential strategies to increase production of medium-chain fatty acids. This work demonstrates the role of metabolic models in augmenting multi-omic studies to gain greater insights into microbiome behavior.

[Source](https://msystems.asm.org/content/5/5/e00755-20)

10.1128/mSystems.00755-20

---

## The Bean Beetle Microbiome Project: A Course-Based Undergraduate Research Experience in Microbiology.
 13.10.2020, _AJ Zelaya, NM Gerardo, LS Blumer, CW Beck_


Course-based undergraduate research experiences (CUREs) are an effective means of transforming the learning and teaching of science by involving students in the scientific process. The potential importance of the microbiome in shaping both environmental health and disease makes investigations of microbiomes an excellent teaching tool for undergraduate microbiology. Here, we present a CURE based on the microbiome of the bean beetle (Callosobruchus maculatus), a model system for undergraduate laboratory education. Despite the extensive research literature on bean beetles, little is known about their microbiome, making them an ideal system for a discovery-based CURE. In the CURE, students acquire microbiological technical skills by characterizing both culturable and unculturable members of the beetle gut-microbial community. Students plate beetle gut homogenates on different media, describe the colonies that are formed to estimate taxonomic diversity, extract DNA from colonies of interest, PCR amplify the16S rRNA gene for Sanger sequencing, and use the NCBI-nBLAST database to taxonomically classify sequences. Additionally, students extract total DNA from beetle gut homogenates for high-throughput paired-end sequencing and perform bioinformatic and statistical analyses of bacterial communities using a combination of open-access data processing software. Each activity allows students to engage with studies of microbiomes in a real-world context, to apply concepts and laboratory techniques to investigate either student or faculty-driven research questions, and to gain valuable experiences working with large high-throughput datasets. The CURE is designed such that it can be implemented over either 6-weeks (half semester) or 12-weeks (full semester), allowing for flexibility within the curriculum. Furthermore, student-generated data from the CURE (including bacterial colony phenotypic data, full-length 16S rRNA gene sequences from cultured isolates, and bacterial community sequences from gut homogenates) has been compiled in a continuously curated open-access database on the Bean Beetle Microbiome Project website, facilitating the generation of broader research questions across laboratory classrooms.

[Source](https://doi.org/10.3389/fmicb.2020.577621)

10.3389/fmicb.2020.577621

---

