## Safety and efficacy of faecal microbiota transplantation in patients with mild to moderate Parkinson's disease (GUT-PARFECT): a double-blind, placebo-controlled, randomised, phase 2 trial.
 30.04.2024, _A Bruggeman, C Vandendriessche, H Hamerlinck, D De Looze, DJ Tate, M Vuylsteke, L De Commer, L Devolder, J Raes, B Verhasselt, D Laukens, RE Vandenbroucke, P Santens_


Dysregulation of the gut microbiome has been implicated in Parkinson's disease (PD). This study aimed to evaluate the clinical effects and safety of a single faecal microbiota transplantation (FMT) in patients with early-stage PD.

[Source](https://www.thelancet.com/journals/eclinm/article/PIIS2589-5370(24)00142-1/fulltext)

10.1016/j.eclinm.2024.102563

---

## Implications of intestinal microecology and immune function alterations for immunotherapy outcomes in advanced unresectable lung adenocarcinoma.
 30.04.2024, _S He, J Tian, J Zang, L Long, P Liu, Y Zhang, J Xiao_


_Humans, Gastrointestinal Microbiome, Male, Female, Lung Neoplasms, Middle Aged, Adenocarcinoma of Lung, Immunotherapy, Aged, RNA, Ribosomal, 16S, Treatment Outcome_
This investigation aims to explore alterations in intestinal microecology and immune function among patients with advanced, unresectable lung adenocarcinoma undergoing different outcomes from immunotherapy.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/crj.13762)

10.1111/crj.13762

---

## Recipient microbiome-related features predicting metabolic improvement following fecal microbiota transplantation in adults with severe obesity and metabolic syndrome: a secondary analysis of a phase 2 clinical trial.
 30.04.2024, _Z Zhang, V Mocanu, EC Deehan, N Hotte, Y Zhu, S Wei, DH Kao, S Karmali, DW Birch, J Walter, KL Madsen_


_Humans, Fecal Microbiota Transplantation, Metabolic Syndrome, Male, Female, Gastrointestinal Microbiome, Adult, Middle Aged, Feces, Bile Acids and Salts, Bacteria, Obesity, Dietary Fiber, Insulin Resistance, Treatment Outcome_
Microbial-based therapeutics in clinical practice are of considerable interest, and a recent study demonstrated fecal microbial transplantation (FMT) followed by dietary fiber supplements improved glucose homeostasis. Previous evidence suggests that donor and recipient compatibility and FMT protocol are key determinants, but little is known about the involvement of specific recipient factors. Using data from our recent randomized placebo-control phase 2 clinical trial in adults with obesity and metabolic syndrome, we grouped participants that received FMT from one of 4 donors with either fiber supplement into HOMA-IR responders (n = 21) and HOMA-IR non-responders (n = 8). We further assessed plasma bile acids using targeted metabolomics and performed subgroup analyzes to evaluate the effects of recipient parameters and gastrointestinal factors on microbiota engraftment and homeostatic model assessment of insulin resistance (HOMA2-IR) response. The baseline fecal microbiota composition at genus level of recipients could predict the improvements in HOMA2-IR at week 6 (ROC-AUC = 0.70). Prevotella was identified as an important predictor, with responders having significantly lower relative abundance than non-responders (p = .02). In addition, recipients displayed a highly individualized degree of microbial engraftment from donors. Compared to the non-responders, the responders had significantly increased bacterial richness (Chao1) after FMT and a more consistent engraftment of donor-specific bacteria ASVs (amplicon sequence variants) such as Faecalibacillus intestinalis (ASV44), Roseburia spp. (ASV103), and Christensenellaceae spp. (ASV140) (p &lt; .05). Microbiota engraftment was strongly associated with recipients' factors at baseline including initial gut microbial diversity, fiber and nutrient intakes, inflammatory markers, and bile acid derivative levels. This study identified that responders to FMT therapy had a higher engraftment rate in the transplantation of specific donor-specific microbes, which were strongly correlated with insulin sensitivity improvements. Further, the recipient baseline gut microbiota and related factors were identified as the determinants for responsiveness to FMT and fiber supplementation. The findings provide a basis for the development of precision microbial therapeutics for the treatment of metabolic syndrome.

[Source](https://www.tandfonline.com/doi/full/10.1080/19490976.2024.2345134)

10.1080/19490976.2024.2345134

---

## Gut microbiome composition and metabolic activity in women with diverticulitis.
 29.04.2024, _W Ma, Y Wang, LH Nguyen, RS Mehta, J Ha, A Bhosle, LJ Mclver, M Song, CB Clish, LL Strate, C Huttenhower, AT Chan_


_Humans, Female, Gastrointestinal Microbiome, Middle Aged, Diverticulitis, Feces, Aged, Prospective Studies, Bilophila, Metabolomics, Case-Control Studies, Clostridiales, Bile Acids and Salts, Adult, Dietary Fiber, Metabolome, Metagenomics_
The etiopathogenesis of diverticulitis, among the most common gastrointestinal diagnoses, remains largely unknown. By leveraging stool collected within a large prospective cohort, we performed shotgun metagenomic sequencing and untargeted metabolomics profiling among 121 women diagnosed with diverticulitis requiring antibiotics or hospitalizations (cases), matched to 121 women without diverticulitis (controls) according to age and race. Overall microbial community structure and metabolomic profiles differed in diverticulitis cases compared to controls, including enrichment of pro-inflammatory Ruminococcus gnavus, 1,7-dimethyluric acid, and histidine-related metabolites, and depletion of butyrate-producing bacteria and anti-inflammatory ceramides. Through integrated multi-omic analysis, we detected covarying microbial and metabolic features, such as Bilophila wadsworthia and bile acids, specific to diverticulitis. Additionally, we observed that microbial composition modulated the protective association between a prudent fiber-rich diet and diverticulitis. Our findings offer insights into the perturbations in inflammation-related microbial and metabolic signatures associated with diverticulitis, supporting the potential of microbial-based diagnostics and therapeutic targets.

[Source](https://www.nature.com/articles/s41467-024-47859-4)

10.1038/s41467-024-47859-4

---

## Microbiome confounders and quantitative profiling challenge predicted microbial targets in colorectal cancer development.
 30.04.2024, _RY Tito, S Verbandt, M Aguirre Vazquez, L Lahti, C Verspecht, V Lloréns-Rico, S Vieira-Silva, J Arts, G Falony, E Dekker, J Reumers, S Tejpar, J Raes_


Despite substantial progress in cancer microbiome research, recognized confounders and advances in absolute microbiome quantification remain underused; this raises concerns regarding potential spurious associations. Here we study the fecal microbiota of 589 patients at different colorectal cancer (CRC) stages and compare observations with up to 15 published studies (4,439 patients and controls total). Using quantitative microbiome profiling based on 16S ribosomal RNA amplicon sequencing, combined with rigorous confounder control, we identified transit time, fecal calprotectin (intestinal inflammation) and body mass index as primary microbial covariates, superseding variance explained by CRC diagnostic groups. Well-established microbiome CRC targets, such as Fusobacterium nucleatum, did not significantly associate with CRC diagnostic groups (healthy, adenoma and carcinoma) when controlling for these covariates. In contrast, the associations of Anaerococcus vaginalis, Dialister pneumosintes, Parvimonas micra, Peptostreptococcus anaerobius, Porphyromonas asaccharolytica and Prevotella intermedia remained robust, highlighting their future target potential. Finally, control individuals (age 22-80 years, mean 57.7 years, standard deviation 11.3) meeting criteria for colonoscopy (for example, through a positive fecal immunochemical test) but without colonic lesions are enriched for the dysbiotic Bacteroides2 enterotype, emphasizing uncertainties in defining healthy controls in cancer microbiome research. Together, these results indicate the importance of quantitative microbiome profiling and covariate control for biomarker identification in CRC microbiome studies.

[Source](https://www.nature.com/articles/s41591-024-02963-2)

10.1038/s41591-024-02963-2

---

## Prediction and causal inference of hyperuricemia using gut microbiota.
 30.04.2024, _Y Miyajima, S Karashima, R Mizoguchi, M Kawakami, K Ogura, K Ogai, A Koshida, Y Ikagawa, Y Ami, Q Zhu, H Tsujiguchi, A Hara, S Kurihara, H Arakawa, H Nakamura, I Tamai, H Nambo, S Okamoto_


_Humans, Gastrointestinal Microbiome, Hyperuricemia, Male, Female, Adult, RNA, Ribosomal, 16S, Machine Learning, Uric Acid, Feces, High-Throughput Nucleotide Sequencing, Middle Aged_
Hyperuricemia (HUA) is a symptom of high blood uric acid (UA) levels, which causes disorders such as gout and renal urinary calculus. Prolonged HUA is often associated with hypertension, atherosclerosis, diabetes mellitus, and chronic kidney disease. Studies have shown that gut microbiota (GM) affect these chronic diseases. This study aimed to determine the relationship between HUA and GM. The microbiome of 224 men and 254 women aged 40 years was analyzed through next-generation sequencing and machine learning. We obtained GM data through 16S rRNA-based sequencing of the fecal samples, finding that alpha-diversity by Shannon index was significantly low in the HUA group. Linear discriminant effect size analysis detected a high abundance of the genera Collinsella and Faecalibacterium in the HUA and non-HUA groups. Based on light gradient boosting machine learning, we propose that HUA can be predicted with high AUC using four clinical characteristics and the relative abundance of nine bacterial genera, including Collinsella and Dorea. In addition, analysis of causal relationships using a direct linear non-Gaussian acyclic model indicated a positive effect of the relative abundance of the genus Collinsella on blood UA levels. Our results suggest abundant Collinsella in the gut can increase blood UA levels.

[Source](https://www.nature.com/articles/s41598-024-60427-6)

10.1038/s41598-024-60427-6

---

## Early-life gut microbiota associates with allergic rhinitis during 13-year follow-up in a Finnish probiotic intervention cohort

[Source](https://journals.asm.org/doi/10.1128/spectrum.04135-23)

---

## The detailed analysis of the microbiome and resistome of artisanal blue-veined cheeses provides evidence on sources and patterns of succession linked with quality and safety traits.
 30.04.2024, _EA Alexa, JF Cobo-Díaz, E Renes, TF O Callaghan, K Kilcawley, D Mannion, I Skibinska, L Ruiz, A Margolles, P Fernández-Gómez, A Alvarez-Molina, P Puente-Gómez, F Crispie, M López, M Prieto, PD Cotter, A Alvarez-Ordóñez_


_Cheese, Microbiota, Food Microbiology, Bacteria, Caves, Corynebacterium, Longitudinal Studies, Metagenome, Lactococcus, RNA, Ribosomal, 16S, Staphylococcus, Food Safety_
Artisanal cheeses usually contain a highly diverse microbial community which can significantly impact their quality and safety. Here, we describe a detailed longitudinal study assessing the impact of ripening in three natural caves on the microbiome and resistome succession across three different producers of Cabrales blue-veined cheese.

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01790-4)

10.1186/s40168-024-01790-4

---

## Sodium Propionate Oral Supplementation ameliorates Depressive-Like Behavior through Gut Microbiome and Histone 3 Epigenetic Regulation.
 30.04.2024, _LMP Behrens, J Gasparotto, PH Rampelotto, MAR Escalona, LDS da Silva, FG Carazza-Kessler, CP Barbosa, MS Campos, M Dorn, DP Gelain, JCF Moreira_


Major Depressive Disorder (MDD) is a global health concern, affecting over 250 million individuals worldwide. In recent years, the gut-brain axis has emerged as a promising field for understanding the pathophysiology of MDD. Microbial metabolites, such as Short-Chain Fatty Acids (SCFAs) - acetate, butyrate, and propionate -, have gained attention for their potential to influence epigenetic modifications within the host brain. However, the precise mechanisms through which these metabolites participate in MDD pathophysiology remain elusive. This study was designed to investigate the effects of oral SCFA supplementation in adult male Wistar rats subjected to Chronic Unpredictable Mild Stress (CUMS). A subset of control and CUMS-exposed rats received different supplementations: sodium acetate (NaOAc) at a concentration of 60 mM, sodium butyrate (NaB) at 40 mM, sodium propionate (NaP) at 50 mM, or a mixture of these SCFAs. The gut microbiome was assessed through 16S rRNA sequencing, and epigenetic profiling was performed using Western blot analysis. Results demonstrated that NaP supplementation significantly alleviated anhedonia in stressed animals, as evidenced by improved performance in the sucrose consumption test. This ameliorative effect was potentially associated with the modulation of gut bacterial communities, accompanied by the attenuation of the region-specific epigenetic dysregulation in the brain of the animals exposed to chronic stress. These findings suggest a potential association between gut dysbiosis and stress response, and NaP could be a promising target for future MDD interventions. However, further studies are needed to fully elucidate the underlying mechanisms of these effects.

[Source](https://www.sciencedirect.com/science/article/pii/S0955286324000937)

10.1016/j.jnutbio.2024.109660

---

## Colon impairments and inflammation driven by an altered gut microbiota leads to social behavior deficits rescued by hyaluronic acid and celecoxib.
 29.04.2024, _O Agranyoni, D Sur, S Amidror, N Shidlovsky, A Bagaev, N Yissachar, A Pinhasov, S Navon-Venezia_


_Animals, Gastrointestinal Microbiome, Celecoxib, Mice, Hyaluronic Acid, Social Behavior, Colon, Inflammation, Male, Behavior, Animal, RNA, Ribosomal, 16S_
The exact mechanisms linking the gut microbiota and social behavior are still under investigation. We aimed to explore the role of the gut microbiota in shaping social behavior deficits using selectively bred mice possessing dominant (Dom) or submissive (Sub) behavior features. Sub mice exhibit asocial, depressive- and anxiety-like behaviors, as well as systemic inflammation, all of which are shaped by their impaired gut microbiota composition.

[Source](https://bmcmedicine.biomedcentral.com/articles/10.1186/s12916-024-03323-0)

10.1186/s12916-024-03323-0

---

## Dissecting the respective roles of microbiota and host genetics in the susceptibility of Card9−/− mice to colitis 

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01798-w)

---

## Synergistic resistance of honeybee (Apis mellifera) and their gut microorganisms to fluvalinate stress.
 30.04.2024, _J Liu, C Liao, Z Li, X Shi, X Wu_


_Animals, Bees, Gastrointestinal Microbiome, Pyrethrins, Insecticides, Insect Proteins, Cytochrome P-450 Enzyme System, RNA, Ribosomal, 16S, Nitriles, Antimicrobial Cationic Peptides_
Fluvalinate is widely used in the control of Varroa destructor, but its residues in colonies threaten honeybees. The effect of fluvalinate-induced dysbiosis on honeybee-related gene expression and the gut microenvironment of honeybees has not yet been fully elucidated. In this study, two-day-old larvae to seven-day-old adult worker bees were continuously fed different amounts of fluvalinate-sucrose solutions (0, 0.5, 5, and 50 mg/kg), after which the expression levels of two immune-related genes (Hymenoptaecin and Defensin1) and three detoxication-related genes (GSTS3, CAT, and CYP450) in worker bees (1, 7, and 20 days old) were measured. The effect of fluvalinate on the gut microbes of worker bees at seven days old also was explored using 16S rRNA Illumina deep sequencing. The results showed that exposure of honeybees to the insecticide fluvalinate affected their gene expression and gut microbial composition. As the age of honeybees increased, the effect of fluvalinate on the expression of Hymenoptaecin, CYP450, and CAT decreased, and the abundance of honeybee gut bacteria was affected by increasing the fluvalinate concentration. These findings provide insights into the synergistic defense of honeybee hosts against exogenous stresses in conjunction with honeybee gut microbes.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S0048357524000981)

10.1016/j.pestbp.2024.105865

---

## Determinants of microbiome composition: Insights from free-ranging hybrid zebras (Equus quagga × grevyi).
 29.04.2024, _JO Abraham, B Lin, AE Miller, LP Henry, MY Demmel, R Warungu, M Mwangi, PM Lobura, LF Pallares, JF Ayroles, RM Pringle, DI Rubenstein_


The composition of mammalian gut microbiomes is highly conserved within species, yet the mechanisms by which microbiome composition is transmitted and maintained within lineages of wild animals remain unclear. Mutually compatible hypotheses exist, including that microbiome fidelity results from inherited dietary habits, shared environmental exposure, morphophysiological filtering and/or maternal effects. Interspecific hybrids are a promising system in which to interrogate the determinants of microbiome composition because hybrids can decouple traits and processes that are otherwise co-inherited in their parent species. We used a population of free-living hybrid zebras (Equus quagga × grevyi) in Kenya to evaluate the roles of these four mechanisms in regulating microbiome composition. We analysed faecal DNA for both the trnL-P6 and the 16S rRNA V4 region to characterize the diets and microbiomes of the hybrid zebra and of their parent species, plains zebra (E. quagga) and Grevy's zebra (E. grevyi). We found that both diet and microbiome composition clustered by species, and that hybrid diets and microbiomes were largely nested within those of the maternal species, plains zebra. Hybrid microbiomes were less variable than those of either parent species where they co-occurred. Diet and microbiome composition were strongly correlated, although the strength of this correlation varied between species. These patterns are most consistent with the maternal-effects hypothesis, somewhat consistent with the diet hypothesis, and largely inconsistent with the environmental-sourcing and morphophysiological-filtering hypotheses. Maternal transmittance likely operates in conjunction with inherited feeding habits to conserve microbiome composition within species.

[Source](https://onlinelibrary.wiley.com/doi/10.1111/mec.17370)

10.1111/mec.17370

---

## Phytoplankton-derived polysaccharides and microbial peptidoglycans are key nutrients for deep-sea microbes in the Mariana Trench

[Source](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-024-01789-x)

---

## Rhizospheric bacteria from the Atacama Desert hyper-arid core: cultured community dynamics and plant growth promotion.
 30.04.2024, _J Castro-Severyn, J Fortt, M Sierralta, P Alegria, G Donoso, A Choque, AM Avellaneda, C Pardo-Esté, CP Saavedra, A Stoll, F Remonsellez_


The Atacama Desert is the oldest and driest desert on Earth, encompassing great temperature variations, high ultraviolet radiation, drought, and high salinity, making it ideal for studying the limits of life and resistance strategies. It is also known for harboring a great biodiversity of adapted life forms. While desertification is increasing as a result of climate change and human activities, it is necessary to optimize soil and water usage, where stress-resistant crops are possible solutions. As many studies have revealed the great impact of the rhizobiome on plant growth efficiency and resistance to abiotic stress, we set up to explore the rhizospheric soils of Suaeda foliosa and Distichlis spicata desert plants. By culturing these soils and using 16S rRNA amplicon sequencing, we address community taxonomy composition dynamics, stability through time, and the ability to promote lettuce plant growth. The rhizospheric soil communities were dominated by the families Pseudomonadaceae, Bacillaceae, and Planococcaceae for S. foliosa and Porphyromonadaceae and Haloferacaceae for D. spicata. Nonetheless, the cultures were completely dominated by the Enterobacteriaceae family (up to 98%). Effectively, lettuce plants supplemented with the cultures showed greater size and biomass accumulation. We identified 12 candidates that could be responsible for these outcomes, of which 5 (Enterococcus, Pseudomonas, Klebsiella, Paenisporosarcina, and Ammoniphilus) were part of the built co-occurrence network. We aim to contribute to the efforts to characterize the microbial communities as key for the plant's survival in extreme environments and as a possible source of consortia with plant growth promotion traits aimed at agricultural applications.IMPORTANCEThe current scenario of climate change and desertification represents a series of incoming challenges for all living organisms. As the human population grows rapidly, so does the rising demand for food and natural resources; thus, it is necessary to make agriculture more efficient by optimizing soil and water usage, thus ensuring future food supplies. Particularly, the Atacama Desert (northern Chile) is considered the most arid place on Earth as a consequence of geological and climatic characteristics, such as the naturally low precipitation patterns and high temperatures, which makes it an ideal place to carry out research that seeks to aid agriculture in future conditions that are predicted to resemble these scenarios. Our main interest lies in utilizing microorganism consortia from plants thriving under extreme conditions, aiming to promote plant growth, improve crops, and render "unsuitable" soils farmable.

[Source](https://journals.asm.org/doi/10.1128/spectrum.00056-24)

10.1128/spectrum.00056-24

---

## Variability and bias in microbiome metagenomic sequencing: an interlaboratory study comparing experimental protocols.
 29.04.2024, _SP Forry, SL Servetas, JG Kralj, K Soh, M Hadjithomas, R Cano, M Carlin, MG Amorim, B Auch, MG Bakker, TF Bartelli, JP Bustamante, I Cassol, M Chalita, E Dias-Neto, AD Duca, DM Gohl, J Kazantseva, MT Haruna, P Menzel, BS Moda, L Neuberger-Castillo, DN Nunes, IR Patel, RD Peralta, A Saliou, R Schwarzer, S Sevilla, IKTM Takenaka, JR Wang, R Knight, D Gevers, SA Jackson_


_Humans, Metagenomics, RNA, Ribosomal, 16S, Feces, Microbiota, Bias, Metagenome, Gastrointestinal Microbiome, Sequence Analysis, DNA, Bacteria, High-Throughput Nucleotide Sequencing_
Several studies have documented the significant impact of methodological choices in microbiome analyses. The myriad of methodological options available complicate the replication of results and generally limit the comparability of findings between independent studies that use differing techniques and measurement pipelines. Here we describe the Mosaic Standards Challenge (MSC), an international interlaboratory study designed to assess the impact of methodological variables on the results. The MSC did not prescribe methods but rather asked participating labs to analyze 7 shared reference samples (5 × human stool samples and 2 × mock communities) using their standard laboratory methods. To capture the array of methodological variables, each participating lab completed a metadata reporting sheet that included 100 different questions regarding the details of their protocol. The goal of this study was to survey the methodological landscape for microbiome metagenomic sequencing (MGS) analyses and the impact of methodological decisions on metagenomic sequencing results. A total of 44 labs participated in the MSC by submitting results (16S or WGS) along with accompanying metadata; thirty 16S rRNA gene amplicon datasets and 14 WGS datasets were collected. The inclusion of two types of reference materials (human stool and mock communities) enabled analysis of both MGS measurement variability between different protocols using the biologically-relevant stool samples, and MGS bias with respect to ground truth values using the DNA mixtures. Owing to the compositional nature of MGS measurements, analyses were conducted on the ratio of Firmicutes: Bacteroidetes allowing us to directly apply common statistical methods. The resulting analysis demonstrated that protocol choices have significant effects, including both bias of the MGS measurement associated with a particular methodological choices, as well as effects on measurement robustness as observed through the spread of results between labs making similar methodological choices. In the analysis of the DNA mock communities, MGS measurement bias was observed even when there was general consensus among the participating laboratories. This study was the result of a collaborative effort that included academic, commercial, and government labs. In addition to highlighting the impact of different methodological decisions on MGS result comparability, this work also provides insights for consideration in future microbiome measurement study design.

[Source](https://www.nature.com/articles/s41598-024-57981-4)

10.1038/s41598-024-57981-4

---

## Nanopore and Illumina sequencing reveal different viral populations from human gut samples.
 29.04.2024, _R Cook, A Telatin, SY Hsieh, F Newberry, MA Tariq, DJ Baker, SR Carding, EM Adriaenssens_


_Humans, High-Throughput Nucleotide Sequencing, Metagenomics, Genome, Viral, Gastrointestinal Microbiome, Feces, Nanopores, Nanopore Sequencing, Viruses, Virome, Sequence Analysis, DNA_
The advent of viral metagenomics, or viromics, has improved our knowledge and understanding of global viral diversity. High-throughput sequencing technologies enable explorations of the ecological roles, contributions to host metabolism, and the influence of viruses in various environments, including the human intestinal microbiome. However, bacterial metagenomic studies frequently have the advantage. The adoption of advanced technologies like long-read sequencing has the potential to be transformative in refining viromics and metagenomics. Here, we examined the effectiveness of long-read and hybrid sequencing by comparing Illumina short-read and Oxford Nanopore Technology (ONT) long-read sequencing technologies and different assembly strategies on recovering viral genomes from human faecal samples. Our findings showed that if a single sequencing technology is to be chosen for virome analysis, Illumina is preferable due to its superior ability to recover fully resolved viral genomes and minimise erroneous genomes. While ONT assemblies were effective in recovering viral diversity, the challenges related to input requirements and the necessity for amplification made it less ideal as a standalone solution. However, using a combined, hybrid approach enabled a more authentic representation of viral diversity to be obtained within samples.

[Source](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001236)

10.1099/mgen.0.001236

---

