## A relational framework for microbiome research with Indigenous communities.
 03.10.2023, _AC Bader, EM Van Zuylen, M Handsley-Davis, RA Alegado, A Benezra, RM Pollet, H Ehau-Taumaunu, LS Weyrich, MZ Anderson_


_Humans, Population Groups, Microbiota_
Ethical practices in human microbiome research have failed to keep pace with scientific advances in the field. Researchers seeking to 'preserve' microbial species associated with Indigenous groups, but absent from industrialized populations, have largely failed to include Indigenous people in knowledge co-production or benefit, perpetuating a legacy of intellectual and material extraction. We propose a framework centred on relationality among Indigenous peoples, researchers and microbes, to guide ethical microbiome research. Our framework centres accountability to flatten historical power imbalances that favour researcher perspectives and interests to provide space for Indigenous worldviews in pursuit of Indigenous research sovereignty. Ethical inclusion of Indigenous communities in microbiome research can provide health benefits for all populations and reinforce mutually beneficial partnerships between researchers and the public.

[Source](https://doi.org/10.1038/s41564-023-01471-2)

10.1038/s41564-023-01471-2

---

## Microbiome ownership for Indigenous peoples.
 03.10.2023, _M Handsley-Davis, MZ Anderson, AC Bader, H Ehau-Taumaunu, K Fox, E Kowal, LS Weyrich_


_Humans, Ownership, Indigenous Peoples_
Several studies have reported increased microbial diversity, or distinct microbial community compositions, in the microbiomes of Indigenous peoples around the world. However, there is a widespread failure to include Indigenous cultures and perspectives in microbiome research programmes, and ethical issues pertaining to microbiome research involving Indigenous participants have not received enough attention. We discuss the benefits and risks arising from microbiome research involving Indigenous peoples and analyse microbiome ownership as an ethical concept in this context. We argue that microbiome ownership represents an opportunity for Indigenous peoples to steward and protect their resident microbial communities at every stage of research.

[Source](https://doi.org/10.1038/s41564-023-01470-3)

10.1038/s41564-023-01470-3

---

## Expanded microbiome niches of RAG-deficient patients

[Source](https://www.cell.com/cell-reports-medicine/fulltext/S2666-3791(23)00372-5)

---

## Low dietary fiber intake impairs small intestinal Th17 and intraepithelial T&nbsp;cell development over generations

[Source](https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01152-X)

---

## Iron acquisition by a commensal bacterium modifies host nutritional immunity during <em>Salmonella</em> infection

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312823003438?via%3Dihub)

---

## Microbiome profile and calprotectin levels as markers of risk of recurrent <em>Clostridioides difficile</em> infection

[Source](https://doi.org/10.3389/fcimb.2023.1237500)

---

## Safety and tolerability of frozen, capsulized autologous faecal microbiota transplantation. A randomized double blinded phase I clinical trial.
 29.09.2023, _M Stefansson, O Bladh, O Flink, O Skolling, HP Ekre, L Rombo, L Engstrand, J Ursing_


Faecal microbiota transplantation (FMT) is recommended treatment for recurrent Clostridioides difficile infection and is studied as a potential modifier of other gastrointestinal and systemic disorders. Autologous FMT limits the potential risks of donor transplant material and enables prophylactic treatment. Capsulized FMT is convenient and accessible, but safety data are lacking.

[Source](https://doi.org/10.1371/journal.pone.0292132)

10.1371/journal.pone.0292132

---

## Metabolism of azo food dyes by bacterial members of the human gut microbiome.
 28.09.2023, _R Elder, SJ Vancuren, AJ Botschner, PD Josephy, E Allen-Vercoe_


We set out to survey the capacities of bacterial isolates from the human gut microbiome to reduce common azo food dyes in vitro.

[Source](https://doi.org/10.1016/j.anaerobe.2023.102783)

10.1016/j.anaerobe.2023.102783

---

## Genetic risk score in multiple sclerosis is associated with unique gut microbiome.
 03.10.2023, _NS Elsayed, RK Valenzuela, T Kitchner, T Le, J Mayer, ZZ Tang, VR Bayanagari, Q Lu, P Aston, K Anantharaman, SK Shukla_


_Humans, Gastrointestinal Microbiome, Multiple Sclerosis, Dysbiosis, Genetic Predisposition to Disease, RNA, Ribosomal, 16S, Risk Factors_
Multiple sclerosis (MS) is a complex autoimmune disease in which both the roles of genetic susceptibility and environmental/microbial factors have been investigated. More than 200 genetic susceptibility variants have been identified along with the dysbiosis of gut microbiota, both independently have been shown to be associated with MS. We hypothesize that MS patients harboring genetic susceptibility variants along with gut microbiome dysbiosis are at a greater risk of exhibiting the disease. We investigated the genetic risk score for MS in conjunction with gut microbiota in the same cohort of 117 relapsing remitting MS (RRMS) and 26 healthy controls. DNA samples were genotyped using Illumina's Infinium Immuno array-24 v2 chip followed by calculating genetic risk score and the microbiota was determined by sequencing the V4 hypervariable region of the 16S rRNA gene. We identified two clusters of MS patients, Cluster A and B, both having a higher genetic risk score than the control group. However, the MS cases in cluster B not only had a higher genetic risk score but also showed a distinct gut microbiome than that of cluster A. Interestingly, cluster A which included both healthy control and MS cases had similar gut microbiome composition. This could be due to (i) the non-active state of the disease in that group of MS patients at the time of fecal sample collection and/or (ii) the restoration of the gut microbiome post disease modifying therapy to treat the MS. Our study showed that there seems to be an association between genetic risk score and gut microbiome dysbiosis in triggering the disease in a small cohort of MS patients. The MS Cluster A who have a higher genetic risk score but microbiome profile similar to that of healthy controls could be due to the remitting phase of the disease or due to the effect of disease modifying therapies.

[Source](https://doi.org/10.1038/s41598-023-43217-4)

10.1038/s41598-023-43217-4

---

## Antisense inhibition of RNA polymerase α subunit of <em>Clostridioides difficile</em>

[Source](https://doi.org/10.1128/spectrum.01755-23)

---

## Metabolic signature for a dysbiotic microbiome in the female genital tract: A systematic review and meta-analysis.
 28.09.2023, _B Gill, I Schwecht, N Rahman, T Dhawan, C Verschoor, A Nazli, C Kaushic_


The vaginal microbiome (VMB) is a critical determinant of reproductive health, where a microbial shift towards a dysbiotic environment has implications for susceptibility to, and clinical presentation of sexually transmitted infections (STIs). Metabolomic profiling of the vaginal microenvironment has led to the identification of metabolic responses to clinical conditions of dysbiosis. However, no studies have examined metabolic markers that are common across conditions and can serve as a signature for vaginal dysbiosis.

[Source](https://doi.org/10.1111/aji.13781)

10.1111/aji.13781

---

## Influence of Plant-Based Diet on the Cardiovascular System: A Narrative Review.
 28.09.2023, _G Jangid, HAO Popoola-Samuel, K Goda, FNU Anamika, V Gupta, SG Kanagala, RS Munjal_


A plant-based diet (PBD) includes multiple dietary patterns such as vegetarianism, veganism, lacto-ovo-vegetarianism, etc. A well-balanced PBD has positive effects on body mass index, hyperlipidemia, and type 2 diabetes, and it is associated with decreased risk of cardiovascular diseases. Due to its high fiber content, which lowers levels of low-density lipoprotein cholesterol, high antioxidant content, which prevents capillary wall damage; the presence of polyunsaturated fatty acids, which have an anti-inflammatory effect, improvement in glucose homeostasis through improved insulin sensitivity, and impact on the gut microbiota, it has been proven to be beneficial for the heart. Despite their possible benefits, PBDs can be low in essential nutrients such as vitamin B12, zinc, vitamin D, iodine, and amino acids. This review aims to analyze the influence of PBDs on the cardiovascular system and associated disease progression.

[Source](https://doi.org/10.1097/CRD.0000000000000613)

10.1097/CRD.0000000000000613

---

## Bile acid metabolism regulatory network orchestrates bone homeostasis.
 30.09.2023, _T Xiang, Z Deng, C Yang, J Tan, C Dou, F Luo, Y Chen_


Bile acids (BAs), synthesized in the liver and modified by the gut microbiota, have been widely appreciated not only as simple lipid emulsifiers, but also as complex metabolic regulators and momentous signaling molecules, which play prominent roles in the complex interaction among several metabolic systems. Recent studies have drawn us eyes on the diverse physiological functions of BAs, to enlarge the knowledge about the "gut-bone" axis due to the participation about the gut microbiota-derived BAs to modulate bone homeostasis at physiological and pathological stations. In this review, we have summarized the metabolic processes of BAs and highlighted the crucial roles of BAs targeting bile acid-activated receptors (BARs), promoting the proliferation and differentiation of osteoblasts (OBs), inhibiting the activity of osteoclasts (OCs), as well as reducing articular cartilage degradation, thus facilitating bone repair. In addition, we have also focused on the bidirectional effects of BA signaling networks in coordinating the dynamic balance of bone matrix and demonstrated the promising effects of BAs on the development or treatment for pathological bone diseases. In a word, further clinical applications targeting BA metabolism or modulating gut metabolome and related derivatives may be developed as effective therapeutic strategies for bone destruction diseases.

[Source](https://doi.org/10.1080/1040841X.2023.2262020)

10.1016/j.phrs.2023.106943

---

## Bile acid and its bidirectional interactions with gut microbiota: a review

[Source](https://doi.org/10.1080/1040841X.2023.2262020)

---

## Population mixing mediates the intestinal flora composition and facilitates invasiveness in a globally invasive fruit fly

[Source](https://doi.org/10.1186/s40168-023-01664-1)

---

## Diet-related factors strongly shaped the gut microbiota of Japanese macaques.
 28.09.2023, _W Lee, T Hayakawa, M Kiyono, N Yamabata, H Enari, HS Enari, S Fujita, T Kawazoe, T Asai, T Oi, T Kondo, T Uno, K Seki, M Shimada, Y Tsuji, A Langgeng, A MacIntosh, K Suzuki, K Yamada, K Onishi, M Ueno, K Kubo, G Hanya_


Although knowledge of the functions of the gut microbiome has increased greatly over the past few decades, our understanding of the mechanisms governing its ecology and evolution remains obscure. While host genetic distance is a strong predictor of the gut microbiome in large-scale studies and captive settings, its influence has not always been evident at finer taxonomic scales, especially when considering among the recently diverged animals in natural settings. Comparing the gut microbiome of 19 populations of Japanese macaques Macaca fuscata across the Japanese archipelago, we assessed the relative roles of host genetic distance, geographic distance and dietary factors in influencing the macaque gut microbiome. Our results suggested that the macaques may maintain a core gut microbiome, while each population may have acquired some microbes from its specific habitat/diet. Diet-related factors such as season, forest, and reliance on anthropogenic foods played a stronger role in shaping the macaque gut microbiome. Among closely related mammalian hosts, host genetics may have limited effects on the gut microbiome since the hosts generally have smaller physiological differences. This study contributes to our understanding of the relative roles of host phylogeography and dietary factors in shaping the gut microbiome of closely related mammalian hosts.

[Source](https://doi.org/10.1002/ajp.23555)

10.1002/ajp.23555

---

## Skin and gut microbiomes of tadpoles vary differently with host and water environment: a short-term experiment using 16S metabarcoding.
 03.10.2023, _B Santos, FMS Martins, J Sabino-Pinto, F Licata, A Crottini_


_Animals, Gastrointestinal Microbiome, Larva, Water, Microbiota, Bufonidae, RNA, Ribosomal, 16S_
The host-microbiome community is influenced by several host and environmental factors. In order to disentangle the individual effects of host and environment, we performed a laboratory experiment to assess the effects of the exposure to different water sources on the skin and gut microbiome of two amphibian species (Pelophylax perezi and Bufo spinosus). We observed that the bacterial communities greatly varied with water environment and host identity. Tadpoles of B. spinosus collected from a waterbody with poorer bacterial diversity exhibited a more diverse skin and gut microbiome after exposed to a richer water source. Tadpoles of P. perezi, originally collected from a richer water environment, exhibited less marked alterations in diversity patterns independently of the water source but showed alterations in gut composition. These results highlight that environment alterations, such as the water source, combined with the host effect, impact the microbiome of amphibian species in different ways; the population history (e.g., previous water environment and habitat) of the host species may also influence future alterations on tadpole microbiome.

[Source](https://doi.org/10.1038/s41598-023-43340-2)

10.1038/s41598-023-43340-2

---

## Multi-omics revealed the long-term effect of ruminal keystone bacteria and the microbial metabolome on lactation performance in adult dairy goats.
 03.10.2023, _D Wang, L Chen, G Tang, J Yu, J Chen, Z Li, Y Cao, X Lei, L Deng, S Wu, LL Guan, J Yao_


_Humans, Animals, Female, Adolescent, Lactation, Diet, Propionates, Multiomics, Bacteria, Metabolome, Goats, Carbohydrates, Rumen, Fermentation, Animal Feed_
The increased growth rate of young animals can lead to higher lactation performance in adult goats; however, the effects of the ruminal microbiome on the growth of young goats, and the contribution of the early-life rumen microbiome to lifelong growth and lactation performance in goats has not yet been well defined. Hence, this study assessed the rumen microbiome in young goats with different average daily gains (ADG) and evaluated its contribution to growth and lactation performance during the first lactation period.

[Source](https://doi.org/10.1186/s40168-023-01652-5)

10.1186/s40168-023-01652-5

---

## The multi-kingdom microbiome of the goat gastrointestinal tract.
 03.10.2023, _Y Cao, T Feng, Y Wu, Y Xu, L Du, T Wang, Y Luo, Y Wang, Z Li, Z Xuan, S Chen, N Yao, NL Gao, Q Xiao, K Huang, X Wang, K Cui, SU Rehman, X Tang, D Liu, H Han, Y Li, WH Chen, Q Liu_


_Animals, Archaea, Bacteria, Gastrointestinal Tract, Goats, Methane, Microbiota_
Goat is an important livestock worldwide, which plays an indispensable role in human life by providing meat, milk, fiber, and pelts. Despite recent significant advances in microbiome studies, a comprehensive survey on the goat microbiomes covering gastrointestinal tract (GIT) sites, developmental stages, feeding styles, and geographical factors is still unavailable. Here, we surveyed its multi-kingdom microbial communities using 497 samples from ten sites along the goat GIT.

[Source](https://doi.org/10.1186/s40168-023-01651-6)

10.1186/s40168-023-01651-6

---

## Multi-breed host rumen epithelium transcriptome and microbiome associations and their relationship with beef cattle feed efficiency.
 03.10.2023, _PAS Fonseca, S Lam, Y Chen, SM Waters, LL Guan, A Cánovas_


Understanding host-microbial interactions in the rumen and its influence on desirable production traits may lead to potential microbiota manipulation or genetic selection for improved cattle feed efficiency. This study investigated the host transcriptome and its correlation with the rumen archaea and bacteria differential abundance of two pure beef cattle breeds (Angus and Charolais) and one composite beef hybrid (Kinsella) divergent for residual feed intake (RFI; low-RFI vs. high-RFI). Using RNA-Sequencing of rumen tissue and 16S rRNA gene amplicon sequencing, differentially expressed genes (FDR ≤ 0.05, |log2(Fold-change) &gt;|2) and differentially abundant (p-value &lt; 0.05) archaea and bacteria amplicon sequence variants (ASV) were determined. Significant correlations between gene expression and ASVs (p-value &lt; 0.05) were determine using Spearman correlation. Interesting associations with muscle contraction and the modulation of the immune system were observed for the genes correlated with bacterial ASVs. Potential functional candidate genes for feed efficiency status were identified for Angus (CCL17, CCR3, and CXCL10), Charolais (KCNK9, GGT1 and IL6), and Kinsella breed (ESR2). The results obtained here provide more insights regarding the applicability of target host and rumen microbial traits for the selection and breeding of more feed efficient beef cattle.

[Source](https://doi.org/10.1038/s41598-023-43097-8)

10.1038/s41598-023-43097-8

---

## Select symbionts drive high IgA levels in the mouse intestine.
 30.09.2023, _S Zhang, Y Han, W Schofield, M Nicosia, PE Karell, KP Newhall, JY Zhou, RJ Musich, S Pan, A Valujskikh, N Sangwan, M Dwidar, Q Lu, TS Stappenbeck_


Immunoglobulin A (IgA) is an important factor in maintaining homeostasis at mucosal surfaces, yet luminal IgA levels vary widely. Total IgA levels are thought to be driven by individual immune responses to specific microbes. Here, we found that the prebiotic, pectin oligosaccharide (pec-oligo), induced high IgA levels in the small intestine in a T cell-dependent manner. Surprisingly, this IgA-high phenotype was retained after cessation of pec-oligo treatment, and microbiome transmission either horizontally or vertically was sufficient to retain high IgA levels in the absence of pec-oligo. Interestingly, the bacterial taxa enriched in the overall pec-oligo bacterial community differed from IgA-coated microbes in this same community. Rather, a group of ethanol-resistant microbes, highly enriched for Lachnospiraceae bacterium A2, drove the IgA-high phenotype. These findings support a model of intestinal adaptive immunity in which a limited number of microbes can promote durable changes in IgA directed to many symbionts.

[Source](https://www.sciencedirect.com/science/article/abs/pii/S1931312823003682?via%3Dihub)

10.1016/j.chom.2023.09.001

---

## Blocked conversion of <em>Lactobacillus johnsonii</em> derived acetate to butyrate mediates copper-induced epithelial barrier damage in a pig model

[Source](https://doi.org/10.1186/s40168-023-01655-2)

---

## Laboratory mice with a wild microbiota generate strong allergic immune responses.
 02.10.2023, _J Ma, E Urgard, S Runge, CH Classon, L Mathä, JM Stark, L Cheng, JA Álvarez, S von Zedtwitz, A Baleviciute, S Martinez Hoyer, M Li, AM Gernand, L Osbelt, AA Bielecka, TR Lesker, HJ Huang, S Vrtala, L Boon, R Beyaert, M Adner, I Martinez Gonzalez, T Strowig, J Du, S Nylén, SP Rosshart, JM Coquet_


_Mice, Animals, Th2 Cells, Hypersensitivity, Cytokines, Allergens, Immunity_
Allergic disorders are caused by a combination of hereditary and environmental factors. The hygiene hypothesis postulates that early-life microbial exposures impede the development of subsequent allergic disease. Recently developed "wildling" mice are genetically identical to standard laboratory specific pathogen-free (SPF) mice but are housed under seminatural conditions and have rich microbial exposures from birth. Thus, by comparing conventional SPF mice with wildlings, we can uncouple the impact of lifelong microbial exposures from genetic factors on the allergic immune response. We found that wildlings developed larger populations of antigen-experienced T cells than conventional SPF mice, which included interleukin-10-producing CD4 T cells specific for commensal Lactobacilli strains and allergy-promoting T helper 2 (TH2) cells. In models of airway exposure to house dust mite (HDM), recombinant interleukin-33, or Alternaria alternata, wildlings developed strong allergic inflammation, characterized by eosinophil recruitment, goblet cell metaplasia, and antigen-specific immunoglobulin G1 (IgG1) and IgE responses. Wildlings developed robust de novo TH2 cell responses to incoming allergens, whereas preexisting TH2 cells could also be recruited into the allergic immune response in a cytokine-driven and TCR-independent fashion. Thus, wildling mice, which experience diverse and lifelong microbial exposures, were not protected from developing pathological allergic immune responses. Instead, wildlings mounted robust allergic responses to incoming allergens, shedding new light on the hygiene hypothesis.

[Source](https://doi.org/10.1126/sciimmunol.adf7702)

10.1126/sciimmunol.adf7702

---

## Host genetic variation drives the differentiation in the ecological role of the native <em>Miscanthus</em> root-associated microbiome

[Source](https://doi.org/10.1186/s40168-023-01646-3)

---

## Changes in structure and assembly of a species-rich soil natural community with contrasting nutrient availability upon establishment of a plant-beneficial <em>Pseudomonas</em> in the wheat rhizosphere

[Source](https://doi.org/10.1186/s40168-023-01660-5)

---

## Microbially produced fertilizer provides rhizobacteria to hydroponic tomato roots by forming beneficial biofilms.
 29.09.2023, _Y Sato, T Miwa, T Inaba, T Akachi, E Tanaka, T Hori, K Murofushi, H Takagi, H Futamata, T Aoyagi, H Habe_


Hydroponic cultivation of Solanum lycopersicum (tomato) is important, and high tomato production depends on the use of nitrogen and phosphate fertilizers. We had developed a microbial fertilizer (MF), which is mainly composed of nitrate. To investigate the effect of MF on plant growth, hydroponic tomato was grown with MF or commercial inorganic fertilizer (IF), and the microbiomes of the rhizosphere and the liquid phase were analyzed by confocal microscopy and high-throughput sequencing. Plant biomass and biofilm formation were increased by growth in MF compared to IF. The microbial community structures of tomato roots and hydroponic water differed between the two conditions, and three operational taxonomic units (OTUs) dominated in plants grown with MF. The three OTUs were related to Rudaea spp., Chitinophaga spp., and Stenotrophobacter terrae, which are reported to be disease-suppressive epiphytic or endophytic microbes of plant roots. Because these three OTUs also predominated in the MF itself, they were likely provided to the rhizosphere or endophytic environments of tomato roots via hydroponic water. KEY POINTS: • Microbial fertilizer for hydroponic growth enhanced biofilm formation on tomato root. • Microbial fertilizer contains tomato-root epiphytic or endophytic microbes. • Microbial fertilizer provided beneficial microbes to the rhizosphere and endophytic environments of tomato roots via hydroponic water.

[Source](https://doi.org/10.1007/s00253-023-12794-9)

10.1007/s00253-023-12794-9

---

## Terpenes modulate bacterial and fungal growth and sorghum rhizobiome communities.
 29.09.2023, _MY Chou, TB Andersen, ME Mechan Llontop, N Beculheimer, A Sow, N Moreno, A Shade, B Hamberger, G Bonito_


Terpenes are among the oldest and largest class of plant-specialized bioproducts that are known to affect plant development, adaptation, and biological interactions. While their biosynthesis, evolution, and function in aboveground interactions with insects and individual microbial species are well studied, how different terpenes impact plant microbiomes belowground is much less understood. Here we designed an experiment to assess how belowground exogenous applications of monoterpenes (1,8-cineole and linalool) and a sesquiterpene (nerolidol) delivered through an artificial root system impacted its belowground bacterial and fungal microbiome. We found that the terpene applications had significant and variable impacts on bacterial and fungal communities, depending on terpene class and concentration; however, these impacts were localized to the artificial root system and the fungal rhizosphere. We complemented this experiment with pure culture bioassays on responsive bacteria and fungi isolated from the sorghum rhizobiome. Overall, higher concentrations (200 µM) of nerolidol were inhibitory to Ferrovibrium and tested Firmicutes. While fungal isolates of Penicillium and Periconia were also more inhibited by higher concentrations (200 µM) of nerolidol, Clonostachys was enhanced at this higher level and together with Humicola was inhibited by the lower concentration tested (100 µM). On the other hand, 1,8-cineole had an inhibitory effect on Orbilia at both tested concentrations but had a promotive effect at 100 µM on Penicillium and Periconia. Similarly, linalool at 100 µM had significant growth promotion in Mortierella, but an inhibitory effect for Orbilia. Together, these results highlight the variable direct effects of terpenes on single microbial isolates and demonstrate the complexity of microbe-terpene interactions in the rhizobiome. IMPORTANCE Terpenes represent one of the largest and oldest classes of plant-specialized metabolism, but their role in the belowground microbiome is poorly understood. Here, we used a "rhizobox" mesocosm experimental set-up to supply different concentrations and classes of terpenes into the soil compartment with growing sorghum for 1 month to assess how these terpenes affect sorghum bacterial and fungal rhizobiome communities. Changes in bacterial and fungal communities between treatments belowground were characterized, followed by bioassays screening on bacterial and fungal isolates from the sorghum rhizosphere against terpenes to validate direct microbial responses. We found that microbial growth stimulatory and inhibitory effects were localized, terpene specific, dose dependent, and transient in time. This work paves the way for engineering terpene metabolisms in plant microbiomes for improved sustainable agriculture and bioenergy crop production.

[Source](https://doi.org/10.1128/spectrum.01332-23)

10.1128/spectrum.01332-23

---

## Rhizosphere and detritusphere habitats modulate expression of soil N-cycling genes during plant development.
 27.09.2023, _ET Sieradzki, EE Nuccio, J Pett-Ridge, MK Firestone_


Interactions between plant roots and rhizosphere bacteria modulate nitrogen (N)-cycling processes and create habitats rich in low molecular weight compounds (exudates) and complex organic molecules (decaying root litter) compared to those of bulk soil. Microbial N-cycling is regulated by edaphic conditions and genes from many interconnected metabolic pathways, but most studies of soil N-cycling gene expression have focused on single pathways. Currently, we lack a comprehensive understanding of the interplay between soil N-cycling gene regulation, spatial habitat, and time. We present results from a replicated time series of soil metatranscriptomes; we followed gene expression of multiple N transformations in four soil habitats (rhizosphere, detritusphere, rhizo-detritusphere, and bulk soil) during active root growth for the annual grass, Avena fatua. The presence of root litter and living roots significantly altered the trajectories of N-cycling gene expression. Upregulation of assimilatory nitrate reduction in the rhizosphere suggests that rhizosphere bacteria were actively competing with roots for nitrate. Simultaneously, ammonium assimilatory pathways were upregulated in both rhizosphere and detritusphere soil, which could have limited N availability to plants. The detritusphere supported dissimilatory processes DNRA and denitrification. Expression of nitrification genes was dominated by three phylotypes of Thaumarchaeota and was upregulated in bulk soil. Unidirectional ammonium assimilation and its regulatory genes (GS/GOGAT) were upregulated near relatively young roots and highly decayed root litter, suggesting N may have been limiting in these habitats (GS/GOGAT is typically activated under N limitation). Our comprehensive analysis indicates that differences in carbon and inorganic N availability control contemporaneous transcription of N-cycling pathways in soil habitats. IMPORTANCE Plant roots modulate microbial nitrogen (N) cycling by regulating the supply of root-derived carbon and nitrogen uptake. These differences in resource availability cause distinct micro-habitats to develop: soil near living roots, decaying roots, near both, or outside the direct influence of roots. While many environmental factors and genes control the microbial processes involved in the nitrogen cycle, most research has focused on single genes and pathways, neglecting the interactive effects these pathways have on each other. The processes controlled by these pathways determine consumption and production of N by soil microorganisms. We followed the expression of N-cycling genes in four soil microhabitats over a period of active root growth for an annual grass. We found that the presence of root litter and living roots significantly altered gene expression involved in multiple nitrogen pathways, as well as tradeoffs between pathways, which ultimately regulate N availability to plants.

[Source](https://doi.org/10.1128/msystems.00315-23)

10.1128/msystems.00315-23

---

## The presence of benzene ring activating CoA ligases for aromatics degradation in the ANaerobic MEthanotrophic (ANME) archaea.
 27.09.2023, _WW Liu, P Pan, NY Zhou_


Petroleum-source and black carbon-source aromatic compounds are present in the cold seep environments, where ANaerobic MEthanotrophic (ANME) archaea as the dominant microbial community mediates the anaerobic oxidation of methane to produce inorganic and organic carbon. Here, by predicting the aromatics catabolic pathways in ANME metagenome-assembled genomes, we provide genomic and biochemical evidences that ANME have the potential of metabolizing aromatics via the strategy of CoA activation of the benzene ring using phenylacetic acid and benzoate as the substrates. Two ring-activating enzymes phenylacetate-CoA ligase (PaaKANME) and benzoate-CoA ligase (BadAANME) are able to convert phenylacetate to phenylacetyl-CoA and benzoate to benzoyl-CoA in vitro, respectively. They are mesophilic, alkali resistance, and with broad substrate spectra showing different affinity with various substrates. An exploration of the relative gene abundance in ANME genomes and cold seep environments indicates that about 50% of ANME genomes contain PCL genes, and various bacteria and archaea contain PCL and BCL genes. The results provide evidences for the capability of heterotrophic metabolism of aromatic compounds by ANME. This has not only enhanced our understanding of the nutrient range of ANME but also helped to explore the additional ecological and biogeochemical significance of this ubiquitous sedimentary archaea in the carbon flow in the cold seep environments. IMPORTANCE ANaerobic MEthanotrophic (ANME) archaea is the dominant microbial community mediating the anaerobic oxidation of methane in the cold seep environments, where aromatic compounds are present. Then it is hypothesized that ANME may be involved in the metabolism of aromatics. Here, we provide genomic and biochemical evidences for the heterotrophic metabolism of aromatic compounds by ANME, enhancing our understanding of their nutrient range and also shedding light on the ecological and biogeochemical significance of these ubiquitous sedimentary archaea in carbon flow within cold seep environments. Overall, this study offers valuable insights into the metabolic capabilities of ANME and their potential contributions to the global carbon cycle.

[Source](https://doi.org/10.1128/spectrum.01766-23)

10.1128/spectrum.01766-23

---

## Microbial biogeography of pit mud from an artificial brewing ecosystem on a large time scale: all roads lead to Rome.
 28.09.2023, _JL Mei, LJ Chai, XZ Zhong, ZM Lu, XJ Zhang, ST Wang, CH Shen, JS Shi, ZH Xu_


The fermentation process of strong-flavor Baijiu occurs in an underground cellar surrounded by mud, where grains are decomposed and generate new flavor substances. Pit mud microbiome plays a pivotal role in the formation of Baijiu flavor, which is positively correlated with cellar age. Previous research on the pit mud microbiome was often conducted for a specific distillery, making it hard to draw a general conclusion to decipher the pit mud microecosystem in such a unique brewing environment. Hence, in this study, we attempt to unravel community formation mechanisms through research in pit mud microbial biogeography based on 12 independent bacterial studies comprising qualified data sets of 302 samples clearly marked with cellar age from five major producing provinces. We found cellar age explained more variations in the bacterial community than geographical region. The bacterial community displayed evident dominant species alternations on a large time scale. Lactobacillus was found to be the dominant species in young pit mud (&lt;10 years), while Caproiciproducens dominated in aged pit mud. Null model analysis revealed that as pit mud aged, the influence of stochastic and deterministic processes in the bacterial community shifted, and the relative contribution of homogeneous selection increased. Positive correlations dominated the bacterial co-occurrence networks, and the network complexity decreased and stability increased with cellar age. These findings shed new light on the formation of pit mud microbiomes under the impact of long-term batch-to-batch continuous brewing, providing a basis for the evaluation and production of high-quality pit mud. IMPORTANCE Baijiu is a typical example of how humans employ microorganisms to convert grains into new flavors. Mud cellars are used as the fermentation vessel for strong-flavor Baijiu (SFB) to complete the decomposition process of grains. The typical flavor of SFB is mainly attributed to the metabolites of the pit mud microbiome. China has a large number of SFB-producing regions. Previous research revealed the temporal profiles of the pit mud microbiome in different geographical regions. However, each single independent study rarely yields a thorough understanding of the pit mud ecosystem. Will the pit mud microbial communities in different production regions exhibit similar succession patterns and structures under the impact of the brewing environment? Hence, we conducted research in pit mud microbial biogeography to uncover the impact of specific environment on the microbial community over a long time scale.

[Source](https://doi.org/10.1128/msystems.00564-23)

10.1128/msystems.00564-23

---

## Stable microbial community in compacted bentonite after 5 years of exposure to natural granitic groundwater.
 29.09.2023, _K Engel, SE Ford, WJ Binns, N Diomidis, GF Slater, JD Neufeld_


The Materials Corrosion Test (MaCoTe) at the Underground Research Laboratory in Grimsel, Switzerland, assesses the microbiology and corrosion behavior of engineered barrier components of a deep geological repository (DGR) for long-term disposal of high-level nuclear waste. Diversity and temporal changes of bentonite-associated microbial community profiles were assessed under DGR-like conditions for compacted Wyoming MX-80 bentonite (1.25 g/cm3 and 1.50 g/cm3 targeted dry densities) exposed to natural groundwater. Using culture-dependent and molecular techniques, samples taken from the outside layer of 5-year borehole modules revealed up to 66% and 23% of 16S rRNA gene sequences affiliated with Desulfosporosinus and Desulfovibrio, respectively. Putatively involved in sulfate reduction, these taxa were almost undetectable within the bentonite core. Instead, microbial profiles of the inner bentonite core were similar to uncompacted bentonite used to pack modules years earlier, and were consistent with a previously published 1-year time point, revealing no detectable microbial growth. Abundances of culturable aerobic and anaerobic heterotrophic bacteria in the uncompacted bentonite were relatively low, with less than 1,000 and 100 colony-forming units (CFUs) per gram dry weight, respectively. Nearly 5 years after emplacement, culturable heterotrophic bacterial CFUs and sulfate-reducing bacteria did not change significantly inside the bentonite core. Phospholipid fatty acid data indicated similar lipid abundance, and corresponding cell abundance estimates, for inner 5-year MaCoTe bentonite samples compared to those previously obtained for 1-year incubations. Collectively, our results provide complementary evidence for microbial stability inside highly compacted bentonite exposed to conditions that mimic engineered barrier components of a deep geological repository. IMPORTANCE The long-term safety of a deep geological repository for used nuclear fuel is dependent on the performance of the engineered and natural barriers. Microbial activity can produce chemical species that can influence the corrosion of the disposal containers for used nuclear fuel. Although previous studies have evaluated the microbiology of compacted bentonite clay within subsurface environments, these have been limited to relatively short incubations (i.e., 1 year). The current study provides a unique 5-year perspective that reinforces previous findings of growth inhibition for bentonite clay exposed to in situ subsurface conditions.

[Source](https://doi.org/10.1128/msphere.00048-23)

10.1128/msphere.00048-23

---

## Impact of Experimental Bias on Compositional Analysis of Microbiome Data.
 03.10.2023, _Y Hu, GA Satten, YJ Hu_


_Bias, Computer Simulation, Microbiota, Polymerase Chain Reaction_
Microbiome data are subject to experimental bias that is caused by DNA extraction and PCR amplification, among other sources, but this important feature is often ignored when developing statistical methods for analyzing microbiome data. McLaren, Willis, and Callahan (2019) proposed a model for how such biases affect the observed taxonomic profiles; this model assumes the main effects of bias without taxon-taxon interactions. Our newly developed method for testing the differential abundance of taxa, LOCOM, is the first method to account for experimental bias and is robust to the main effect biases. However, there is also evidence for taxon-taxon interactions. In this report, we formulated a model for interaction biases and used simulations based on this model to evaluate the impact of interaction biases on the performance of LOCOM as well as other available compositional analysis methods. Our simulation results indicate that LOCOM remained robust to a reasonable range of interaction biases. The other methods tend to have an inflated FDR even when there were only main effect biases. LOCOM maintained the highest sensitivity even when the other methods could not control the FDR. We thus conclude that LOCOM outperforms the other methods for compositional analysis of microbiome data considered here.

[Source](https://doi.org/10.3390/genes14091777)

10.3390/genes14091777

---

## happi: a hierarchical approach to pangenomics inference

[Source](https://doi.org/10.1186/s13059-023-03040-6)

---

## Integrating a tailored recurrent neural network with Bayesian experimental design to optimize microbial community functions.
 02.10.2023, _JC Thompson, VM Zavala, OS Venturelli_


Microbiomes interact dynamically with their environment to perform exploitable functions such as production of valuable metabolites and degradation of toxic metabolites for a wide range of applications in human health, agriculture, and environmental cleanup. Developing computational models to predict the key bacterial species and environmental factors to build and optimize such functions are crucial to accelerate microbial community engineering. However, there is an unknown web of interactions that determine the highly complex and dynamic behavior of these systems, which precludes the development of models based on known mechanisms. By contrast, entirely data-driven machine learning models can produce physically unrealistic predictions and often require significant amounts of experimental data to learn system behavior. We develop a physically-constrained recurrent neural network that preserves model flexibility but is constrained to produce physically consistent predictions and show that it can outperform existing machine learning methods in the prediction of certain experimentally measured species abundance and metabolite concentrations. Further, we present a closed-loop, Bayesian experimental design algorithm to guide data collection by selecting experimental conditions that simultaneously maximize information gain and target microbial community functions. Using a bioreactor case study, we demonstrate how the proposed framework can be used to efficiently navigate a large design space to identify optimal operating conditions. The proposed methodology offers a flexible machine learning approach specifically tailored to optimize microbiome target functions through the sequential design of informative experiments that seek to explore and exploit community functions.

[Source](https://doi.org/10.1371/journal.pcbi.1011436)

10.1371/journal.pcbi.1011436

---

## Uncultured prokaryotic genomes in the spotlight: An examination of publicly available data from metagenomics and single-cell genomics

[Source](https://www.csbj.org/article/S2001-0370(23)00322-7/fulltext)

---

