## Microbiome diversity protects against pathogens by nutrient blocking.
 21.12.2023, _F Spragge, E Bakkeren, MT Jahn, E B N Araujo, CF Pearson, X Wang, L Pankhurst, O Cunrath, KR Foster_


_Animals, Humans, Mice, Gastrointestinal Microbiome, Nutrients, Host-Pathogen Interactions, Klebsiella pneumoniae, Salmonella typhimurium, Symbiosis, Germ-Free Life, Klebsiella Infections, Salmonella Infections, Escherichia coli_
The human gut microbiome plays an important role in resisting colonization of the host by pathogens, but we lack the ability to predict which communities will be protective. We studied how human gut bacteria influence colonization of two major bacterial pathogens, both in vitro and in gnotobiotic mice. Whereas single species alone had negligible effects, colonization resistance greatly increased with community diversity. Moreover, this community-level resistance rested critically upon certain species being present. We explained these ecological patterns through the collective ability of resistant communities to consume nutrients that overlap with those used by the pathogen. Furthermore, we applied our findings to successfully predict communities that resist a novel target strain. Our work provides a reason why microbiome diversity is beneficial and suggests a route for the rational design of pathogen-resistant communities.

[Source](https://doi.org/10.1126/science.adj3502)

10.1126/science.adj3502

---

## Shotgun metagenomics reveals interkingdom association between intestinal bacteria and fungi involving competition for nutrients.
 17.12.2023, _Z Xie, A Canalda-Baltrons, C d'Enfert, C Manichanh_


_Humans, Fungi, Microbiota, Mycobiome, Bacteria, Metagenomics, Nutrients_
The accuracy of internal-transcribed-spacer (ITS) and shotgun metagenomics has not been robustly evaluated, and the effect of diet on the composition and function of the bacterial and fungal gut microbiome in a longitudinal setting has been poorly investigated. Here we compared two approaches to study the fungal community (ITS and shotgun metagenomics), proposed an enrichment protocol to perform a reliable mycobiome analysis using a comprehensive in-house fungal database, and correlated dietary data with both bacterial and fungal communities.

[Source](https://doi.org/10.1186/s40168-023-01693-w)

10.1186/s40168-023-01693-w

---

## Functional alterations and predictive capacity of gut microbiome in type 2 diabetes.
 19.12.2023, _NR Dash, MT Al Bataineh, R Alili, H Al Safar, N Alkhayyal, E Prifti, JD Zucker, E Belda, K Clément_


_Humans, Diabetes Mellitus, Type 2, Gastrointestinal Microbiome, Firmicutes, Metagenome, Diabetic Cardiomyopathies_
The gut microbiome plays a significant role in the development of Type 2 Diabetes Mellitus (T2DM), but the functional mechanisms behind this association merit deeper investigation. Here, we used the nanopore sequencing technology for metagenomic analyses to compare the gut microbiome of individuals with T2DM from the United Arab Emirates (n = 40) with that of control (n = 44). DMM enterotyping of the cohort resulted concordantly with previous results, in three dominant groups Bacteroides (K1), Firmicutes (K2), and Prevotella (K3) lineages. The diversity analysis revealed a high level of diversity in the Firmicutes group (K2) both in terms of species richness and evenness (Wilcoxon rank-sum test, p value &lt; 0.05 vs. K1 and K3 groups), consistent with the Ruminococcus enterotype described in Western populations. Additionally, functional enrichment analyses of KEGG modules showed significant differences in abundance between individuals with T2DM and controls (FDR &lt; 0.05). These differences include modules associated with the degradation of amino acids, such as arginine, the degradation of urea as well as those associated with homoacetogenesis. Prediction analysis with the Predomics approach suggested potential biomarkers for T2DM, including a balance between a depletion of Enterococcus faecium and Blautia lineages with an enrichment of Absiella spp or Eubacterium limosum in T2DM individuals, highlighting the potential of metagenomic analysis in predicting predisposition to diabetic cardiomyopathy in T2DM patients.

[Source](https://doi.org/10.1038/s41598-023-49679-w)

10.1038/s41598-023-49679-w

---

## Fecal metabolite profiling identifies liver transplant recipients at risk for postoperative infection.
 16.12.2023, _CJ Lehmann, NP Dylla, M Odenwald, R Nayak, M Khalid, J Boissiere, J Cantoral, E Adler, MR Stutz, M Dela Cruz, A Moran, H Lin, R Ramaswamy, A Sundararajan, AM Sidebottom, J Little, EG Pamer, A Aronsohn, J Fung, TB Baker, A Kacha_


Metabolites produced by the intestinal microbiome modulate mucosal immune defenses and optimize epithelial barrier function. Intestinal dysbiosis, including loss of intestinal microbiome diversity and expansion of antibiotic-resistant pathobionts, is accompanied by changes in fecal metabolite concentrations and increased incidence of systemic infection. Laboratory tests that quantify intestinal dysbiosis, however, have yet to be incorporated into clinical practice. We quantified fecal metabolites in 107 patients undergoing liver transplantation (LT) and correlated these with fecal microbiome compositions, pathobiont expansion, and postoperative infections. Consistent with experimental studies implicating microbiome-derived metabolites with host-mediated antimicrobial defenses, reduced fecal concentrations of short- and branched-chain fatty acids, secondary bile acids, and tryptophan metabolites correlate with compositional microbiome dysbiosis in LT patients and the relative risk of postoperative infection. Our findings demonstrate that fecal metabolite profiling can identify LT patients at increased risk of postoperative infection and may provide guideposts for microbiome-targeted therapies.

[Source](https://doi.org/10.1016/j.chom.2023.11.016)

10.1016/j.chom.2023.11.016

---

## Establishment of oral microbiome in very low birth weight infants during the first weeks of life and the impact of oral diet implementation.
 18.12.2023, _PAR Vanzele, LG Sparvoli, PP de Camargo, CR Tragante, GPNS Beozzo, VLJ Krebs, RV Cortez, CR Taddei_


_Infant, Humans, Infant, Newborn, Milk, Human, RNA, Ribosomal, 16S, Infant, Very Low Birth Weight, Microbiota, Diet_
Very low birth weight (VLBW) infants, mostly preterm, have many barriers to feeding directly from the mother's breast, and need to be fed alternatively. Feeding is a major influencer in oral microbial colonization, and this colonization in early life is crucial for the promotion of human health. Therefore, this research aimed to observe the establishment of oral microbiome in VLBW infants during their first month of life through hospitalization, and to verify the impact caused by the implementation of oral diet on the colonization of these newborns. We included 23 newborns followed during hospitalization and analyzed saliva samples collected weekly, using 16S rRNA gene sequencing. We observed a significant decrease in richness and diversity and an increase in dominance over time (q-value &lt; 0.05). The oral microbiome is highly dynamic during the first weeks of life, and beta diversity suggests a microbial succession in early life. The introduction of oral diet does not change the community structure, but affects the abundance, especially of Streptococcus. Our results indicate that although time is related to significant changes in the oral microbial profile, oral feeding benefits genera that will remain colonizers throughout the host's life.

[Source](https://doi.org/10.1371/journal.pone.0295962)

10.1371/journal.pone.0295962

---

## Infant gut DNA bacteriophage strain persistence during the first 3 years of life.
 14.12.2023, _YC Lou, L Chen, AL Borges, J West-Roberts, BA Firek, MJ Morowitz, JF Banfield_


Bacteriophages are key components of gut microbiomes, yet the phage colonization process in the infant gut remains uncertain. Here, we establish a large phage sequence database and use strain-resolved analyses to investigate DNA phage succession in infants throughout the first 3 years of life. Analysis of 819 fecal metagenomes collected from 28 full-term and 24 preterm infants and their mothers revealed that early-life phageome richness increases over time and reaches adult-like complexity by age 3. Approximately 9% of early phage colonizers, which are mostly maternally transmitted and infect Bacteroides, persist for 3 years and are more prevalent in full-term than in preterm infants. Although rare, phages with stop codon reassignment are more likely to persist than non-recoded phages and generally display an increase in in-frame reassigned stop codons over 3 years. Overall, maternal seeding, stop codon reassignment, host CRISPR-Cas locus prevalence, and diverse phage populations contribute to stable viral colonization.

[Source](https://doi.org/10.1016/j.chom.2023.11.015)

10.1016/j.chom.2023.11.015

---

## Metabolic diversity in commensal protists regulates intestinal immunity and trans-kingdom competition.
 14.12.2023, _ER Gerrick, S Zlitni, PT West, MM Carter, CM Mechler, MR Olm, EB Caffrey, JA Li, SK Higginbottom, CJ Severyn, F Kracke, AM Spormann, JL Sonnenburg, AS Bhatt, MR Howitt_


The microbiota influences intestinal health and physiology, yet the contributions of commensal protists to the gut environment have been largely overlooked. Here, we discover human- and rodent-associated parabasalid protists, revealing substantial diversity and prevalence in nonindustrialized human populations. Genomic and metabolomic analyses of murine parabasalids from the genus Tritrichomonas revealed species-level differences in excretion of the metabolite succinate, which results in distinct small intestinal immune responses. Metabolic differences between Tritrichomonas species also determine their ecological niche within the microbiota. By manipulating dietary fibers and developing in vitro protist culture, we show that different Tritrichomonas species prefer dietary polysaccharides or mucus glycans. These polysaccharide preferences drive trans-kingdom competition with specific commensal bacteria, which affects intestinal immunity in a diet-dependent manner. Our findings reveal unappreciated diversity in commensal parabasalids, elucidate differences in commensal protist metabolism, and suggest how dietary interventions could regulate their impact on gut health.

[Source](https://www.cell.com/cell/fulltext/S0092-8674(23)01271-0)

10.1016/j.cell.2023.11.018

---

## Poly-omic risk scores predict inflammatory bowel disease diagnosis.
 14.12.2023, _CH Arehart, JD Sterrett, RL Garris, RE Quispe-Pilco, CR Gignoux, LM Evans, MA Stanislawski_


Complex traits are characterized by many biological and environmental factors, such that multi-omic data sets are well-positioned to help us understand their underlying etiologies. We applied a prediction framework across multiple omics (metagenomics, metatranscriptomics, metabolomics, and viromics) from the gut ecosystem to predict inflammatory bowel disease (IBD) diagnosis. The predicted scores from our models highlighted key features and allowed us to compare the relative utility of each omic data set in single-omic versus multi-omic models. Our results emphasized the importance of metabolomics and viromics over metagenomics and metatranscriptomics for predicting IBD status. The greater predictive capability of metabolomics and viromics is likely because these omics serve as markers of lifestyle factors such as diet. This study provides a modeling framework for multi-omic data, and our results show the utility of combining multiple omic data types to disentangle complex disease etiologies and biological signatures.

[Source](https://doi.org/10.1128/msystems.00677-23)

10.1128/msystems.00677-23

---

## Impact of glyphosate (Roundup(<sup>TM</sup>)) on the composition and functionality of the gut microbiome

[Source](https://doi.org/10.1080/19490976.2023.2263935)

---

## Rapid turnover and short-term blooms of <em>Escherichia coli</em> in the human gut

[Source](https://doi.org/10.1128/jb.00239-23)

---

## Designing a multi-epitope vaccine against <em>Peptostreptococcus anaerobius</em> based on an immunoinformatics approach

[Source](https://doi.org/10.1016/j.synbio.2023.11.004)

---

## Twnbiome: a public database of the healthy Taiwanese gut microbiome

[Source](https://doi.org/10.1186/s12859-023-05585-6)

---

## Identification of inulin-responsive bacteria in the gut microbiota via multi-modal activity-based sorting

[Source](https://doi.org/10.1038/s41467-023-43448-z)

---

## Application of omics technologies in cariology research: A critical review with bibliometric analysis

[Source](https://www.sciencedirect.com/science/article/pii/S0300571223003871?via%3Dihub)

---

## Dysbiosis and associated stool features improve prediction of response to biological therapy in inflammatory bowel disease

[Source](https://www.gastrojournal.org/article/S0016-5085(23)05598-1/pdf)

---

## Culturomics in Unraveling the Upper Female Reproductive Tract Microbiota.
 15.12.2023, _R Vanstokstraeten, T Demuyser, D Piérard, I Wybo, C Blockeel, S Mackens_


In recent years, the study of the human microbiome has surged, shedding light on potential connections between microbiome composition and various diseases. One specific area of intense interest within this research is the female reproductive tract, as it holds the potential to influence the process of embryo implantation. Advanced sequencing technologies have delivered unprecedented insights into the microbial communities, also known as microbiota, residing in the female reproductive tract. However, their efficacy encounters significant challenges when analyzing low-biomass microbiota, such as those present in the endometrium. These molecular techniques are susceptible to contamination from laboratory reagents and extraction kits, leading to sequencing bias that can significantly alter the perceived taxonomy of a sample. Consequently, investigating the microbiota of the upper female reproductive tract necessitates the exploration of alternative methods. In this context, the current review delves into the application of culturomics in unraveling the upper female reproductive tract microbiota. While culturomics holds value in research, its transition to routine clinical practice appears remote, at least in the foreseeable future.

[Source](https://doi.org/10.1055/s-0043-1777758)

10.1055/s-0043-1777758

---

## A manifold-based framework for studying the dynamics of the vaginal microbiome.
 19.12.2023, _M Tsamir-Rimon, E Borenstein_


_Humans, Female, Vagina, Vaginosis, Bacterial, Lactobacillus, Microbiota_
The vaginal microbiome plays a crucial role in our health. The composition of this community can be classified into five community state types (CSTs), four of which are primarily consisted of Lactobacillus species and considered healthy, while the fifth features non-Lactobacillus populations and signifies a disease state termed Bacterial vaginosis (BV), which is associated with various symptoms and increased susceptibility to diseases. Importantly, however, the exact mechanisms and dynamics underlying BV development are not yet fully understood, including specifically possible routes from a healthy to a BV state. To address this gap, this study set out to characterize the progression from healthy- to BV-associated compositions by analyzing 8026 vaginal samples and using a manifold-detection framework. This approach, inspired by single-cell analysis, aims to identify low-dimensional trajectories in the high-dimensional composition space. It further orders samples along these trajectories and assigns a score (pseudo-time) to each analyzed or new sample based on its proximity to the BV state. Our results reveal distinct routes of progression between healthy and BV states for each CST, with pseudo-time scores correlating with community diversity and quantifying the health state of each sample. Several BV indicators can also be successfully predicted based on pseudo-time scores, and key taxa involved in BV development can be identified using this approach. Taken together, these findings demonstrate how manifold detection can be used to successfully characterize the progression from healthy Lactobacillus-dominant populations to BV and to accurately quantify the health condition of new samples along the route of BV development.

[Source](https://doi.org/10.1038/s41522-023-00471-8)

10.1038/s41522-023-00471-8

---

## Culturomics reveals a hidden world of vaginal microbiota with the isolation of 206 bacteria from a single vaginal sample.
 17.12.2023, _L Abou Chacra, A Benatmane, R Iwaza, C Ly, S Alibar, N Armstrong, O Mediannikov, F Bretelle, F Fenollar_


_Humans, Female, Bacteria, Microbiota, Firmicutes, Vagina, Bacteroidetes_
The composition of the vaginal microbiota is known to be influenced by various factors and to be associated with several disorders affecting women's health. Although metagenomics is currently a widely used method for studying the human microbiota, it has certain limitations, such as a lack of information on bacterial viability. It is therefore important to use culture-based methods such as culturomics. Here, we used 35 different culture conditions to comprehensively characterize the vaginal bacterial diversity of a single woman's flora. A total of 206 bacterial species, belonging to six phyla (for a little more than half to Firmicutes, followed mainly by Actinobacteria, Bacteroidetes, and Proteobacteria) and 45 families, and 2 fungal species were cultivated. While several species of lactobacilli have been isolated, a wide variety of other bacteria were also separated, including 65 never reported before in vaginal flora, including a new bacterial species, Porphyromonas vaginalis sp. nov. Extensive culture-based methods are essential to establish a comprehensive, evidence-based repertoire of bacterial viability. If combined with molecular methods, they can provide a much more thorough understanding of the vaginal microbiota and fulfil the unknown part of metagenomic studies.

[Source](https://doi.org/10.1007/s00203-023-03742-2)

10.1007/s00203-023-03742-2

---

## Effects of variation in sample storage conditions and swab order on 16S vaginal microbiome analyses.
 14.12.2023, _T Kumar, M Bryant, K Cantrell, SJ Song, D McDonald, HM Tubb, S Farmer, A Lewis, ES Lukacz, L Brubaker, R Knight_


The composition of the human vaginal microbiome has been linked to a variety of medical conditions including yeast infection, bacterial vaginosis, and sexually transmitted infection. The vaginal microbiome is becoming increasingly acknowledged as a key factor in personal health, and it is essential to establish methods to collect and process accurate samples with self-collection techniques to allow large, population-based studies. In this study, we investigate if using AssayAssure Genelock, a nucleic acid preservative, introduces microbial biases in self-collected vaginal samples. To our knowledge, we also contribute some of the first evidence regarding the impacts of multiple swabs taken at one time point. Vaginal samples have relatively low biomass, so the ability to collect multiple swabs from a unique participant at a single time would greatly improve the replicability and data available for future studies. This will hopefully lay the groundwork to gain a more complete and accurate understanding of the vaginal microbiome.

[Source](https://doi.org/10.1128/spectrum.03712-23)

10.1128/spectrum.03712-23

---

## Activity budget and gut microbiota stability and flexibility across reproductive states in wild capuchin monkeys in a seasonal tropical dry forest

[Source](https://doi.org/10.1186/s42523-023-00280-6)

---

## Validation of method for faecal sampling in cats and dogs for faecal microbiome analysis.
 18.12.2023, _X Langon_


_Cats, Animals, Dogs, Cat Diseases, Dog Diseases, Feces, Microbiota, Bacteria, DNA, Bacterial_
Reproducible and reliable studies of cat and dog faecal microbiomes are dependent on many methodology-based variables including how the faecal stools are sampled and stored prior to processing. The current study aimed to establish an appropriate method for sampling and storing faecal stools from cats and dogs which may also be applied to privately-owned pets. The approach investigated the effects of storing faeces for up to 12 h at room temperature and sampling from various locations within the stool in terms of microbial diversity, relative taxa abundances and DNA yield. Faeces were collected from 10 healthy cats and 10 healthy dogs and stored at room temperature (20 °C). Samples were taken from various locations within the stool (the first emitted part (i), the middle (ii) and the last emitted end (iii), at either surface or core) at 0, 0.5, 1, 2, 3, 6 and 12 h, stabilised and stored at -80 °C. DNA was extracted from all samples, using Illumina NovaSeq.

[Source](https://doi.org/10.1186/s12917-023-03842-7)

10.1186/s12917-023-03842-7

---

## Transmission of antimicrobial resistance in the gut microbiome of gregarious cockroaches: the importance of interaction between antibiotic exposed and non-exposed populations.
 14.12.2023, _A Bogri, EEB Jensen, AV Borchert, C Brinch, S Otani, FM Aarestrup_


Antimicrobial resistance is a rising threat to human and animal health. The spread of resistance through the transmission of the symbiotic gut microbiome is of concern and has been explored in theoretical modeling studies. In this study, we employ gregarious insect populations to examine the emergence and transmission of antimicrobial resistance in vivo and validate modeling hypotheses. We find that antimicrobial treatment increases the levels of resistance in treated populations. Most importantly, we show that resistance increased in untreated populations after interacting with the treated ones. The level of resistance transmission was affected by the magnitude and frequency of population mixing. Our results highlight the importance of microbial transmission in the spread of antimicrobial resistance.

[Source](https://doi.org/10.1128/msystems.01018-23)

10.1128/msystems.01018-23

---

## Fungal community composition changes and reduced bacterial diversity drive improvements in the soil quality index during arable land restoration.
 17.12.2023, _P Wan, Z Zhou, Z Yuan, H Wei, F Huang, Z Li, FM Li, F Zhang_


Arable land is facing the growing challenge of land degradation due to intensive use and this is beginning to affect global food security. However, active and passive restoration can improve soil characteristics and reshape microbial communities. Despite the increasing focus on changes in microbial communities during restoration, the mechanisms underlying how microbes drive the soil quality index (SQI) in arable land restoration remain unclear. In this study, we selected conventional farmland (CF, heavily intensified) and two restoration strategies (AR, artificial restoration; NR, natural restoration), with the same context (including soil texture, climate, etc.), and measured the microbial indicators over 2 years to investigate the mechanisms driving SQI improvement on restored arable land. The AR and NR treatments resulted in a 50% and 58% increase in SQI, respectively, compared to CF as soil nutrient levels increased, resulting in higher microbial biomasses and enzyme activities. Microbial abundance on the AR land was approximately two times greater than on the NR land due to the introduction of legumes. Bacterial diversity declined, while fungi developed in a more diverse direction under the restoration strategies. The AR and NR areas were mainly enriched with rhizobium (Microvirga, Bradyrhizobium), which contribute to healthy plant growth. The pathogenic fungi (Gibberella, Fusarium, Volutella) were more abundant in the CF area and the plant pathogen guild was about five times higher in the restored areas. Following arable land restoration, microbial life history strategies shifted from r-to K-strategists due to the higher proportion of recalcitrant SOC (DOC/SOC decreased by 18%-30%). The altered microbial community in the restored areas created new levels of functionality, with a 2.6%-4.3% decrease in bacterial energy metabolism (oxidative phosphorylation, C fixation, and N metabolism decreased by 7%, 4%, and 6%, respectively). Structural equation modelling suggested that restoration strategy affected SQI either directly by increasing total soil nutrient levels or indirectly by altering the microbial community and that fungal community composition and bacterial diversity made the largest contributions to SQI. These results provided new insights into soil quality improvement from a microbial perspective and can help guide future arable land restoration.

[Source](https://doi.org/10.1016/j.envres.2023.117931)

10.1016/j.envres.2023.117931

---

## Two-for-one: root microbiota orchestrates both soil pH and plant nutrition.
 15.12.2023, _JA Siqueira, DG Gouveia, A Nunes-Nesi, WL Araújo_


Aluminum (Al) toxicity is a crucial limiting factor for crop growth in acid soils. Recently, Liu et al. demonstrated that the root microbiota of rice modulates the responses to Al toxicity and phosphorus limitation, offering intriguing insights into microbiome function and opening new research opportunities.

[Source](https://doi.org/10.1016/j.tplants.2023.11.026)

10.1016/j.tplants.2023.11.026

---

## Cotton microbiome profiling and Cotton Leaf Curl Disease (CLCuD) suppression through microbial consortia associated with <em>Gossypium arboreum</em>

[Source](https://doi.org/10.1038/s41522-023-00470-9)

---

## Benthic microbial biogeographic trends in the North Sea are shaped by an interplay of environmental drivers and bottom trawling effort.
 18.12.2023, _G Bonthond, J Beermann, L Gutow, A Neumann, FR Barboza, A Desiderato, V Fofonova, SB Helber, S Khodami, C Kraan, H Neumann, S Rohde, PJ Schupp_


Microbial composition and diversity in marine sediments are shaped by environmental, biological, and anthropogenic processes operating at different scales. However, our understanding of benthic microbial biogeography remains limited. Here, we used 16S rDNA amplicon sequencing to characterize benthic microbiota in the North Sea from the top centimeter of 339 sediment samples. We utilized spatially explicit statistical models, to disentangle the effects of the different predictors, including bottom trawling intensity, a prevalent industrial fishing practice which heavily impacts benthic ecosystems. Fitted models demonstrate how the geographic interplay of different environmental and anthropogenic drivers shapes the diversity, structure and potential metabolism of benthic microbial communities. Sediment properties were the primary determinants, with diversity increasing with sediment permeability but also with mud content, highlighting different underlying processes. Additionally, diversity and structure varied with total organic matter content, temperature, bottom shear stress and bottom trawling. Changes in diversity associated with bottom trawling intensity were accompanied by shifts in predicted energy metabolism. Specifically, with increasing trawling intensity, we observed a transition toward more aerobic heterotrophic and less denitrifying predicted metabolism. Our findings provide first insights into benthic microbial biogeographic patterns on a large spatial scale and illustrate how anthropogenic activity such as bottom trawling may influence the distribution and abundances of microbes and potential metabolism at macroecological scales.

[Source](https://doi.org/10.1038/s43705-023-00336-3)

10.1038/s43705-023-00336-3

---

## Increased prokaryotic diversity in the Red Sea deep scattering layer.
 17.12.2023, _TM Huete-Stauffer, R Logares, MI Ansari, A Røstad, ML Calleja, XAG Morán_


The diel vertical migration (DVM) of fish provides an active transport of labile dissolved organic matter (DOM) to the deep ocean, fueling the metabolism of heterotrophic bacteria and archaea. We studied the impact of DVM on the mesopelagic prokaryotic diversity of the Red Sea focusing on the mesopelagic deep scattering layer (DSL) between 450-600 m.

[Source](https://doi.org/10.1186/s40793-023-00542-5)

10.1186/s40793-023-00542-5

---

