## A compendium of 32,277 metagenome-assembled genomes and over 80 million genes from the early-life human gut microbiome.
 04.09.2022, _S Zeng, D Patangia, A Almeida, Z Zhou, D Mu, R Paul Ross, C Stanton, S Wang_


Age-specific reference genomes of the human gut microbiome can provide higher resolution for metagenomic analyses including taxonomic classification, strain-level genomic investigation and functional characterization. We present the Early-Life Gut Genomes (ELGG) catalog with 32,277 genomes representing 2172 species from 6122 fecal metagenomes collected from children under 3 years old spanning delivery mode, gestational age, feeding pattern, and geography. The ELGG substantially expanded the phylogenetic diversity by 38% over the isolate microbial genomes, and the genomic landscape of the early-life microbiome by increasing recruitment of metagenomic reads to 82.8%. More than 60% of the ELGG species lack an isolate representative. The conspecific genomes of the most abundant species from children differed in gene diversity and functions compared to adults. The ELGG genomes encode over 80 million protein sequences, forming the Early-Life Gut Proteins (ELGP) catalog with over four million protein clusters, 29.5% of which lacked functional annotations. The ELGG and ELGP references provided new insights into the early-life human gut microbiome and will facilitate studies to understand the development and mechanisms of disturbances of the human gut microbiome in early life.

[Source](https://doi.org/10.1038/s41467-022-32805-z)

10.1038/s41467-022-32805-z

---

## A previously uncharacterized O-glycopeptidase from Akkermansia muciniphila requires the Tn-antigen for cleavage of the peptide bond.
 01.09.2022, _BJ Medley, L Leclaire, N Thompson, KE Mahoney, B Pluvinage, MAH Parson, JE Burke, S Malaker, W Wakarchuk, AB Boraston_


Akkermansia muciniphila is key member of the human gut microbiota, impacting many features of host health. A major characteristic of this bacterium is its interaction with host mucin, which is abundant in the gut environment, and its ability to metabolize mucin as a nutrient source. The machinery deployed by A. muciniphila to enable this interaction appears to be extensive and sophisticated, yet it is incompletely defined. The uncharacterized protein AMUC_1438 is encoded by a gene that was previously shown to be upregulated when the bacterium is grown on mucin. This uncharacterized protein has features suggestive of carbohydrate-recognition and peptidase activity, which led us to hypothesize that it has a role in mucin depolymerization. Here we provide structural and functional support for the assignment of AMUC_1438 as a unique O-glycopeptidase with mucin degrading capacity. O-glycopeptidase enzymes recognize glycans but hydrolyze the peptide backbone and are common in host-adapted microbes that colonize or invade mucus layers. Structural, kinetic, and mutagenic analyses point to a metzincin metalloprotease catalytic motif but specific recognition of a GalNAc residue α-linked to serine or threonine (i.e. the Tn-antigen) within the AMUC_1438 active site. The enzyme catalyzes hydrolysis of the bond immediately N-terminal to the glycosylated residue. Additional modelling analyses suggest the presence of a carbohydrate-binding module that may assist in substrate recognition. We anticipate that these results will be fundamental to a wider understanding of the O-glycopeptidase class of enzymes and how they may contribute to host-adaptation.

[Source](https://doi.org/10.1016/j.jbc.2022.102439)

10.1016/j.jbc.2022.102439

---

## A previously uncharacterized O-glycopeptidase from <em>Akkermansia muciniphila </em>requires the Tn-antigen for cleavage of the peptide bond

[Source](https://doi.org/10.1016/j.jbc.2022.102439)

---

## Lactic acid from vaginal microbiota enhances cervicovaginal epithelial barrier integrity by promoting tight junction protein expression.
 04.09.2022, _DJ Delgado-Diaz, B Jesaveluk, JA Hayward, D Tyssen, A Alisoltani, M Potgieter, L Bell, E Ross, A Iranzadeh, I Allali, S Dabee, S Barnabas, H Gamieldien, JM Blackburn, N Mulder, SB Smith, VL Edwards, AD Burgener, LG Bekker, J Ravel, JS Passmore, L Masson, AC Hearps, G Tachedjian_


Women with a cervicovaginal microbiota dominated by Lactobacillus spp. are at reduced risk of acquiring sexually transmitted infections including HIV, but the biological mechanisms involved remain poorly defined. Here, we performed metaproteomics on vaginal swab samples from young South African women (n = 113) and transcriptomics analysis of cervicovaginal epithelial cell cultures to examine the ability of lactic acid, a metabolite produced by cervicovaginal lactobacilli, to modulate genital epithelial barrier function.

[Source](https://doi.org/10.1186/s40168-022-01337-5)

10.1186/s40168-022-01337-5

---

## A Perspective of the Urinary Microbiome in Lower Urinary Tract Infections - A Review.
 02.09.2022, _R Pallares-Mendez, DE Cervantes-Miranda, AD Gonzalez-Colmenero, MA Ochoa-Arvizo, A Gutierrez-Gonzalez_


Greater availability of sequencing methods has broadened the knowledge of the urinary microbiome in an environment previously believed to be sterile. This review discusses internal and external influences that promote either a balance or a dysbiosis of the urinary tract and the future perspectives in understanding lower urinary tract infections.

[Source](https://doi.org/10.1007/s11934-022-01108-7)

10.1007/s11934-022-01108-7

---

## Methanogenic archaea in the human gastrointestinal tract.
 01.09.2022, _C Hoegenauer, HF Hammer, A Mahnert, C Moissl-Eichinger_


The human microbiome is strongly interwoven with human health and disease. Besides bacteria, viruses and eukaryotes, numerous archaea are located in the human gastrointestinal tract and are responsible for methane production, which can be measured in clinical methane breath analyses. Methane is an important readout for various diseases, including intestinal methanogen overgrowth. Notably, the archaea responsible for methane production are largely overlooked in human microbiome studies due to their non-bacterial biology and resulting detection issues. As such, their importance for health and disease remains largely unclear to date, in particular as not a single archaeal representative has been deemed to be pathogenic. In this Perspective, we discuss the current knowledge on the clinical relevance of methanogenic archaea. We explain the archaeal unique response to antibiotics and their negative and positive effects on human physiology, and present the current understanding of the use of methane as a diagnostic marker.

[Source](https://doi.org/10.1038/s41575-022-00673-z)

10.1038/s41575-022-00673-z

---

## Meta-analysis of caries microbiome studies can improve upon disease prediction outcomes.
 02.09.2022, _MC Butcher, B Short, CLR Veena, D Bradshaw, JR Pratten, W McLean, SMA Shaban, G Ramage, C Delaney_


As one of the most prevalent infective diseases worldwide, it is crucial that we not only know the constituents of the oral microbiome in dental caries but also understand its functionality.

[Source](https://doi.org/10.1111/apm.13272)

10.1111/apm.13272

---

## Metagenomes of rectal swabs in larger, advanced stage cervical cancers have enhanced mucus degrading functionalities and distinct taxonomic structure.
 01.09.2022, _TV Karpinets, X Wu, T Solley, MB El Alam, TT Sims, K Yoshida-Court, E Lynn, M Ahmed-Kaddar, G Biegert, J Yue, X Song, H Sun, JF Petrosino, MP Mezzari, P Okhuysen, PJ Eifel, A Jhingran, LL Lin, KM Schmeler, L Ramondetta, N Ajami, RR Jenq, A Futreal, J Zhang, AH Klopp, LE Colbert_


Gut microbiome community composition differs between cervical cancer (CC) patients and healthy controls, and increased gut diversity is associated with improved outcomes after treatment. We proposed that functions of specific microbial species adjoining the mucus layer may directly impact the biology of CC.

[Source](https://doi.org/10.1186/s12885-022-09997-0)

10.1186/s12885-022-09997-0

---

## Horizontal gene transfer enables programmable gene stability in synthetic microbiota.
 01.09.2022, _T Wang, A Weiss, A Aqeel, F Wu, AJ Lopatkin, LA David, L You_


The functions of many microbial communities exhibit remarkable stability despite fluctuations in the compositions of these communities. To date, a mechanistic understanding of this function-composition decoupling is lacking. Statistical mechanisms have been commonly hypothesized to explain such decoupling. Here, we proposed that dynamic mechanisms, mediated by horizontal gene transfer (HGT), also enable the independence of functions from the compositions of microbial communities. We combined theoretical analysis with numerical simulations to illustrate that HGT rates can determine the stability of gene abundance in microbial communities. We further validated these predictions using engineered microbial consortia of different complexities transferring one or more than a dozen clinically isolated plasmids, as well as through the reanalysis of data from the literature. Our results demonstrate a generalizable strategy to program the gene stability of microbial communities.

[Source](https://doi.org/10.1038/s41589-022-01114-3)

10.1038/s41589-022-01114-3

---

## A Multiscale Spatiotemporal Model Including a Switch from Aerobic to Anaerobic Metabolism Reproduces Succession in the Early Infant Gut Microbiota.
 01.09.2022, _DM Versluis, R Schoemaker, E Looijesteijn, D Muysken, PV Jeurink, M Paques, JMW Geurts, RMH Merks_


The human intestinal microbiota starts to form immediately after birth and is important for the health of the host. During the first days, facultatively anaerobic bacterial species generally dominate, such as Enterobacteriaceae. These are succeeded by strictly anaerobic species, particularly Bifidobacterium species. An early transition to Bifidobacterium species is associated with health benefits; for example, Bifidobacterium species repress growth of pathogenic competitors and modulate the immune response. Succession to Bifidobacterium is thought to be due to consumption of intracolonic oxygen present in newborns by facultative anaerobes, including Enterobacteriaceae. To study if oxygen depletion suffices for the transition to Bifidobacterium species, here we introduced a multiscale mathematical model that considers metabolism, spatial bacterial population dynamics, and cross-feeding. Using publicly available metabolic network data from the AGORA collection, the model simulates ab initio the competition of strictly and facultatively anaerobic species in a gut-like environment under the influence of lactose and oxygen. The model predicts that individual differences in intracolonic oxygen in newborn infants can explain the observed individual variation in succession to anaerobic species, in particular Bifidobacterium species. Bifidobacterium species became dominant in the model by their use of the bifid shunt, which allows Bifidobacterium to switch to suboptimal yield metabolism with fast growth at high lactose concentrations, as predicted here using flux balance analysis. The computational model thus allows us to test the internal plausibility of hypotheses for bacterial colonization and succession in the infant colon. IMPORTANCE The composition of the infant microbiota has a great impact on infant health, but its controlling factors are still incompletely understood. The frequently dominant anaerobic Bifidobacterium species benefit health, e.g., they can keep harmful competitors under control and modulate the intestinal immune response. Controlling factors could include nutritional composition and intestinal mucus composition, as well as environmental factors, such as antibiotics. We introduce a modeling framework of a metabolically realistic intestinal microbial ecology in which hypothetical scenarios can be tested and compared. We present simulations that suggest that greater levels of intraintestinal oxygenation more strongly delay the dominance of Bifidobacterium species, explaining the observed variety of microbial composition and demonstrating the use of the model for hypothesis generation. The framework allowed us to test a variety of controlling factors, including intestinal mixing and transit time. Future versions will also include detailed modeling of oligosaccharide and mucin metabolism.

[Source](https://doi.org/10.1128/msystems.00446-22)

10.1128/msystems.00446-22

---

## Gut microbiome dysbiosis is associated with increased mortality after solid organ transplantation.
 31.08.2022, _JC Swarte, Y Li, S Hu, JR Björk, R Gacesa, A Vich Vila, RM Douwes, V Collij, A Kurilshikov, A Post, MAY Klaassen, MF Eisenga, AW Gomes-Neto, D Kremer, BH Jansen, TJ Knobbe, SP Berger, JF Sanders, MR Heiner-Fokkema, RJ Porte, FJC Cuperus, VE de Meijer, C Wijmenga, EAM Festen, A Zhernakova, J Fu, HJM Harmsen, H Blokzijl, SJL Bakker, RK Weersma_


Organ transplantation is a life-saving treatment for patients with end-stage disease, but survival rates after transplantation vary considerably. There is now increasing evidence that the gut microbiome is linked to the survival of patients undergoing hematopoietic cell transplant, yet little is known about the role of the gut microbiome in solid organ transplantation. We analyzed 1370 fecal samples from 415 liver and 672 renal transplant recipients using shotgun metagenomic sequencing to assess microbial taxonomy, metabolic pathways, antibiotic resistance genes, and virulence factors. To quantify taxonomic and metabolic dysbiosis, we also analyzed 1183 age-, sex-, and body mass index-matched controls from the same population. In addition, a subset of 78 renal transplant recipients was followed longitudinally from pretransplantation to 24 months after transplantation. Our data showed that both liver and kidney transplant recipients suffered from gut dysbiosis, including lower microbial diversity, increased abundance of unhealthy microbial species, decreased abundance of important metabolic pathways, and increased prevalence and diversity of antibiotic resistance genes and virulence factors. These changes were found to persist up to 20 years after transplantation. Last, we demonstrated that the use of immunosuppressive drugs was associated with the observed dysbiosis and that the extent of dysbiosis was associated with increased mortality after transplantation. This study represents a step toward potential microbiome-targeted interventions that might influence the outcomes of recipients of solid organ transplantation.

[Source](https://doi.org/10.1126/scitranslmed.abn7566)

10.1126/scitranslmed.abn7566

---

## Intestinal Microbiomics and Metabolomics Insights into the Hepatoprotective Effects of <em>Lactobacillus paracasei</em> CCFM1222 Against the Acute Liver Injury in Mice

[Source](https://doi.org/10.1007/s12602-022-09986-6)

---

## An Alternative Oat-Containing Ready-To-Use Therapeutic Food Does Not Alter Intestinal Permeability or the 16S Rrna Fecal Microbiome Configuration Among Children With Severe Malnutrition in Sierra Leone: A Randomized Controlled Trial.
 02.09.2022, _DT Hendrixson, N Naskidashvili, KB Stephenson, ML Laury, AS Koroma, MJ Manary_


Previously, a novel oat-ready-to-use therapeutic food (o-RUTF) resulted in improved recovery from severe acute malnutrition (SAM) when compared to standard RUTF (s-RUTF). O-RUTF contained 18% oat, while s-RUTF has no cereal ingredients.

[Source](https://doi.org/10.1093/jn/nxac207)

10.1093/jn/nxac207

---

## Study on the Effect of Molecular Weight on the Gut Microbiota Fermentation Properties of Blackberry Polysaccharides <em>In Vitro</em>

[Source](https://doi.org/10.1021/acs.jafc.2c03091)

---

## Tissue-wide metabolomics reveals wide impact of gut microbiota on mice metabolite composition.
 02.09.2022, _I Zarei, VM Koistinen, M Kokla, A Klåvus, AF Babu, M Lehtonen, S Auriola, K Hanhineva_


The essential role of gut microbiota in health and disease is well recognized, but the biochemical details that underlie the beneficial impact remain largely undefined. To maintain its stability, microbiota participates in an interactive host-microbiota metabolic signaling, impacting metabolic phenotypes of the host. Dysbiosis of microbiota results in alteration of certain microbial and host metabolites. Identifying these markers could enhance early detection of certain diseases. We report LC-MS based non-targeted metabolic profiling that demonstrates a large effect of gut microbiota on mammalian tissue metabolites. It was hypothesized that gut microbiota influences the overall biochemistry of host metabolome and this effect is tissue-specific. Thirteen different tissues from germ-free (GF) and conventionally-raised (MPF) C57BL/6NTac mice were selected and their metabolic differences were analyzed. Our study demonstrated a large effect of microbiota on mammalian biochemistry at different tissues and resulted in statistically-significant modulation of metabolites from multiple metabolic pathways (p ≤ 0.05). Hundreds of molecular features were detected exclusively in one mouse group, with the majority of these being unique to specific tissue. A vast metabolic response of host to metabolites generated by the microbiota was observed, suggesting gut microbiota has a direct impact on host metabolism.

[Source](https://doi.org/10.1038/s41598-022-19327-w)

10.1038/s41598-022-19327-w

---

## Gut microbiota mediates the inhibition of lymphopoiesis in dietary-restricted mice by suppressing glycolysis.
 01.09.2022, _S Tao, Y Wang, C Yu, R Qiu, Y Jiang, J Jia, Z Tao, L Zhang, B Zou, D Tang_


Dietary restriction (DR) is one of the most robust interventions shown to extend health-span and remains on the forefront of anti-aging intervention studies, though conflicting results have been shown on its effect on lifespan both in rodents and primates. The severe inhibitory effects on the lymphoid lineage by DR remains one of its major negative downsides which reduces its overall beneficial effects on organismal health. Yet, the underlying mechanism of how DR suppresses the lymphoid system remains to be explored. Here, we show that antibiotic ablation of gut microbiota significantly rescued the inhibition of lymphopoiesis by DR. Interestingly, glycolysis in lymphocytes was significantly down-regulated in DR mice and pharmacological inhibition of glycolysis reverted this rescue effect of lymphopoiesis in DR mice with ablated gut microbiota. Furthermore, DR remarkably reconstructed gut microbiota with a significant increase in butyrate-producing bacterial taxa and in expression of But, a key gene involved in butyrate synthesis. Moreover, supplemental butyrate feeding in AL mice suppressed glycolysis in lymphoid cells and mimicked the inhibition of lymphopoiesis in AL mice. Together, our study reveals that gut microbiota mediates the inhibition on lymphopoiesis via down-regulation of glycolysis under DR conditions, which is associated with increased butyrate-synthesis. Our study uncovered a candidate that could potentially be targeted for ameliorating the negative effects of DR on lymphopoiesis, and therefore may have important implications for the wider application of DR and promoting healthy aging.

[Source](https://doi.org/10.1080/19490976.2022.2117509)

10.1080/19490976.2022.2117509

---

## Limited microbiome differences in captive and semi-wild primate populations consuming similar diets.
 01.09.2022, _S Kuthyar, K Watson, S Huang, LJN Brent, M Platt, J Horvath, J Gonzalez-Martinez, M Martínez, F Godoy-Vitorino, R Knight, MG Dominguez-Bello, KR Amato_


Gut microbial communities are shaped by a myriad of extrinsic factors, including diet and the environment. Although distinct human populations consistently exhibit different gut microbiome compositions, variation in diet and environmental factors are almost always coupled, making it difficult to disentangle their relative contributions to shaping the gut microbiota. Data from discrete animal populations with similar diets can help reduce confounds. Here, we assessed the gut microbiota of free-ranging and captive rhesus macaques with at least 80% diet similarity to test the hypothesis that hosts in difference environments will have different gut microbiomes despite a shared diet. Although we found that location was a significant predictor of gut microbial composition, the magnitude of observed differences was relatively small. These patterns suggest that a shared diet may limit the typical influence of environmental microbial exposure on the gut microbiota.

[Source](https://doi.org/10.1093/femsec/fiac098)

10.1093/femsec/fiac098

---

## Biodiversity, environmental drivers, and sustainability of the global deep-sea sponge microbiome.
 02.09.2022, _K Busch, BM Slaby, W Bach, A Boetius, I Clefsen, A Colaço, M Creemers, J Cristobo, L Federwisch, A Franke, A Gavriilidou, A Hethke, E Kenchington, F Mienis, S Mills, A Riesgo, P Ríos, EM Roberts, D Sipkema, L Pita, PJ Schupp, J Xavier, HT Rapp, U Hentschel_


In the deep ocean symbioses between microbes and invertebrates are emerging as key drivers of ecosystem health and services. We present a large-scale analysis of microbial diversity in deep-sea sponges (Porifera) from scales of sponge individuals to ocean basins, covering 52 locations, 1077 host individuals translating into 169 sponge species (including understudied glass sponges), and 469 reference samples, collected anew during 21 ship-based expeditions. We demonstrate the impacts of the sponge microbial abundance status, geographic distance, sponge phylogeny, and the physical-biogeochemical environment as drivers of microbiome composition, in descending order of relevance. Our study further discloses that fundamental concepts of sponge microbiology apply robustly to sponges from the deep-sea across distances of &gt;10,000 km. Deep-sea sponge microbiomes are less complex, yet more heterogeneous, than their shallow-water counterparts. Our analysis underscores the uniqueness of each deep-sea sponge ground based on which we provide critical knowledge for conservation of these vulnerable ecosystems.

[Source](https://doi.org/10.1038/s41467-022-32684-4)

10.1038/s41467-022-32684-4

---

## Strain-level profiling with picodroplet microfluidic cultivation reveals host-specific adaption of honeybee gut symbionts.
 04.09.2022, _Y Meng, S Li, C Zhang, H Zheng_


Symbiotic gut microbes have a rich genomic and metabolic pool and are closely related to hosts' health. Traditional sequencing profiling masks the genomic and phenotypic diversity among strains from the same species. Innovative droplet-based microfluidic cultivation may help to elucidate the inter-strain interactions. A limited number of bacterial phylotypes colonize the honeybee gut, while individual strains possess unique genomic potential and critical capabilities, which provides a particularly good model for strain-level analyses.

[Source](https://doi.org/10.1186/s40168-022-01333-9)

10.1186/s40168-022-01333-9

---

## Mucin O-glycan-microbiota axis orchestrates gut homeostasis in a diarrheal pig model

[Source](https://doi.org/10.1186/s40168-022-01326-8)

---

## Stressful, isolated, yet diverse: Green roofs have rich microbiomes that are not dominated by oligotrophic taxa

[Source](https://doi.org/10.1111/1758-2229.13120)

---

## Nodulation and nitrogen fixation in <em>Medicago truncatula</em> strongly alters the abundance of its root microbiota and subtly affects its structure

[Source](https://doi.org/10.1111/1462-2920.16164)

---

## Gut bacterial nutrient preferences quantified in&nbsp;vivo

[Source](https://doi.org/10.1016/j.cell.2022.07.020)

---

## Metric learning for comparing genomic data with triplet network.
 01.09.2022, _Z Ma, YY Lu, Y Wang, R Lin, Z Yang, F Zhang, Y Wang_


Many biological applications are essentially pairwise comparison problems, such as evolutionary relationships on genomic sequences, contigs binning on metagenomic data, cell type identification on gene expression profiles of single-cells, etc. To make pair-wise comparison, it is necessary to adopt suitable dissimilarity metric. However, not all the metrics can be fully adapted to all possible biological applications. It is necessary to employ metric learning based on data adaptive to the application of interest. Therefore, in this study, we proposed MEtric Learning with Triplet network (MELT), which learns a nonlinear mapping from original space to the embedding space in order to keep similar data closer and dissimilar data far apart. MELT is a weakly supervised and data-driven comparison framework that offers more adaptive and accurate dissimilarity learned in the absence of the label information when the supervised methods are not applicable. We applied MELT in three typical applications of genomic data comparison, including hierarchical genomic sequences, longitudinal microbiome samples and longitudinal single-cell gene expression profiles, which have no distinctive grouping information. In the experiments, MELT demonstrated its empirical utility in comparison to many widely used dissimilarity metrics. And MELT is expected to accommodate a more extensive set of applications in large-scale genomic comparisons. MELT is available at https://github.com/Ying-Lab/MELT.

[Source](https://doi.org/10.1093/bib/bbac345)

10.1093/bib/bbac345

---

## STENSL: Microbial Source Tracking with ENvironment SeLection.
 01.09.2022, _U An, L Shenhav, CA Olson, EY Hsiao, E Halperin, S Sankararaman_


Microbial source tracking analysis has emerged as a widespread technique for characterizing the properties of complex microbial communities. However, this analysis is currently limited to source environments sampled in a specific study. In order to expand the scope beyond one single study and allow the exploration of source environments using large databases and repositories, such as the Earth Microbiome Project, a source selection procedure is required. Such a procedure will allow differentiating between contributing environments and nuisance ones when the number of potential sources considered is high. Here, we introduce STENSL (microbial Source Tracking with ENvironment SeLection), a machine learning method that extends common microbial source tracking analysis by performing an unsupervised source selection and enabling sparse identification of latent source environments. By incorporating sparsity into the estimation of potential source environments, STENSL improves the accuracy of true source contribution, while significantly reducing the noise introduced by noncontributing ones. We therefore anticipate that source selection will augment microbial source tracking analyses, enabling exploration of multiple source environments from publicly available repositories while maintaining high accuracy of the statistical inference. IMPORTANCE Microbial source tracking is a powerful tool to characterize the properties of complex microbial communities. However, this analysis is currently limited to source environments sampled in a specific study. In many applications there is a clear need to consider source selection over a large array of microbial environments, external to the study. To this end, we developed STENSL (microbial Source Tracking with ENvironment SeLection), an expectation-maximization algorithm with sparsity that enables the identification of contributing sources among a large set of potential microbial environments. With the unprecedented expansion of microbiome data repositories such as the Earth Microbiome Project, recording over 200,000 samples from more than 50 types of categorized environments, STENSL takes the first steps in performing automated source exploration and selection. STENSL is significantly more accurate in identifying the contributing sources as well as the unknown source, even when considering hundreds of potential source environments, settings in which state-of-the-art microbial source tracking methods add considerable error.

[Source](https://doi.org/10.1128/msystems.00995-21)

10.1128/msystems.00995-21

---

## Emerging microfluidic technologies for microbiome research.
 03.09.2022, _Y Yu, H Wen, S Li, H Cao, X Li, Z Ma, X She, L Zhou, S Huang_


The importance of the microbiome is increasingly prominent. For example, the human microbiome has been proven to be strongly associated with health conditions, while the environmental microbiome is recognized to have a profound influence on agriculture and even the global climate. Furthermore, the microbiome can serve as a fascinating reservoir of genes that encode tremendously valuable compounds for industrial and medical applications. In the past decades, various technologies have been developed to better understand and exploit the microbiome. In particular, microfluidics has demonstrated its strength and prominence in the microbiome research. By taking advantage of microfluidic technologies, inherited shortcomings of traditional methods such as low throughput, labor-consuming, and high-cost are being compensated or bypassed. In this review, we will summarize a broad spectrum of microfluidic technologies that have addressed various needs in the field of microbiome research, as well as the achievements that were enabled by the microfluidics (or technological advances). Finally, how microfluidics overcomes the limitations of conventional methods by technology integration will also be discussed.

[Source](https://doi.org/10.3389/fmicb.2022.906979)

10.3389/fmicb.2022.906979

---

