## Scale-dependent tipping points of bacterial colonization resistance.
 11.02.2022, _Y Karita, DT Limmer, O Hallatschek_


Bacteria are efficient colonizers of a wide range of secluded microhabitats, such as soil pores, skin follicles, or intestinal crypts. How the structural diversity of these habitats modulates microbial self-organization remains poorly understood, in part because of the difficulty to precisely manipulate the physical structure of microbial environments. Using a microfluidic device to grow bacteria in crypt-like incubation chambers of systematically varied lengths, we show that small variations in the physical structure of the microhabitat can drastically alter bacterial colonization success and resistance against invaders. Small crypts are uncolonizable; intermediately sized crypts can stably support dilute populations, while beyond a second critical length scale, populations phase separate into a dilute region and a jammed region. The jammed state is characterized by extreme colonization resistance, even if the resident strain is suppressed by an antibiotic. Combined with a flexible biophysical model, we demonstrate that colonization resistance and associated priority effects can be explained by a crowding-induced phase transition, which results from a competition between proliferation and density-dependent cell leakage. The emerging sensitivity to scale underscores the need to control for scale in microbial ecology experiments. Systematic flow-adjustable length-scale variations may serve as a promising strategy to elucidate further scale-sensitive tipping points and to rationally modulate the stability and resilience of microbial colonizers.

[Source](https://doi.org/10.1073/pnas.2115496119)

10.1073/pnas.2115496119

---

## Microbial protection favors parasite tolerance and alters host-parasite coevolutionary dynamics.
 12.02.2022, _C Rafaluk-Mohr, M Gerth, JE Sealey, AKE Ekroth, AA Aboobaker, A Kloock, KC King_


Coevolution between hosts and parasites is a major driver of rapid evolutionary change1 and diversification.2,3 However, direct antagonistic interactions between hosts and parasites could be disrupted4 when host microbiota form a line of defense, a phenomenon widespread across animal and plant species.5,6 By suppressing parasite infection, protective microbiota could reduce the need for host-based defenses and favor host support for microbiota colonization,6 raising the possibility that the microbiota can alter host-parasite coevolutionary patterns and processes.7 Here, using an experimental evolution approach, we co-passaged populations of nematode host (Caenorhabditis elegans) and parasites (Staphylococcus aureus) when hosts were colonized (or not) by protective bacteria (Enterococcus faecalis). We found that microbial protection during coevolution resulted in the evolution of host mortality tolerance-higher survival following parasite infection-and in parasites adapting to microbial defenses. Compared to unprotected host-parasite coevolution, the protected treatment was associated with reduced dominance of fluctuating selection dynamics in host populations. No differences in host recombination rate or genetic diversity were detected. Genomic divergence was observed between parasite populations coevolved in protected and unprotected hosts. These findings indicate that protective host microbiota can determine the evolution of host defense strategies and shape host-parasite coevolutionary dynamics.

[Source](https://doi.org/10.1016/j.cub.2022.01.063)

10.1016/j.cub.2022.01.063

---

## Seasonal Airway Microbiome and Transcriptome Interactions Promote Childhood Asthma Exacerbations

[Source](https://doi.org/10.1016/j.jaci.2022.01.020)

---

## Secreted Toxins From Staphylococcus aureus Strains Isolated From Keratinocyte Skin Cancers Mediate Pro-tumorigenic Inflammatory Responses in the Skin

[Source](https://doi.org/10.3389/fmicb.2021.789042)

---

## Aged related human skin microbiome and mycobiome in Korean women

[Source](https://doi.org/10.1038/s41598-022-06189-5)

---

## Extracellular Vesicles Generated by Gram-Positive Bacteria Protect Human Tissues Ex Vivo From HIV-1 Infection

[Source](https://doi.org/10.3389/fcimb.2021.822882)

---

## A gut-derived metabolite alters brain activity and anxiety behavior in mice

[Source](https://doi.org/10.1038/s41586-022-04396-8)

---

## Safety and target engagement of an oral small-molecule sequestrant in adolescents with autism spectrum disorder: an open-label phase 1b/2a trial

[Source](https://doi.org/10.1038/s41591-022-01683-9)

---

## Tryptophan-derived microbial metabolites activate the aryl hydrocarbon receptor in tumor-associated macrophages to suppress anti-tumor immunity

[Source](https://doi.org/10.1016/j.immuni.2022.01.006)

---

## Stochastic microbiome assembly depends on context

[Source](https://doi.org/10.1073/pnas.2115877119)

---

## -Associated Atg16L1T300A Polymorphism Regulates Commensal Microbiota of the Intestine

[Source](https://doi.org/10.3389/fimmu.2021.772189)

---

## A homeostatic interferon-lambda response to bacterial microbiota stimulates preemptive antiviral defense within discrete pockets of intestinal epithelium

[Source](https://doi.org/10.7554/eLife.74072)

---

## Enteric nervous system modulation of luminal pH modifies the microbial environment to promote intestinal health

[Source](https://doi.org/10.1371/journal.ppat.1009989)

---

## Characterization of the Gut Microbiome and Resistomes of Wild and Zoo-Captive Macaques.
 11.02.2022, _T Jia, WS Chang, VR Marcelino, S Zhao, X Liu, Y You, EC Holmes, M Shi, C Zhang_


Rhesus macaques (Macaca mulatta) are the most widely distributed species of Old World monkey and are frequently used as animal models to study human health and disease. Their gastrointestinal microbial community likely plays a major role in their physiology, ecology and evolution. Herein, we compared the fecal microbiome and antibiotic resistance genes in 15 free-ranging and 81 zoo-captive rhesus macaques sampled from two zoos in China, using both 16S amplicon sequencing and whole genome shotgun DNA sequencing approaches. Our data revealed similar levels of microbial diversity/richness among the three groups, although the composition of each group differed significantly and were particularly marked between the two zoo-captive and one wild groups. Zoo-captive animals also demonstrated a greater abundance and diversity of antibiotic genes. Through whole genome shotgun sequencing we also identified a mammalian (simian) associated adenovirus. Overall, this study provides a comprehensive analysis of resistomes and microbiomes in zoo-captive and free-ranging monkeys, revealing that semi-captive wildlife might harbor a higher diversity of antimicrobial resistant genes.

[Source](https://doi.org/10.3389/fvets.2021.778556)

10.3389/fvets.2021.778556

---

## Deep Functional Profiling of Wild Animal Microbiomes Reveals Probiotic <em>Bacillus pumilus</em> Strains with a Common Biosynthetic Fingerprint

[Source](https://doi.org/10.3390/ijms23031168)

---

## Plant immunity suppression via PHR1-RALF-FERONIA shapes the root microbiome to alleviate phosphate starvation.
 11.02.2022, _J Tang, D Wu, X Li, L Wang, L Xu, Y Zhang, F Xu, H Liu, Q Xie, S Dai, D Coleman-Derr, S Zhu, F Yu_


The microbiome plays an important role in shaping plant growth and immunity, but few plant genes and pathways impacting plant microbiome composition have been reported. In Arabidopsis thaliana, the phosphate starvation response (PSR) was recently found to modulate the root microbiome upon phosphate (Pi) starvation through the transcriptional regulator PHR1. Here, we report that A. thaliana PHR1 directly binds to the promoters of rapid alkalinization factor (RALF) genes, and activates their expression under phosphate-starvation conditions. RALFs in turn suppress complex formation of pathogen-associated molecular pattern (PAMP)-triggered immunity (PTI) receptor through FERONIA, a previously-identified PTI modulator that increases resistance to certain detrimental microorganisms. Suppression of immunity via the PHR1-RALF-FERONIA axis allows colonization by specialized root microbiota that help to alleviate phosphate starvation by upregulating the expression of PSR genes. These findings provide a new paradigm for coordination of host-microbe homeostasis through modulating plant innate immunity after environmental perturbations.

[Source](https://doi.org/10.15252/embj.2021109102)

10.15252/embj.2021109102

---

## Extracts from Environmental Strains of Pseudomonas spp. Effectively Control Fungal Plant Diseases.
 18.02.2022, _V Librizzi, A Malacrinò, MG Li Destri Nicosia, N Barger, T Luzzatto-Knaan, S Pangallo, GE Agosteo, L Schena_


The use of synthetic chemical products in agriculture is causing severe damage to the environment and human health, but agrochemicals are still widely used to protect our crops. To counteract this trend, we have been looking for alternative strategies to control plant diseases without causing harm to the environment or damage to our health. However, these alternatives are still far from completely replacing chemical products. Microorganisms have been widely known as a biological tool to control plant diseases, but their use is still limited due to the high variability in their efficacy, together with issues in product registration. However, the metabolites produced by these microorganisms can represent a novel tool for the environment-friendly management of plant diseases, while reducing the issues mentioned above. In this study, we explore the soil microbial diversity in natural systems to look for microorganisms with the potential to be used in pre- and post-harvest protection against fungal plant pathogens. Using a simple workflow, we isolated 22 bacterial strains that were tested both in vitro and in vivo for their ability to counteract the growth of common plant pathogens. The three best isolates, identified as members of the bacterial genus Pseudomonas, were used to produce a series of alcoholic extracts, which were then tested for their action against plant pathogens in simulated real-world applications. Results show that extracts from these isolates have an exceptional biocontrol activity and can be successfully used to control plant pathogens in operational setups. Thus, this study shows that the environmental microbiome is an important source of microorganisms producing metabolites that might provide an alternative strategy to synthetic chemical products.

[Source](https://doi.org/10.3390/plants11030436)

10.3390/plants11030436

---

## Mega-fire in Redwood Tanoak Forest Reduces Bacterial and Fungal Richness and Selects for Pyrophilous Taxa that are Phylogenetically Conserved.
 13.02.2022, _DJ Enright, KM Frangioso, K Isobe, DM Rizzo, SI Glassman_


Mega-fires of unprecedented size, intensity, and socio-economic impacts have surged globally due to climate change, fire suppression, and development. Soil microbiomes are critical for post-fire plant regeneration and nutrient cycling, yet how mega-fires impact the soil microbiome remains unclear. We had a serendipitous opportunity to obtain pre- and post-fire soils from the same sampling locations after the 2016 Soberanes mega-fire burned with high severity throughout several of our established redwood-tanoak plots. This makes our study the first to examine microbial fire response in redwood-tanoak forests. We re-sampled soils immediately post-fire from two burned plots and one unburned plot to elucidate the effect of mega-fire on soil microbiomes. We used Illumina MiSeq sequencing of 16S and ITS1 to determine that bacterial and fungal richness were reduced by 38-70% in burned plots, with richness unchanged in the unburned plot. Fire altered composition by 27% for bacteria and 24% for fungi, whereas the unburned plots experienced no change in fungal and negligible change in bacterial composition. We observed pyrophilous taxa that positively responded to fire were phylogenetically conserved, suggesting shared evolutionary traits. For bacteria, fire selected for increased Firmicutes and Actinobacteria. For fungi, fire selected for the Ascomycota classes Pezizomycetes and Eurotiomycetes and for a Basidiomycota class of heat-resistant Geminibasidiomycete yeasts. We build from Grime's Competitor-Stress tolerator-Ruderal (C-S-R) framework and its recent microbial applications to show how our results might fit into a trait-based conceptual model to help predict generalizable microbial responses to fire.

[Source](https://doi.org/10.1111/mec.16399)

10.1111/mec.16399

---

## Analysis of antibiotic resistance genes reveals their important roles in influencing the community structure of ocean microbiome.
 17.02.2022, _P Yang, S Hao, M Han, J Xu, S Yu, C Chen, H Zhang, K Ning_


Antibiotic resistance gene (ARG) content is a well-established driver of microbial abundance and diversity in an environment. By reanalyzing 132 metagenomic datasets from the Tara Oceans project, we aim to unveil the associations between environmental factors, the ocean microbial community structure and ARG contents. We first investigated the structural patterns of microbial communities including both prokaryotes such as bacteria and eukaryotes such as protists. Additionally, several ARG-dominant horizontal gene transfer events between Protist and Prokaryote have been identified, indicating the potential roles of ARG in shaping the ocean microbial communities. For a deeper insight into the role of ARGs in ocean microbial communities on a global scale, we identified 1926 unique types of ARGs and discovered that the ARGs are more abundant and diverse in the mesopelagic zone than other water layers, potentially caused by limited resources. Finally, we found that ARG-enriched genera were often more abundant compared to their ARG-less neighbors in the same environment (e.g. coastal oceans). A deeper understanding of the ARG-microbiome relationships could help in the conservation of the oceanic ecosystem.

[Source](https://doi.org/10.1016/j.scitotenv.2022.153731)

10.1016/j.scitotenv.2022.153731

---

## Analysis of antibiotic resistance genes reveals its important role in influencing the community structure of ocean microbiome

[Source](https://doi.org/10.1016/j.scitotenv.2022.153731)

---

## Seaweeds influence oyster microbiota and disease susceptibility.
 09.02.2022, _E Dugeny, J de Lorgeril, B Petton, E Toulza, Y Gueguen, F Pernet_


A growing awareness of role that microbiota can play in mediating the effects of pathogens on hosts has given rise to the concept of the pathobiome. Recently, we demonstrated that the Pacific oyster mortality syndrome affecting Crassostrea gigas oysters is caused by infection with the Ostreid herpesvirus type 1 (OsHV-1) followed by infection with multiple bacterial taxa. Here we extend the concept of this pathobiome beyond the host species and its bacterial microbiota by investigating how seaweed living in association with oysters influences their response to the disease. We hypothesized that by their mere presence in the environment, different species of seaweeds can positively or negatively influence the risk of disease in oysters by shaping their bacterial microbiota and their immune response. Although seaweed and oysters do not have direct ecological interactions, they are connected by seawater and likely share microbes. To test our hypothesis, oysters were acclimated with green, brown or red algae for 2 weeks and then challenged with OsHV-1. We monitored host survival and pathogen proliferation and performed bacterial microbiota and transcriptome analyses. We found that seaweeds can alter the bacterial microbiota of the host and its response to the disease. More particularly, green algae belonging to the genus Ulva spp. induced bacterial microbiota dysbiosis in oyster and modification of its transcriptional immune response leading to increased susceptibility to the disease. This work provides a better understanding of a marine disease and highlights the importance of considering both macrobiotic and microbiotic interactions for conservation, management and exploitation of marine ecosystems and resources.

[Source](https://doi.org/10.1111/1365-2656.13662)

10.1111/1365-2656.13662

---

## Vertical stratification of the air microbiome in the lower troposphere.
 08.02.2022, _DI Drautz-Moses, I Luhung, ES Gusareva, C Kee, NE Gaultier, BNV Premkrishnan, CF Lee, ST Leong, C Park, ZH Yap, CE Heinle, KJX Lau, RW Purbojati, SBY Lim, YH Lim, SK Kutmutia, NW Aung, EL Oliveira, SG Ng, J Dacanay, PN Ang, SD Spence, WJ Phung, A Wong, RJ Kennedy, N Kalsi, SP Sasi, L Chandrasekaran, A Uchida, ACM Junqueira, HL Kim, R Hankers, T Feuerle, U Corsmeier, SC Schuster_


The troposphere constitutes the final frontier of global ecosystem research due to technical challenges arising from its size, low biomass, and gaseous state. Using a vertical testing array comprising a meteorological tower and a research aircraft, we conducted synchronized measurements of meteorological parameters and airborne biomass (n = 480) in the vertical air column up to 3,500 m. The taxonomic analysis of metagenomic data revealed differing patterns of airborne microbial community composition with respect to time of day and height above ground. The temporal and spatial resolution of our study demonstrated that the diel cycle of airborne microorganisms is a ground-based phenomenon that is entirely absent at heights &gt;1,000 m. In an integrated analysis combining meteorological and biological data, we demonstrate that atmospheric turbulence, identified by potential temperature and high-frequency three-component wind measurements, is the key driver of bioaerosol dynamics in the lower troposphere. Multivariate regression analysis shows that at least 50% of identified airborne microbial taxa (n = ∼10,000) are associated with either ground or height, allowing for an understanding of dispersal patterns of microbial taxa in the vertical air column. Due to the interconnectedness of atmospheric turbulence and temperature, the dynamics of microbial dispersal are likely to be impacted by rising global temperatures, thereby also affecting ecosystems on the planetary surface.

[Source](https://doi.org/10.1073/pnas.2117293119)

10.1073/pnas.2117293119

---

## Evaluating supervised and unsupervised background noise correction in human gut microbiome data.
 18.02.2022, _L Briscoe, B Balliu, S Sankararaman, E Halperin, NR Garud_


The ability to predict human phenotypes and identify biomarkers of disease from metagenomic data is crucial for the development of therapeutics for microbiome-associated diseases. However, metagenomic data is commonly affected by technical variables unrelated to the phenotype of interest, such as sequencing protocol, which can make it difficult to predict phenotype and find biomarkers of disease. Supervised methods to correct for background noise, originally designed for gene expression and RNA-seq data, are commonly applied to microbiome data but may be limited because they cannot account for unmeasured sources of variation. Unsupervised approaches address this issue, but current methods are limited because they are ill-equipped to deal with the unique aspects of microbiome data, which is compositional, highly skewed, and sparse. We perform a comparative analysis of the ability of different denoising transformations in combination with supervised correction methods as well as an unsupervised principal component correction approach that is presently used in other domains but has not been applied to microbiome data to date. We find that the unsupervised principal component correction approach has comparable ability in reducing false discovery of biomarkers as the supervised approaches, with the added benefit of not needing to know the sources of variation apriori. However, in prediction tasks, it appears to only improve prediction when technical variables contribute to the majority of variance in the data. As new and larger metagenomic datasets become increasingly available, background noise correction will become essential for generating reproducible microbiome analyses.

[Source](https://doi.org/10.1371/journal.pcbi.1009838)

10.1371/journal.pcbi.1009838

---

