## Effects of Extended Postmortem Interval on Microbial Communities in Organs of the Human Cadaver

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.569630/full)

---

## Landscapes of bacterial and metabolic signatures and their interaction in major depressive disorders.
 09.12.2020, _J Yang, P Zheng, Y Li, J Wu, X Tan, J Zhou, Z Sun, X Chen, G Zhang, H Zhang, Y Huang, T Chai, J Duan, W Liang, B Yin, J Lai, T Huang, Y Du, P Zhang, J Jiang, C Xi, L Wu, J Lu, T Mou, Y Xu, SW Perry, ML Wong, J Licinio, S Hu, G Wang, P Xie_


Gut microbiome disturbances have been implicated in major depressive disorder (MDD). However, little is known about how the gut virome, microbiome, and fecal metabolome change, and how they interact in MDD. Here, using whole-genome shotgun metagenomic and untargeted metabolomic methods, we identified 3 bacteriophages, 47 bacterial species, and 50 fecal metabolites showing notable differences in abundance between MDD patients and healthy controls (HCs). Patients with MDD were mainly characterized by increased abundance of the genus Bacteroides and decreased abundance of the genera Blautia and Eubacterium These multilevel omics alterations generated a characteristic MDD coexpression network. Disturbed microbial genes and fecal metabolites were consistently mapped to amino acid (γ-aminobutyrate, phenylalanine, and tryptophan) metabolism. Furthermore, we identified a combinatorial marker panel that robustly discriminated MDD from HC individuals in both the discovery and validation sets. Our findings provide a deep insight into understanding of the roles of disturbed gut ecosystem in MDD.

[Source](https://advances.sciencemag.org/content/6/49/eaba8555)

10.1126/sciadv.aba8555

---

## Improved high-molecular-weight DNA extraction, nanopore sequencing and metagenomic assembly from the human gut microbiome.
 05.12.2020, _DG Maghini, EL Moss, SE Vance, AS Bhatt_


Short-read metagenomic sequencing and de novo genome assembly of the human gut microbiome can yield draft bacterial genomes without isolation and culture. However, bacterial genomes assembled from short-read sequencing are often fragmented. Furthermore, these metagenome-assembled genomes often exclude repeated genomic elements, such as mobile genetic elements, compromising our understanding of the contribution of these elements to important bacterial phenotypes. Although long-read sequencing has been applied successfully to the assembly of contiguous bacterial isolate genomes, extraction of DNA of sufficient molecular weight, purity and quantity for metagenomic sequencing from stool samples can be challenging. Here, we present a protocol for the extraction of microgram quantities of high-molecular-weight DNA from human stool samples that are suitable for downstream long-read sequencing applications. We also present Lathe ( www.github.com/bhattlab/lathe ), a computational workflow for long-read basecalling, assembly, consensus refinement with long reads or Illumina short reads and genome circularization. Altogether, this protocol can yield high-quality contiguous or circular bacterial genomes from a complex human gut sample in approximately 10 d, with 2 d of hands-on bench and computational effort.

[Source](https://www.nature.com/articles/s41596-020-00424-x)

10.1038/s41596-020-00424-x

---

## Elucidation of Gut Microbiota-Associated Lipids Using LC-MS/MS and 16S rRNA Sequence Analyses

[Source](https://www.cell.com/iscience/fulltext/S2589-0042(20)31038-5)

---

## The Healthy Microbiome (What is the definition of a healthy gut microbiome?).
 30.11.2020, _F Shanahan, TS Ghosh, PW O'Toole_


Use of microbiome-based biomarkers in diagnosis, prognosis, risk profiling, and precision therapy requires definition of a healthy microbiome in different populations. To determine features of the intestinal microbiota associated with health, however, we need improved microbiome profiling technologies, with strain-level resolution. We must also learn more about how the microbiome varies among apparently healthy people, how it changes with age, and the effects of diet, medications, ethnicity, geography, and lifestyle. Furthermore, many intestinal microbes, including viruses, phage, fungi, and archaea, have not been characterized, and little is known about their contributions to health and disease.Whether a healthy microbiome can be defined is an important and seemingly simple question, but with a complex answer in continual need of refinement.

[Source](https://www.gastrojournal.org/article/S0016-5085(20)35508-6/)

10.1053/j.gastro.2020.09.057

---

## The gut microbiota is associated with immune cell dynamics in humans.
 09.12.2020, _J Schluter, JU Peled, BP Taylor, KA Markey, M Smith, Y Taur, R Niehus, A Staffas, A Dai, E Fontana, LA Amoretti, RJ Wright, S Morjaria, M Fenelus, MS Pessin, NJ Chao, M Lew, L Bohannon, A Bush, AD Sung, TM Hohl, MA Perales, MRM van den Brink, JB Xavier_


The gut microbiota influences development1-3 and homeostasis4-7 of the mammalian immune system, and is associated with human inflammatory8 and immune diseases9,10 as well as responses to immunotherapy11-14. Nevertheless, our understanding of how gut bacteria modulate the immune system remains limited, particularly in humans, where the difficulty of direct experimentation makes inference challenging. Here we study hundreds of hospitalized-and closely monitored-patients with cancer receiving haematopoietic cell transplantation as they recover from chemotherapy and stem-cell engraftment. This aggressive treatment causes large shifts in both circulatory immune cell and microbiota populations, enabling the relationships between the two to be studied simultaneously. Analysis of observed daily changes in circulating neutrophil, lymphocyte and monocyte counts and more than 10,000 longitudinal microbiota samples revealed consistent associations between gut bacteria and immune cell dynamics. High-resolution clinical metadata and Bayesian inference allowed us to compare the effects of bacterial genera in relation to those of immunomodulatory medications, revealing a considerable influence of the gut microbiota-together and over time-on systemic immune cell dynamics. Our analysis establishes and quantifies the link between the gut microbiota and the human immune system, with implications for microbiota-driven modulation of immunity.

[Source](https://www.nature.com/articles/s41586-020-2971-8)

10.1038/s41586-020-2971-8

---

## Utilizing the gut microbiome in decompensated cirrhosis and acute-on-chronic liver failure.
 01.12.2020, _J Trebicka, P Bork, A Krag, M Arumugam_


The human gut microbiome has emerged as a major player in human health and disease. The liver, as the first organ to encounter microbial products that cross the gut epithelial barrier, is affected by the gut microbiome in many ways. Thus, the gut microbiome might play a major part in the development of liver diseases. The common end stage of liver disease is decompensated cirrhosis and the further development towards acute-on-chronic liver failure (ACLF). These conditions have high short-term mortality. There is evidence that translocation of components of the gut microbiota, facilitated by different pathogenic mechanisms such as increased gut epithelial permeability and portal hypertension, is an important driver of decompensation by induction of systemic inflammation, and thereby also ACLF. Elucidating the role of the gut microbiome in the aetiology of decompensated cirrhosis and ACLF deserves further investigation and improvement; and might be the basis for development of diagnostic and therapeutic strategies. In this Review, we focus on the possible pathogenic, diagnostic and therapeutic role of the gut microbiome in decompensation of cirrhosis and progression to ACLF.

[Source](https://www.nature.com/articles/s41575-020-00376-3)

10.1038/s41575-020-00376-3

---

## Remodeling of the maternal gut microbiome during pregnancy is shaped by parity

[Source](https://www.biorxiv.org/content/10.1101/2020.12.04.412205v1.full.pdf)

---

## <em>Gardnerella vaginalis</em> promotes group B <em>Streptococcus</em> vaginal colonization, enabling ascending uteroplacental infection in pregnant mice

[Source](https://www.ajog.org/article/S0002-9378(20)31331-4/fulltext)

---

## <em>Akkermansia muciniphila</em>: A potential novel mechanism of nuciferine to improve hyperlipidemia

[Source](https://www.sciencedirect.com/science/article/pii/S0753332220312063)

---

## Gut microbiota modulation by both <em>Lactobacillus fermentum</em> MSK 408 and ketogenic diet in a murine model of pentylenetetrazole-induced acute seizure

[Source](https://www.sciencedirect.com/science/article/pii/S092012112030557X)

---

## Host-age prediction from fecal microbiome composition in laboratory mice

[Source](https://www.biorxiv.org/content/10.1101/2020.12.04.412734v1.full.pdf)

---

## Commensal Microbiota Regulates Skin Barrier Function And Repair Via Signaling Through The Aryl Hydrocarbon Receptor

[Source](https://www.biorxiv.org/content/10.1101/2020.12.05.413096v1.full.pdf)

---

## Homogeneous selection promotes microdiversity in the glacier-fed stream microbiome

[Source](https://www.biorxiv.org/content/10.1101/2020.12.03.409391v1.full.pdf)

---

## Long-term persistence of crAss-like phage crAss001 is associated with phase variation in <em>Bacteroides intestinalis</em>

[Source](https://www.biorxiv.org/content/10.1101/2020.12.02.408625v1.full.pdf)

---

## Diversity and Distribution of a Novel Genus of Hyperthermophilic <em>Aquificae </em>Viruses Encoding a Proof-Reading Family-A DNA Polymerase

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.583361/full)

---

## Broad Virus Detection and Variant Discovery in Fecal Samples of Hematopoietic Transplant Recipients Using Targeted Sequence Capture Metagenomics

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.560179/full)

---

## Multidrug-resistant high-risk <em>Enterococcus faecium</em> clones: can we really define them?

[Source](https://www.sciencedirect.com/science/article/pii/S0924857920304386)

---

## Current Trends in Experimental and Computational Approaches to Combat Antimicrobial Resistance 

[Source](https://www.frontiersin.org/articles/10.3389/fgene.2020.563975/full)

---

## Predicting clinical resistance prevalence using sewage metagenomic data

[Source](https://www.nature.com/articles/s42003-020-01439-6)

---

## Fermented-Food Metagenomics Reveals Substrate-Associated Differences in Taxonomy and Health-Associated and Antibiotic Resistance Determinants.
 18.11.2020, _J Leech, R Cabrera-Rubio, AM Walsh, G Macori, CJ Walsh, W Barton, L Finnegan, F Crispie, O O'Sullivan, MJ Claesson, PD Cotter_


Fermented foods have been the focus of ever greater interest as a consequence of purported health benefits. Indeed, it has been suggested that consumption of these foods helps to address the negative consequences of "industrialization" of the human gut microbiota in Western society. However, as the mechanisms via which the microbes in fermented foods improve health are not understood, it is necessary to develop an understanding of the composition and functionality of the fermented-food microbiota to better harness desirable traits. Here, we considerably expand the understanding of fermented-food microbiomes by employing shotgun metagenomic sequencing to provide a comprehensive insight into the microbial composition, diversity, and functional potential (including antimicrobial resistance and carbohydrate-degrading and health-associated gene content) of a diverse range of 58 fermented foods from artisanal producers from a number of countries. Food type, i.e., dairy-, sugar-, or brine-type fermented foods, was the primary driver of microbial composition, with dairy foods found to have the lowest microbial diversity. From the combined data set, 127 high-quality metagenome-assembled genomes (MAGs), including 10 MAGs representing putatively novel species of Acetobacter, Acidisphaera, Gluconobacter, Companilactobacillus, Leuconostoc, and Rouxiella, were generated. Potential health promoting attributes were more common in fermented foods than nonfermented equivalents, with water kefirs, sauerkrauts, and kvasses containing the greatest numbers of potentially health-associated gene clusters. Ultimately, this study provides the most comprehensive insight into the microbiomes of fermented foods to date and yields novel information regarding their relative health-promoting potential.IMPORTANCE Fermented foods are regaining popularity worldwide due in part to a greater appreciation of the health benefits of these foods and the associated microorganisms. Here, we use state-of-the-art approaches to explore the microbiomes of 58 of these foods, identifying the factors that drive the microbial composition of these foods and potential functional benefits associated with these populations. Food type, i.e., dairy-, sugar-, or brine-type fermented foods, was the primary driver of microbial composition, with dairy foods found to have the lowest microbial diversity and, notably, potential health promoting attributes were more common in fermented foods than nonfermented equivalents. The information provided here will provide significant opportunities for the further optimization of fermented-food production and the harnessing of their health-promoting potential.

[Source](https://msystems.asm.org/content/msys/5/6/e00522-20.full.pdf)

10.1128/mSystems.00522-20

---

## Author Correction: Uncovering hidden antimicrobial resistance patterns within the hospital microbiome.
 08.12.2020, _BW Haak, WJ Wiersinga_


A Correction to this paper has been published: https://doi.org/10.1038/s41591-020-0919-z.

[Source](https://www.nature.com/articles/s41591-020-0919-z)

10.1038/s41591-020-01184-7

---

## Uncovering hidden antimicrobial resistance patterns within the hospital microbiome 

[Source](https://www.nature.com/articles/s41591-020-0919-z)

---

## Isolation and characterization of human pathogenic multidrug resistant bacteria associated with plastic litter collected in Zanzibar 

[Source](https://www.sciencedirect.com/science/article/pii/S0304389420325814)

---

## Integrated metagenomic and metatranscriptomic analyses of ultraviolet disinfection effects on antibiotic resistance genes and bacterial communities during wastewater treatment.
 05.12.2020, _S Jia, T Li, XX Zhang_


Ultraviolet (UV) disinfection is now widely implemented in wastewater treatment plants (WWTPs) worldwide, but its effect on antibiotic resistome of the surviving bacteria remains unclear. In this study, we employed high-throughput sequencing-based metagenomic and metatranscriptomic approaches to comprehensively elucidate the effects of UV disinfection on the shifts of bacterial community and antibiotic resistance genes (ARGs) on both DNA and mRNA levels in one WWTP. Metagenomic analyses revealed an insignificant change in the bacterial community after UV disinfection, while metatranscriptomic analyses showed that UV disinfection significantly changed the abundance of 13.79% of phyla and 10.32% of genera. In total, 38 ARG-like open reading frames (ORFs) and 327 ARG-like transcripts were identified in the DNA and RNA samples, respectively. The relative abundances of the total ARGs, each ARG type, and each ARG subtype also varied after UV disinfection. Additionally, UV disinfection significantly reduced the expression of total ARGs from 49.40 transcripts per kilobase of exon model per million mapped reads (TPM) to 47.62 TPM, and significantly changed the expression of 10.75% of ARG subtypes in wastewater (p &lt; 0.05). Notably, the significant increase in the expression and obvious increase in the relative abundance of macrolide-lincosamide-streptogramin B (MLSB) resistance genes revealed that UV disinfection increases the potential health risk of MLSB resistance genes in wastewater. Moreover, potential host analyses of ARGs revealed the different preferences of antibiotic resistant bacteria (ARB) to ARGs. This study may shed new light on the underlying mechanism of the UV disinfection effect on antibiotic resistance.

[Source](https://link.springer.com/article/10.1007%2Fs10646-020-02313-1)

10.1007/s10646-020-02313-1

---

## A metagenomic glimpse into the gut of wild and domestic animals: Quantification of antimicrobial resistance and more.
 03.12.2020, _M Skarżyńska, P Leekitcharoenphon, RS Hendriksen, FM Aarestrup, D Wasyl_


Antimicrobial resistance (AMR) in bacteria is a complex subject, why one need to look at this phenomenon from a wider and holistic perspective. The extensive use of the same antimicrobial classes in human and veterinary medicine as well as horticulture is one of the main drivers for the AMR selection. Here, we applied shotgun metagenomics to investigate the AMR epidemiology in several animal species including farm animals, which are often exposed to antimicrobial treatment opposed to an unique set of wild animals that seems not to be subjected to antimicrobial pressure. The comparison of the domestic and wild animals allowed to investigate the possible anthropogenic impact on AMR spread. Inclusion of animals with different feeding behaviors (carnivores, omnivores) enabled to further assess which AMR genes that thrives within the food chain. We tested fecal samples not only of intensively produced chickens, turkeys, and pigs, but also of wild animals such as wild boars, red foxes, and rodents. A multi-directional approach mapping obtained sequences to several databases provided insight into the occurrence of the different AMR genes. The method applied enabled also analysis of other factors that may influence AMR of intestinal microbiome such as diet. Our findings confirmed higher levels of AMR in farm animals than in wildlife. The results also revealed the potential of wildlife in the AMR dissemination. Particularly in red foxes, we found evidence of several AMR genes conferring resistance to critically important antimicrobials like quinolones and cephalosporins. In contrast, the lowest abundance of AMR was observed in rodents originating from natural environment with presumed limited exposure to antimicrobials. Shotgun metagenomics enabled us to demonstrate that discrepancies between AMR profiles found in the intestinal microbiome of various animals probably resulted from the different antimicrobial exposure, habitats, and behavior of the tested animal species.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0242987)

10.1371/journal.pone.0242987

---

## Mass allelic exchange: enabling sexual genetics in <em>Escherichia coli</em>

[Source](https://www.biorxiv.org/content/10.1101/2020.11.30.405282v1.full.pdf)

---

## Microbial model communities: To understand complexity, harness the power of simplicity

[Source](https://www.sciencedirect.com/science/article/pii/S2001037020305109)

---

## MALDI-TOF mass spectrometry for sub-typing of <em>Streptococcus pneumoniae</em>

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-020-02052-7)

---

## Automated Prediction and Annotation of Small Open Reading Frames in Microbial Genomes

[Source](https://www.cell.com/cell-host-microbe/fulltext/S1931-3128(20)30619-3)

---

## Accurate Microbiome Sequencing with Synthetic Long Read Sequencing

[Source](https://www.biorxiv.org/content/10.1101/2020.10.02.324038v1.full.pdf)

---

## Species level resolution of female bladder microbiota from marker gene surveys

[Source](https://www.biorxiv.org/content/10.1101/2020.10.27.358408v1.full.pdf)

---

## Synthetic Sequencing Standards: A Guide to Database Choice for Rumen Microbiota Amplicon Sequencing Analysis

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.606825/full)

---

## H3ABioNet Seminars and Webinars

[Source](https://www.youtube.com/playlist?list=PLcQ0XMykNhCRpi9UwTcbxH3J4iv02HTC6)

---

## Inspiring the next generation of scientists… online

[Source](https://www.youtube.com/watch?v=IWuCiIENLKY&feature=youtu.be)

---

## MLCB 2020 Day 1 and 2 talks

[Source](https://sites.google.com/cs.washington.edu/mlcb2020/)

---

## Why We’re Giving Thanks To Microbes For Stinky Cheese

[Source](https://www.sciencefriday.com/segments/cheese-mold-microbe/)

---

## Planning Our “Home-mas” this December

[Source](https://misspond.com/life/tips-for-home-mas-2020/)

---

## Holiday Gift Guide For Microbe Lovers

[Source](https://joyfulmicrobe.com/holiday-gift-guide-for-microbe-lovers/)

---

## The 2020 cookbook giftlist

[Source](https://www.matchingfoodandwine.com/news/book_reviews/the-2020-matching-food--wine-cookbook-giftlist-/)

---

