## Altered diversity and composition of gut microbiota in Wilson's disease.
 29.12.2020, _X Cai, L Deng, X Ma, Y Guo, Z Feng, M Liu, Y Guan, Y Huang, J Deng, H Li, H Sang, F Liu, X Yang_


Wilson's disease (WD) is an autosomal recessive inherited disorder of chronic copper toxicosis with high mortality and disability. Recent evidence suggests a correlation between dysbiosis in gut microbiome and multiple diseases such as genetic and metabolic disease. However, the impact of intestinal microbiota polymorphism in WD have not been fully elaborated and need to be explore for seeking some microbiota benefit for WD patients. In this study, the 16S rRNA sequencing was performed on fecal samples from 14 patients with WD and was compared to the results from 16 healthy individuals. The diversity and composition of the gut microbiome in the WD group were significantly lower than those in healthy individuals. The WD group presented unique richness of Gemellaceae, Pseudomonadaceae and Spirochaetaceae at family level, which were hardly detected in healthy controls. The WD group had a markedly lower abundance of Actinobacteria, Firmicutes and Verrucomicrobia, and a higher abundance of Bacteroidetes, Proteobacteria, Cyanobacteria and Fusobacteria than that in healthy individuals. The Firmicutes to Bacteroidetes ratio in the WD group was significantly lower than that of healthy control. In addition, the functional profile of the gut microbiome from WD patients showed a lower abundance of bacterial groups involved in the host immune and metabolism associated systems pathways such as transcription factors and ABC-type transporters, compared to healthy individuals. These results implied dysbiosis of gut microbiota may be influenced by the host metabolic disorders of WD, which may provide a new understanding of the pathogenesis and new possible therapeutic targets for WD.

[Source](https://www.nature.com/articles/s41598-020-78988-7)

10.1038/s41598-020-78988-7

---

## Emotional well-being and gut microbiome profiles by enterotype.
 02.12.2020, _SH Lee, SH Yoon, Y Jung, N Kim, U Min, J Chun, I Choi_


With increasing attention being paid to improving emotional well-being, recent evidence points to gut microbiota as a key player in regulating mental and physical health via bidirectional communication between the brain and gut. Here, we examine the association between emotional well-being and gut microbiome profiles (i.e., gut microbiome composition, diversity, and the moderating role of the enterotypes) among healthy Korean adults (n = 83, mean age = 48.9, SD = 13.2). The research was performed using high-throughput 16S rRNA gene sequencing to obtain gut microbiome profiles, as well as a self-report survey that included the Positive Affect Negative Affect Schedule (PANAS). The cluster-based analysis identified two enterotypes dominated by the genera Bacteroides (n = 49) and Prevotella (n = 34). Generalized linear regression analysis reveals significant associations between positive emotion and gut microbiome diversity (Shannon Index) among participants in the Prevotella dominant group, whereas no such relationship emerged among participants in the Bacteroides group. Moreover, a novel genus from the family Lachnospiraceae is associated with emotional well-being scores, both positive and negative. Together, the current findings highlight the enterotype-specific links between the gut microbiota community and emotion in healthy adults and suggest the possible roles of the gut microbiome in promoting mental health.

[Source](https://www.nature.com/articles/s41598-020-77673-z)

10.1038/s41598-020-77673-z

---

## The Composition and Concordance of Lactobacillus Populations of Infant Gut and the Corresponding Breast-Milk and Maternal Gut.
 07.01.2021, _X Zhang, S Mushajiang, B Luo, F Tian, Y Ni, W Yan_


The maternal gut is the principal source of commensal bacteria in the infant gut during the lactation stage, where breast milk acts as an intermediary for the transfer of potential probiotic bacteria consortia, including Lactobacillus. This study aimed to characterize the bacterial communities in human milk, maternal, and infant feces in a small yet very homogeneous cohort of 25 healthy mother-infant pairs in northwestern China (n = 25, infant age from 7 days to 2 years), with special emphasis on the cooccurrence and vertical transfer of Lactobacillus phylotypes at the species or strain level in mother-breast milk-infant triads. Accurate sequencing analysis revealed that among 73 Lactobacillus zero-radius operational classification units (ZOTUs) identified, 58 belonging to 18 recognized species or species groups were distributed in all three types of samples. Lactobacillus ruminis, L. mucosae and L. gasseri-johnsonii as true residents were the most represented in all three ecosystems, whereas the content of Lactobacillus phylotypes commonly developed as probiotics was not dominant. While the numbers of Lactobacillus species in breast milk and infant feces were greater than that in maternal feces, principal coordinates analysis (PCoA) based on beta diversity, coupled with the frequency of isolates determined by culture methods, showed that the Lactobacillus community in the infant gut was more similar to that in the maternal gut than to that in breast milk, suggesting that the gut is niche selective for Lactobacillus populations. In addition, identical strains of L. ruminis, L. paracasei, L. mucosae and L. salivarius were isolated from multiple mother-infant pairs, supporting the hypothesis that vertical transfer of bacteria via breastfeeding contributes to the initial establishment of the microbiota in the developing infant intestine.

[Source](https://www.frontiersin.org/articles/10.3389/fmicb.2020.597911/full)

10.3389/fmicb.2020.597911

---

## Lifestyle modifications result in alterations in the gut microbiota in obese children.
 07.01.2021, _KY Cho_


The association between the gut microbiota and pediatric obesity was analyzed in a cross-sectional study. A prospective study of obese children was conducted to assess the gut microbial alterations after a weight change. We collected fecal samples from obese children before and after a 2-month weight reduction program that consisted of individual counseling for nutritional education and physical activity, and we performed 16S rRNA gene amplicon sequencing using an Illumina MiSeq platform.

[Source](https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-020-02002-3)

10.1186/s12866-020-02002-3

---

## Nutritional combinatorial impact on the gut microbiota and plasma short-chain fatty acids levels in the prevention of mammary cancer in Her2/neu estrogen receptor-negative transgenic mice.
 06.01.2021, _M Sharma, I Arora, ML Stoll, Y Li, CD Morrow, S Barnes, TF Berryhill, S Li, TO Tollefsbol_


Breast cancer is the second leading cause of cancer-related mortality in women. Various nutritional compounds possess anti-carcinogenic properties which may be mediated through their effects on the gut microbiota and its production of short-chain fatty acids (SCFAs) for the prevention of breast cancer. We evaluated the impact of broccoli sprouts (BSp), green tea polyphenols (GTPs) and their combination on the gut microbiota and SCFAs metabolism from the microbiota in Her2/neu transgenic mice that spontaneously develop estrogen receptor-negative [ER(-)] mammary tumors. The mice were grouped based on the dietary treatment: control, BSp, GTPs or their combination from beginning in early life (BE) or life-long from conception (LC). We found that the combination group showed the strongest inhibiting effect on tumor growth volume and a significant increase in tumor latency. BSp treatment was integrally more efficacious than the GTPs group when compared to the control group. There was similar clustering of microbiota of BSp-fed mice with combination-fed mice, and GTPs-fed mice with control-fed mice at pre-tumor in the BE group and at pre-tumor and post-tumor in the LC group. The mice on all dietary treatment groups incurred a significant increase of Adlercreutzia, Lactobacillus genus and Lachnospiraceae, S24-7 family in the both BE and LC groups. We found no change in SCFAs levels in the plasma of BSp-fed, GTPs-fed and combination-fed mice of the BE group. Marked changes were observed in the mice of the LC group consisting of significant increases in propionate and isobutyrate in GTPs-fed and combination-fed mice. These studies indicate that nutrients such as BSp and GTPs differentially affect the gut microbial composition in both the BE and LC groups and the key metabolites (SCFAs) levels in the LC group. The findings also suggest that temporal factors related to different time windows of consumption during the life-span can have a promising influence on the gut microbial composition, SCFAs profiles and ER(-) breast cancer prevention.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234893)

10.1371/journal.pone.0234893

---

## Modeling dysbiosis of human NASH in mice: Loss of gut microbiome diversity and overgrowth of Erysipelotrichales.
 04.01.2021, _JK Carter, D Bhattacharya, JN Borgerding, MI Fiel, JJ Faith, SL Friedman_


Non-alcoholic steatohepatitis (NASH) is a severe form of non-alcoholic fatty liver disease (NAFLD) that is responsible for a growing fraction of cirrhosis and liver cancer cases worldwide. Changes in the gut microbiome have been implicated in NASH pathogenesis, but the lack of suitable murine models has been a barrier to progress. We have therefore characterized the microbiome in a well-validated murine NASH model to establish its value in modeling human disease.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0244763)

10.1371/journal.pone.0244763

---

## Geography as non-genetic modulation factor of chicken cecal microbiota.
 06.01.2021, _N Pin Viso, E Redondo, JM Díaz Carrasco, L Redondo, J Sabio Y Garcia, M Fernández Miyakawa, MD Farber_


The gastrointestinal tract of chickens harbors a highly diverse microbiota contributing not only to nutrition, but also to the physiological development of the gastrointestinal tract. Microbiota composition depends on many factors such as the portion of the intestine as well as the diet, age, genotype, or geographical origin of birds. The aim of the present study was to demonstrate the influence of the geographical location over the cecal microbiota from broilers. We used metabarcoding sequencing datasets of the 16S rRNA gene publicly available to compare the composition of the Argentine microbiota against the microbiota of broilers from another seven countries (Germany, Australia, Croatia, Slovenia, United States of America, Hungary, and Malaysia). Geographical location played a dominant role in shaping chicken gut microbiota (Adonis R2 = 0.6325, P = 0.001; Mantel statistic r = 0.1524, P = 4e-04) over any other evaluated factor. The geographical origin particularly affected the relative abundance of the families Bacteroidaceae, Lactobacillaceae, Lachnospiraceae, Ruminococcaceae, and Clostridiaceae. Because of the evident divergence of microbiota among countries we coined the term "local microbiota" as convergent feature that conflates non-genetic factors, in the perspective of human-environmental geography. Local microbiota should be taken into consideration as a native overall threshold value for further appraisals when testing the production performance and performing correlation analysis of gut microbiota modulation against different kind of diet and/or management approaches. In this regard, we described the Argentine poultry cecal microbiota by means of samples both from experimental trials and commercial farms. Likewise, we were able to identify a core microbiota composed of 65 operational taxonomic units assigned to seven phyla and 38 families, with the four most abundant taxa belonging to Bacteroides genus, Rikenellaceae family, Clostridiales order, and Ruminococcaceae family.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0244724)

10.1371/journal.pone.0244724

---

## Intestinal microbial communities of rainbow trout (Oncorhynchus mykiss) may be improved by feeding a Hermetia illucens meal/low-fishmeal diet.
 03.01.2021, _S Rimoldi, M Antonini, L Gasco, F Moroni, G Terova_


With demands and reliance on aquaculture still growing, there are various challenges to allow sustainable growth and the shift from fishmeal (FM) to other protein sources in aquafeed formulations is one of the most important. In this regard, interest in the use of insect meal (IM) in aquafeeds has grown rapidly. Accordingly, the aim of the present study was to assess the effects of dietary IM from Hermetia illucens (Hi) larvae included in a low-FM diet on gut microbial communities of rainbow trout (Oncorhynchus mykiss), in terms of both composition and function of microbiome. A feeding trial was conducted using 192 trout of about 100-g mean initial weight. Fish were fed in quadruplicate (4 tanks/diet) for 131 days with two diets: the control (Ctrl) contained 20% of FM as well as other protein sources, whereas the Hi diet contained 15% of Hi larvae meal to replace 50% of the FM contained in the Ctrl diet. High-throughput sequencing of 16S rRNA gene was used to identify the major feed and gut bacterial taxa, whereas Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt) analysis was performed on gut bacterial genomes to identify the major active biological pathways. The inclusion of IM led to an increase in Firmicutes, mainly represented by Bacilli class and to a drastic reduction of Proteobacteria. Beneficial genera, such as Lactobacillus and Bacillus, were enriched in the gut of fish fed with the Hi diet, whereas the number of bacteria assigned to the pathogenic Aeromonas genus was drastically reduced in the same fish group. The metagenome functional data provided evidence that dietary IM inclusion can shape the metabolic activity of trout gut microbiota. In particular, intestinal microbiome of fish fed with IM may have the capacity to improve dietary carbohydrate utilization. Therefore, H. illucens meal is a promising protein source for trout nutrition, able to modulate gut microbial community by increasing the abundance of some bacteria taxa that are likely to play a key role in fish health.

[Source](https://link.springer.com/article/10.1007/s10695-020-00918-1)

10.1007/s10695-020-00918-1

---

## Dramatic shifts in intestinal fungal community between wintering Hooded Crane and Domestic Goose

[Source](https://avianres.biomedcentral.com/articles/10.1186/s40657-020-00238-1)

---

## Bacterial diversity in intestinal mucosa of mice fed with Dendrobium officinale and high-fat diet

[Source](https://link.springer.com/article/10.1007/s13205-020-02558-x)

---

## Shift of Maternal Gut Microbiota of Tibetan Antelope (Pantholops hodgsonii) During the Periparturition Period.
 07.01.2021, _Y Shi, ZY Miao, JP Su, SK Wasser_


The maternal gut microbiota can influence and be affected by the substantial physiological changes taking place during the periparturition period. However, little information is known about the changes in the maternal gut microbiota and hormonal variations during this period in nonmodel organisms. Tibetan antelope (Pantholops hodgsonii) provide a unique system to address this issue because their summer migration cycle is synchronized with the periparturition period. Here, we used fecal microbiota as a proxy of gut microbiota. We characterized fecal microbial community of female migratory Tibetan antelope in the late pregnancy and postpartum periods using 16S rRNA gene sequencing and quantified fecal glucocorticoids (GCs) and triiodothyronine (T3) metabolite concentrations through enzyme immunoassays to identify the associations between maternal gut microbiota and physiological changes related with reproduction. We found that the fecal microbiota of Tibetan antelope was dominated by Firmicutes and Bacteroidetes. The microbial composition was significantly altered during the transition from late pregnancy to the postpartum period. Fecal T3 concentration was significantly higher in the postpartum period compared to late pregnancy, whereas GC metabolite concentration did not significantly differ between two reproductive states. We identified six genera (Anaerofustis, Bacteroides, Coprococcus_2, Ruminiclostridium_5, Ruminococcaceae_UCG-007, and Tyzzerella) that were significantly associated with reproductive states. We also found two genera (Christensenellaceae_R-7_group and Rikenellaceae_RC9_gut_group) significantly associated with GC metabolite concentration and two genera (Agathobacter and Papillibacter) significantly associated with T3 metabolite concentration, though these correlations were weak with coefficient values ranging from - 0.007 to 0.03. Our results indicate that many members of the gut microbiota are associated with the physiological changes in the transition from late pregnancy to the postpartum period, likely reflecting the metabolic and immune system dynamics during the periparturition period. This study highlights the importance of integrating microbiota, hormones and migration pattern to study the reproductive health of wildlife. By establishing a baseline of the physiological changes during the migration/periparturition period, we can have a better understanding of the impacts of increasing human activities on the Tibetan Plateau on the reproductive health of Tibetan antelope.

[Source](https://link.springer.com/article/10.1007%2Fs00284-020-02339-y)

10.1007/s00284-020-02339-y

---

## Bacterial seed endophyte shapes disease resistance in rice.
 05.01.2021, _H Matsumoto, X Fan, Y Wang, P Kusstatscher, J Duan, S Wu, S Chen, K Qiao, Y Wang, B Ma, G Zhu, Y Hashidoko, G Berg, T Cernava, M Wang_


Cereal crop production is severely affected by seed-borne bacterial diseases across the world. Locally occurring disease resistance in various crops remains elusive. Here, we have observed that rice plants of the same cultivar can be differentiated into disease-resistant and susceptible phenotypes under the same pathogen pressure. Following the identification of a seed-endophytic bacterium as the resistance-conferring agent, integration of high-throughput data, gene mutagenesis and molecular interaction assays facilitated the discovery of the underlying mode of action. Sphingomonas melonis that is accumulated and transmitted across generations in disease-resistant rice seeds confers resistance to disease-susceptible phenotypes by producing anthranilic acid. Without affecting cell growth, anthranilic acid interferes with the sigma factor RpoS of the seed-borne pathogen Burkholderia plantarii, probably leading to impairment of upstream cascades that are required for virulence factor biosynthesis. The overall findings highlight the hidden role of seed endophytes in the phytopathology paradigm of 'disease triangles', which encompass the plant, pathogens and environmental conditions. These insights are potentially exploitable for modern crop cultivation threatened by globally widespread bacterial diseases.

[Source](https://www.nature.com/articles/s41477-020-00826-5)

10.1038/s41477-020-00826-5

---

## Metagenomics survey unravels diversity of biogas microbiomes with potential to enhance productivity in Kenya.
 04.01.2021, _SM Muturi, LW Muthui, PM Njogu, JM Onguso, FN Wachira, SO Opiyo, R Pelle_


The obstacle to optimal utilization of biogas technology is poor understanding of biogas microbiomes diversities over a wide geographical coverage. We performed random shotgun sequencing on twelve environmental samples. Randomized complete block design was utilized to assign the twelve treatments to four blocks, within eastern and central regions of Kenya. We obtained 42 million paired-end reads that were annotated against sixteen reference databases using two ENVO ontologies, prior to β-diversity studies. We identified 37 phyla, 65 classes and 132 orders. Bacteria dominated and comprised 28 phyla, 42 classes and 92 orders, conveying substrate's versatility in the treatments. Though, Fungi and Archaea comprised 5 phyla, the Fungi were richer; suggesting the importance of hydrolysis and fermentation in biogas production. High β-diversity within the taxa was largely linked to communities' metabolic capabilities. Clostridiales and Bacteroidales, the most prevalent guilds, metabolize organic macromolecules. The identified Cytophagales, Alteromonadales, Flavobacteriales, Fusobacteriales, Deferribacterales, Elusimicrobiales, Chlamydiales, Synergistales to mention but few, also catabolize macromolecules into smaller substrates to conserve energy. Furthermore, δ-Proteobacteria, Gloeobacteria and Clostridia affiliates syntrophically regulate PH2 and reduce metal to provide reducing equivalents. Methanomicrobiales and other Methanomicrobia species were the most prevalence Archaea, converting formate, CO2(g), acetate and methylated substrates into CH4(g). Thermococci, Thermoplasmata and Thermoprotei were among the sulfur and other metal reducing Archaea that contributed to redox balancing and other metabolism within treatments. Eukaryotes, mainly fungi were the least abundant guild, comprising largely Ascomycota and Basidiomycota species. Chytridiomycetes, Blastocladiomycetes and Mortierellomycetes were among the rare species, suggesting their metabolic and substrates limitations. Generally, we observed that environmental and treatment perturbations influenced communities' abundance, β-diversity and reactor performance largely through stochastic effect. Understanding diversity of biogas microbiomes over wide environmental variables and its' productivity provided insights into better management strategies that ameliorate biochemical limitations to effective biogas production.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0244755)

10.1371/journal.pone.0244755

---

## Wastewater treatment alters microbial colonization of microplastics.
 06.01.2021, _JJ Kelly, MG London, AR McCormick, M Rojas, JW Scott, TJ Hoellein_


Microplastics are ubiquitous contaminants in aquatic habitats globally, and wastewater treatment plants (WWTPs) are point sources of microplastics. Within aquatic habitats microplastics are colonized by microbial biofilms, which can include pathogenic taxa and taxa associated with plastic breakdown. Microplastics enter WWTPs in sewage and exit in sludge or effluent, but the role that WWTPs play in establishing or modifying microplastic bacterial assemblages is unknown. We analyzed microplastics and associated biofilms in raw sewage, effluent water, and sludge from two WWTPs. Both plants retained &gt;99% of influent microplastics in sludge, and sludge microplastics showed higher bacterial species richness and higher abundance of taxa associated with bioflocculation (e.g. Xanthomonas) than influent microplastics, suggesting that colonization of microplastics within the WWTP may play a role in retention. Microplastics in WWTP effluent included significantly lower abundances of some potentially pathogenic bacterial taxa (e.g. Campylobacteraceae) compared to influent microplastics; however, other potentially pathogenic taxa (e.g. Acinetobacter) remained abundant on effluent microplastics, and several taxa linked to plastic breakdown (e.g. Klebsiella, Pseudomonas, and Sphingomonas) were significantly more abundant on effluent compared to influent microplastics. These results indicate that diverse bacterial assemblages colonize microplastics within sewage and that WWTPs can play a significant role in modifying the microplastic-associated assemblages, which may affect the fate of microplastics within the WWTPs and the environment.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0244443)

10.1371/journal.pone.0244443

---

## MetGEMs Toolbox: Metagenome-scale models as integrative toolbox for uncovering metabolic functions and routes of human gut microbiome.
 06.01.2021, _P Patumcharoenpol, M Nakphaichit, G Panagiotou, A Senavonge, N Suratannon, W Vongsangnak_


Investigating metabolic functional capability of a human gut microbiome enables the quantification of microbiome changes, which can cause a phenotypic change of host physiology and disease. One possible way to estimate the functional capability of a microbial community is through inferring metagenomic content from 16S rRNA gene sequences. Genome-scale models (GEMs) can be used as scaffold for functional estimation analysis at a systematic level, however up to date, there is no integrative toolbox based on GEMs for uncovering metabolic functions. Here, we developed the MetGEMs (metagenome-scale models) toolbox, an open-source application for inferring metabolic functions from 16S rRNA gene sequences to facilitate the study of the human gut microbiome by the wider scientific community. The developed toolbox was validated using shotgun metagenomic data and shown to be superior in predicting functional composition in human clinical samples compared to existing state-of-the-art tools. Therefore, the MetGEMs toolbox was subsequently applied for annotating putative enzyme functions and metabolic routes related in human disease using atopic dermatitis as a case study.

[Source](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008487)

10.1371/journal.pcbi.1008487

---

## Assessment of two DNA extraction kits for profiling poultry respiratory microbiota from multiple sample types.
 06.01.2021, _MEC Abundo, JM Ngunjiri, KJM Taylor, H Ji, A Ghorbani, M K C, BP Weber, TJ Johnson, CW Lee_


Characterization of poultry microbiota is becoming increasingly important due to the growing need for microbiome-based interventions to improve poultry health and production performance. However, the lack of standardized protocols for sampling, sample processing, DNA extraction, sequencing, and bioinformatic analysis can hinder data comparison between studies. Here, we investigated how the DNA extraction process affects microbial community compositions and diversity metrics in different chicken respiratory sample types including choanal and tracheal swabs, nasal cavity and tracheal washes, and lower respiratory lavage. We did a side-by-side comparison of the performances of Qiagen DNeasy blood and tissue (BT) and ZymoBIOMICS DNA Miniprep (ZB) kits. In general, samples extracted with the BT kit yielded higher concentrations of total DNA while those extracted with the ZB kit contained higher numbers of bacterial 16S rRNA gene copies per unit volume. Therefore, the samples were normalized to equal amounts of 16S rRNA gene copies prior to sequencing. For each sample type, all predominant bacterial taxa detected in samples extracted with one kit were present in replicate samples extracted with the other kit and did not show significant differences at the class level. However, a few differentially abundant shared taxa were observed at family and genus levels. Furthermore, between-kit differences in alpha and beta diversity metrics at the amplicon sequence variant level were statistically indistinguishable. Therefore, both kits perform similarly in terms of 16S rRNA gene-based poultry microbiome analysis for the sample types analyzed in this study.

[Source](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0241732)

10.1371/journal.pone.0241732

---

