## <em>Aggregatibacter</em> is inversely associated with inflammatory mediators in sputa of patients with chronic airway diseases and reduces inflammation <em>in vitro</em>

[Source](https://doi.org/10.1186/s12931-024-02983-z)

---

## Oral Microbiota Development in the First 60 Months: A Longitudinal Study

[Source](https://doi.org/10.1177/00220345241272011)

---

## Microbiome-derived metabolites in early to mid-pregnancy and risk of gestational diabetes: a metabolome-wide association study.
 14.10.2024, _SM Susarla, O Fiehn, I Thiele, AL Ngo, DK Barupal, RF Chehab, A Ferrara, Y Zhu_


_Humans, Female, Diabetes, Gestational, Pregnancy, Adult, Prospective Studies, Metabolome, Case-Control Studies, Microbiota, Metabolomics_
Pre-diagnostic disturbances in the microbiome-derived metabolome have been associated with an increased risk of diabetes in non-pregnant populations. However, the roles of microbiome-derived metabolites, the end-products of microbial metabolism, in gestational diabetes (GDM) remain understudied. We examined the prospective association of microbiome-derived metabolites in early to mid-pregnancy with GDM risk in a diverse population.

[Source](https://doi.org/10.1186/s12916-024-03606-6)

10.1186/s12916-024-03606-6

---

## Unique Gut Microbiome and Metabolic Profiles in Chinese Workers Exposed to Dust: Insights from a Case-Control Study.
 11.10.2024, _X Qian, Y Liu, X Wei, X Chen, G Rong, X Hu_


This study aimed to identify distinct gut microbiome and serum metabolic features in workers exposed to dust compared to healthy controls.

[Source](https://doi.org/10.1097/JOM.0000000000003243)

10.1097/JOM.0000000000003243

---

## Dysbiotic signatures and diagnostic potential of gut microbial markers for inflammatory bowel disease in Korean population.
 13.10.2024, _HS Kim, SJ Oh, BK Kim, JE Kim, BH Kim, YK Park, BG Yang, JY Lee, JW Bae, CK Lee_


_Humans, Gastrointestinal Microbiome, Dysbiosis, Female, Male, Republic of Korea, Adult, Middle Aged, Biomarkers, RNA, Ribosomal, 16S, Feces, Inflammatory Bowel Diseases, Colitis, Ulcerative, Metagenomics, Crohn Disease, Case-Control Studies, Cross-Sectional Studies, Young Adult, Aged_
Fecal samples were collected from 640 individuals in Korea, including 523 patients with IBD (223 with Crohn's disease [CD] and 300 with ulcerative colitis [UC]) and 117 healthy controls. The samples were subjected to cross-sectional gut metagenomic analysis using 16 S rRNA sequencing and bioinformatics analysis. Patients with IBD, particularly those with CD, exhibited significantly lower alpha diversities than the healthy subjects. Differential abundance analysis revealed dysbiotic signatures, characterized by an expansion of the genus Escherichia-Shigella in patients with CD. Functional annotations showed that functional pathways related to bacterial pathogenesis and production of hydrogen sulfide (H2S) were strongly upregulated in patients with CD. A dysbiosis score, calculated based on functional characteristics, highly correlated with disease severity. Markers distinguishing between healthy subjects and patients with IBD showed accurate classification based on a small number of microbial taxa, which may be used to diagnose ambiguous cases. These findings confirm the taxonomic and functional dysbiosis of the gut microbiota in patients with IBD, especially those with CD. Taxa indicative of dysbiosis may have significant implications for future clinical research on the management and diagnosis of IBD.

[Source](https://doi.org/10.1038/s41598-024-74002-6)

10.1038/s41598-024-74002-6

---

## A search for bacteria identified from cerebrospinal fluid shunt infections in previous surgical events

[Source](https://doi.org/10.1371/journal.pone.0311605)

---

## Microbiome and metabolome patterns after lung transplantation reflect underlying disease and chronic lung allograft dysfunction

[Source](https://doi.org/10.1186/s40168-024-01893-y)

---

## Reexamining the role of <em>Fusobacterium nucleatum</em> subspecies in clinical and experimental studies

[Source](https://doi.org/10.1080/19490976.2024.2415490)

---

## Host-microbe interaction paradigms in acute and recurrent vulvovaginal candidiasis

[Source](https://doi.org/10.1016/j.chom.2024.08.018)

---

## Balancing act: counteracting adverse drug effects on the microbiome

[Source](https://doi.org/10.1016/j.tim.2024.09.011)

---

## Gut microbiome and bladder cancer: A new link through nitrosamine metabolism

[Source](https://doi.org/10.1016/j.chom.2024.09.003)

---

## Prostate cancer microenvironment: multidimensional regulation of immune cells, vascular system, stromal cells, and microbiota

[Source](https://doi.org/10.1186/s12943-024-02137-1)

---

## Gut and oral microbiota in gynecological cancers: interaction, mechanism, and therapeutic value

[Source](https://doi.org/10.1038/s41522-024-00577-7)

---

## Effects of complex probiotics on intestinal function and its regulatory mechanism in patients with constipation

[Source](https://doi.org/10.1163/18762891-bja00039)

---

## <em>Lactiplantibacillus plantarum</em> P101 Alleviated Alcohol-Induced Hepatic Lipid Accumulation in Mice via AMPK Signaling Pathway: Gut Microbiota and Metabolomics Analysis

[Source](https://doi.org/10.1007/s12602-024-10373-6)

---

## Unlocking the genetic influence on milk variation and its potential implication for infant health

[Source](https://doi.org/10.1016/j.xgen.2024.100676)

---

## The gut microbiome and dietary fibres: implications in obesity, cardiometabolic diseases and cancer

[Source](https://doi.org/10.1038/s41579-024-01108-z)

---

## Microbiome-based therapeutics for Parkinson’s disease

[Source](https://doi.org/10.1016/j.neurot.2024.e00462)

---

## Gut bacteria of lepidopteran herbivores facilitate digestion of plant toxins

[Source](https://doi.org/10.1073/pnas.2412165121)

---

## Sea cucumbers and their symbiotic microbiome have evolved to feed on seabed sediments.
 14.10.2024, _W Pan, X Wang, C Ren, X Jiang, S Gong, Z Xie, NK Wong, X Li, J Huang, D Fan, P Luo, Y Yang, X Ren, S Yu, Z Qin, X Wu, D Huo, B Ma, Y Liu, X Zhang, Z E, J Liang, H Sun, L Yuan, X Liu, C Cheng, H Long, J Li, Y Wang, C Hu, T Chen_


_Animals, Symbiosis, Geologic Sediments, Sea Cucumbers, Glycoside Hydrolases, Gastrointestinal Microbiome, Holothuria, Phylogeny, Biological Evolution, Ecosystem, Feeding Behavior, Microbiota, Bacteria_
Sea cucumbers are predominant deposit feeders in benthic ecosystems, providing protective benefits to coral reefs by reducing disease prevalence. However, how they receive sufficient nutrition from seabed sediments remains poorly understood. Here, we investigate Holothuria leucospilota, an ecologically significant tropical sea cucumber, to elucidate digestive mechanisms underlying marine deposit-feeding. Genomic analysis reveals intriguing evolutionary adaptation characterized by an expansion of digestive carbohydrase genes and a contraction of digestive protease genes, suggesting specialization in digesting microalgae. Developmentally, two pivotal dietary shifts, namely, from endogenous nutrition to planktonic feeding, and from planktonic feeding to deposit feeding, induce changes in digestive tract enzyme profiles, with adults mainly expressing carbohydrases and lipases. A nuanced symbiotic relationship exists between gut microbiota and the host, namely, specific resident bacteria supply crucial enzymes for food digestion, while other bacteria are digested and provide assimilable nutrients. Our study further identifies Holothuroidea lineage-specific lysozymes that are restrictedly expressed in the intestines to support bacterial digestion. Overall, this work advances our knowledge of the evolutionary innovations in the sea cucumber digestive system which enable them to efficiently utilize nutrients from seabed sediments and promote food recycling within marine ecosystems.

[Source](https://doi.org/10.1038/s41467-024-53205-5)

10.1038/s41467-024-53205-5

---

## Comprehensive profile of the companion animal gut microbiome integrating reference-based and reference-free methods

[Source](https://doi.org/10.1093/ismejo/wrae201)

---

## Quantitative Synthesis of Microbe-Driven Acclimation and Adaptation in Wild Vertebrates

[Source](https://doi.org/10.1111/eva.70025)

---

## The role of foraging pollinators in assembling the flower microbiota and transmitting the fire blight pathogen <em>Erwinia amylovora</em>

[Source](https://doi.org/10.1111/1462-2920.16702)

---

## Distinct microbial communities supported by iron oxidation

[Source](https://doi.org/10.1111/1462-2920.16706)

---

## Normalization of short-chain fatty acid concentration by bacterial count of stool samples improves discrimination between eubiotic and dysbiotic gut microbiota caused by <em>Clostridioides difficile </em>infection

[Source](https://doi.org/10.1080/19490976.2024.2415488)

---

## SmT/SHM-seq: simultaneously capturing spatial transcriptome and microbiome information in plants.
 12.10.2024, _P Mu, W Li, LP Tran, X Li_



[Source](https://doi.org/10.1016/j.tplants.2024.09.010)

10.1016/j.tplants.2024.09.010

---

## High accuracy meets high throughput for near full-length 16S ribosomal RNA amplicon sequencing on the Nanopore platform.
 11.10.2024, _X Lin, K Waring, H Ghezzi, C Tropini, J Tyson, RM Ziels_


Small subunit (SSU) ribosomal RNA (rRNA) gene amplicon sequencing is a foundational method in microbial ecology. Currently, short-read platforms are commonly employed for high-throughput applications of SSU rRNA amplicon sequencing, but at the cost of poor taxonomic classification due to limited fragment lengths. The Oxford Nanopore Technologies (ONT) platform can sequence full-length SSU rRNA genes, but its lower raw-read accuracy has so-far limited accurate taxonomic classification and de novo feature generation. Here, we present a sequencing workflow, termed ssUMI, that combines unique molecular identifier (UMI)-based error correction with newer (R10.4+) ONT chemistry and sample barcoding to enable high throughput near full-length SSU rRNA (e.g. 16S rRNA) amplicon sequencing. The ssUMI workflow generated near full-length 16S rRNA consensus sequences with 99.99% mean accuracy using a minimum subread coverage of 3×, surpassing the accuracy of Illumina short reads. The consensus sequences generated with ssUMI were used to produce error-free de novo sequence features with no false positives with two microbial community standards. In contrast, Nanopore raw reads produced erroneous de novo sequence features, indicating that UMI-based error correction is currently necessary for high-accuracy microbial profiling with R10.4+ ONT sequencing chemistries. We showcase the cost-competitive scalability of the ssUMI workflow by sequencing 87 time-series wastewater samples and 27 human gut samples, obtaining quantitative ecological insights that were missed by short-read amplicon sequencing. ssUMI, therefore, enables accurate and low-cost full-length 16S rRNA amplicon sequencing on Nanopore, improving accessibility to high-resolution microbiome science.

[Source](https://doi.org/10.1093/pnasnexus/pgae411)

10.1093/pnasnexus/pgae411

---

## A deployable film method to enable replicable sampling of low-abundance environmental microbiomes.
 14.10.2024, _P Mankiewicz Ledins, EZ Lin, C Bhattacharya, KJG Pollitt, AH Dyson, EM Hénaff_


_Microbiota, Humans, Specimen Handling, Dimethylpolysiloxanes, DNA, Bacterial, Environmental Monitoring, Environmental Microbiology_
Urbanizing global populations spend over 90% of their time indoors where microbiome abundance and diversity are low. Chronic exposure to microbiomes with low abundance and diversity have demonstrated negative long-term impacts on human health. Sequencing-based analyses of environmental nucleic acids are critical to understanding the impact of the indoor microbiome on human health, however low DNA yields indoors, alongside sample collection and processing inconsistencies, currently challenge study replicability. This study presents a comparative assessment of a novel, passive, easily replicable sampling strategy using polydimethylsiloxane (PDMS) sheets alongside a representative swab-based collection protocol. Deployable, customizable PDMS films designed for whole-sample insertion into standardized extraction kits demonstrated 43% higher DNA yields per sample, and 76% higher yields per cm2 of sampler over swab-based protocols. These results indicate that this accessible, scalable method enables sufficient DNA collection to comprehensively evaluate indoor microbiome exposures and potential human health impacts using smaller, more space efficient samplers, representing an attractive alternative to swab-based collection. In addition, this process reduces the manual steps required for microbiome sampling which could address inter-study variability, transform the current microbiome sampling paradigm, and ultimately benefit the replicability and accessibility of microbiome exposure studies.

[Source](https://doi.org/10.1038/s41598-024-72341-y)

10.1038/s41598-024-72341-y

---

